<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">

<!--Converted with LaTeX2HTML 2002-2-1 (1.71)
original version by:  Nikos Drakos, CBLU, University of Leeds
* revised and updated by:  Marcus Hennecke, Ross Moore, Herb Swan
* with significant contributions from:
  Jens Lippmann, Marek Rouchal, Martin Wilck and others -->
<HTML>
<HEAD>
<TITLE>MELTING - nearest-neighbor computation of nucleic acid hybridation</TITLE>
<META NAME="description" CONTENT="MELTING - nearest-neighbor computation of nucleic acid hybridation">
<META NAME="keywords" CONTENT="melting">
<META NAME="resource-type" CONTENT="document">
<META NAME="distribution" CONTENT="global">

<META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=iso-8859-1">
<META NAME="Generator" CONTENT="LaTeX2HTML v2002-2-1">
<META HTTP-EQUIV="Content-Style-Type" CONTENT="text/css">

<LINK REL="STYLESHEET" HREF="html/melting.css">

</HEAD>

<BODY >

<DIV CLASS="navigation"><!--Navigation Panel-->
<IMG WIDTH="81" HEIGHT="24" ALIGN="BOTTOM" BORDER="0" ALT="next_inactive"
 SRC="/usr/share/latex2html/icons/nx_grp_g.png"> 
<IMG WIDTH="26" HEIGHT="24" ALIGN="BOTTOM" BORDER="0" ALT="up"
 SRC="/usr/share/latex2html/icons/up_g.png"> 
<IMG WIDTH="63" HEIGHT="24" ALIGN="BOTTOM" BORDER="0" ALT="previous"
 SRC="/usr/share/latex2html/icons/prev_g.png">   
<BR>
<BR><BR></DIV>
<!--End of Navigation Panel-->

<H1 ALIGN=CENTER><SMALL>MELTING</SMALL> - nearest-neighbor computation of nucleic acid hybridation</H1>
<P ALIGN=CENTER><STRONG>Marine Dumousseau</STRONG>
</P><P ALIGN=CENTER><STRONG>Nicolas Le Nov&#232;re</STRONG>
<BR><I><TT>lenov@ebi.ac.uk</TT></I>
</P><BR><P ALIGN=CENTER><B>Date:</B> August 2009</P>

<HR>

<P>

<P>
<BR>

<H2><A NAME="SECTION00010000000000000000">
Contents</A>
</H2>
<!--Table of Contents-->

<UL CLASS="TofC">
<LI><A NAME="tex2html87"
  HREF="melting.html#SECTION00020000000000000000">Synopsis</A>
<LI><A NAME="tex2html88"
  HREF="melting.html#SECTION00030000000000000000">Description </A>
<LI><A NAME="tex2html89"
  HREF="melting.html#SECTION00040000000000000000">Usage</A>
<UL>
<LI><A NAME="tex2html90"
  HREF="melting.html#SECTION00041000000000000000">Information about MELTING</A>
<LI><A NAME="tex2html91"
  HREF="melting.html#SECTION00042000000000000000">Mandatory options</A>
<LI><A NAME="tex2html92"
  HREF="melting.html#SECTION00043000000000000000">General options</A>
<LI><A NAME="tex2html93"
  HREF="melting.html#SECTION00044000000000000000">Set of thermodynamic parameters and methods (models)</A>
</UL><BR>
<LI><A NAME="tex2html94"
  HREF="melting.html#SECTION00050000000000000000">Algorithm </A>
<UL>
<LI><A NAME="tex2html95"
  HREF="melting.html#SECTION00051000000000000000">Thermodynamics of helix-coil transition of nucleic acid</A>
<LI><A NAME="tex2html96"
  HREF="melting.html#SECTION00052000000000000000">The melting temperature </A>
<LI><A NAME="tex2html97"
  HREF="melting.html#SECTION00053000000000000000">Correction for the concentration of nucleic acid </A>
<LI><A NAME="tex2html98"
  HREF="melting.html#SECTION00054000000000000000">Correction for the concentration of cations</A>
<LI><A NAME="tex2html99"
  HREF="melting.html#SECTION00055000000000000000">Correction for the concentration of denaturing agents</A>
<LI><A NAME="tex2html100"
  HREF="melting.html#SECTION00056000000000000000">Long sequences </A>
</UL><BR>
<LI><A NAME="tex2html101"
  HREF="melting.html#SECTION00060000000000000000">Bibliography</A>
<LI><A NAME="tex2html102"
  HREF="melting.html#SECTION00070000000000000000">See Also </A>
<LI><A NAME="tex2html103"
  HREF="melting.html#SECTION00080000000000000000">Copyright </A>
<LI><A NAME="tex2html104"
  HREF="melting.html#SECTION00090000000000000000">Acknowledgements</A>
<LI><A NAME="tex2html105"
  HREF="melting.html#SECTION000100000000000000000">Authors </A>
<LI><A NAME="tex2html106"
  HREF="melting.html#SECTION000110000000000000000">History </A>
</UL>
<!--End of Table of Contents-->
<P>

<P>

<P>

<H1><A NAME="SECTION00020000000000000000">
Synopsis</A>
</H1>
The nearest-neighbor approach is based on the fact that the helix-coil
transition works as a zipper. After an initial attachment, the hybridisation
propagates laterally.
The hybridization process depends on the adjacent nucleotides on each strand (the Crick's pairs).  
Two duplexes with the same base pairs could have different stabilities, and on the contrary, two 
duplexes with different sequences but identical sets of Crick's pairs will have the same
thermodynamics properties (see Sugimoto et al. 1994).
See Wetmur J.G (1991) and Santalucia (1998) for deep reviews on the nucleic acid hybridization
and on the different set of nearest-neighbor parameters.

<P>

<H1><A NAME="SECTION00030000000000000000">
Description </A>
</H1>

<P>
<SMALL>MELTING</SMALL> computes, for a nucleic acid duplex, the enthalpy and the
entropy of the helix-coil transition, and then its melting temperature. Four
types of hybridisation are possible: DNA/DNA, DNA/RNA, RNA/RNA and 2-O-Methyl RNA/RNA. The program
uses the method of nearest-neighbors. The set of thermodynamic parameters can be
easely changed, for instance following an experimental breakthrough. Melting is
a free program in both sense of the term. It comes with no cost and it is
open-source. In addition it is coded in Java (1.5) and can be compiled on any
operating system.

<P>
If you use <SMALL>MELTING</SMALL>, please quote

<P>
<BLOCKQUOTE>
Le Nov&#232;re. <SMALL>MELTING</SMALL>, a free tool to compute the
    melting temperature of nucleic acid duplex. <SPAN  CLASS="textit">Bioinformatics</SPAN>, 17: 1226-1227. 

</BLOCKQUOTE>

<P>

<H1><A NAME="SECTION00040000000000000000">
Usage</A>
</H1>

<P>
The options are treated sequentially. If there is a conflict between the value
of two options, the latter normally erases the former.

<P>

<H2><A NAME="SECTION00041000000000000000">
Information about MELTING</A>
</H2>
<DL>
<DT><STRONG><SPAN  CLASS="textbf">-h</SPAN></STRONG></DT>
<DD>
<BR>  Displays a short help and quit.
</DD>
<DT><STRONG><SPAN  CLASS="textbf">-L</SPAN></STRONG></DT>
<DD>
<BR>  Prints the legal informations and quit. 
</DD>
<DT><STRONG><SPAN  CLASS="textbf">-V</SPAN>  </STRONG></DT>
<DD>
<BR>  Displays the version number and quit.  
</DD>
<DT><STRONG><SPAN  CLASS="textbf">-p</SPAN></STRONG></DT>
<DD>
<BR>  Return the directory supposed to contain the sets of calorimetric parameters and quit. 
  If the environment variable NN_PATH is set, it is returned. Otherwise, the value
  defined by default during the compilation is returned.
</DD>
</DL>

<P>

<H2><A NAME="SECTION00042000000000000000">
Mandatory options</A>
</H2>
<DL>
<DT><STRONG><SPAN  CLASS="textbf">-S</SPAN> <SPAN  CLASS="textit">sequence</SPAN>  </STRONG></DT>
<DD>
<BR>  Sequence of one strand of the nucleic 
  acid duplex, entered 5' to 3'. <SPAN  CLASS="textbf">Important:</SPAN> 
  Uridine and thymidine are not considered as identical. The bases can be upper or lowercase.
</DD>
<DT><STRONG><SPAN  CLASS="textbf">-C</SPAN> <SPAN  CLASS="textit">complementary_sequence</SPAN></STRONG></DT>
<DD>
<BR>  Enters the complementary sequence, from 3' to 5'. This option is mandatory if
  there are mismatches, inosine(s) or hydroxyadenine(s) between the two strands. If it is not used, the program
  will compute it as the complement of the sequence entered with the option <SPAN  CLASS="textbf">-S</SPAN>. In case of self complementary sequences,
  The program can automatically detect the symmetry and deduce the complementary even though there is (are) dangling
  end(s) and it is not necessary to write the complementary sequence with the option <SPAN  CLASS="textbf">-C</SPAN>.
  Uridine and thymidine are not considered as identical. The bases can be upper or lowercase.
</DD>
<DT><STRONG><SPAN  CLASS="textbf">-E</SPAN> <SPAN  CLASS="textit">ion1_name=x.xxe-xx:ion2_name=x.xxe-xx:agent1_name=x.xxe-xx...</SPAN></STRONG></DT>
<DD>
<BR>  Enters the different ion (Na, Mg, Tris, K) or agent (dNTP, DMSO, formamide) concentrations. The effect  
  of  ions and denaturing agents on  thermodynamic  stability  of nucleic  acid duplexes is complex,
  and the correcting functions are  at  best rough  approximations. All the concentrations must be positive numeric
  values and in M. There are some exceptions for the DMSO concentrations (in
  %) and the formamide concentrations
  (in % or M depending on the used correction method). Be aware, the <!-- MATH
 $[\mbox{Tris}^+]$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">Tris<IMG
 WIDTH="19" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img8.png"
 ALT="$ ^+]$"></SPAN> is about half of the total tris buffer
  concentration.
  At least one cation concentration is mandatory, the other agents are optional. See the documentation for the concentration 
  limits. It depends on the used correction.
</DD>
<DT><STRONG><SPAN  CLASS="textbf">-P</SPAN> <SPAN  CLASS="textit">x.xxe-xx</SPAN></STRONG></DT>
<DD>
<BR>  Concentration of the nucleic acid strand in excess. It must be a strict positive numeric value and it is mandatory. The oligomer
  concentration is in mol/L.
</DD>
<DT><STRONG><SPAN  CLASS="textbf">-H</SPAN> <SPAN  CLASS="textit">hybridisation_type</SPAN></STRONG></DT>
<DD>
<BR>  Specifies the hybridisation type. Moreover this parameter determines the nature of the sequences entered by the user.
  Possible values are :
  <DL COMPACT>
<DT><SPAN  CLASS="textit">dnadna</SPAN></DT>
<DD>: DNA sequence (option <SPAN  CLASS="textbf">-S</SPAN>) and DNA complementary sequence (option <SPAN  CLASS="textbf">-C</SPAN>)
  
</DD>
<DT><SPAN  CLASS="textit">rnarna</SPAN></DT>
<DD>: RNA sequence (option <SPAN  CLASS="textbf">-S</SPAN>) and RNA complementary sequence (option <SPAN  CLASS="textbf">-C</SPAN>)
  
</DD>
<DT><SPAN  CLASS="textit">dnarna</SPAN></DT>
<DD>: DNA sequence (option <SPAN  CLASS="textbf">-S</SPAN>) and RNA complementary sequence (option <SPAN  CLASS="textbf">-C</SPAN>)
  
</DD>
<DT><SPAN  CLASS="textit">rnadna</SPAN></DT>
<DD>: RNA sequence (option <SPAN  CLASS="textbf">-S</SPAN>) and DNA complementary sequence (option <SPAN  CLASS="textbf">-C</SPAN>)
  
</DD>
<DT><SPAN  CLASS="textit">mrnarna</SPAN></DT>
<DD>: 2-o-methyl RNA sequence (option <SPAN  CLASS="textbf">-S</SPAN>) and RNA complementary sequence (option <SPAN  CLASS="textbf">-C</SPAN>)
  
</DD>
<DT><SPAN  CLASS="textit">mrnarna</SPAN></DT>
<DD>: RNA sequence (option <SPAN  CLASS="textbf">-S</SPAN>) and 2-o-methyl RNA complementary sequence (option <SPAN  CLASS="textbf">-C</SPAN>)
  
</DD>
</DL>
  This option is mandatory to select the default equations and methods to use.
</DD>
</DL>

<P>

<H2><A NAME="SECTION00043000000000000000">
General options</A>
</H2>
<DL>
<DT><STRONG><SPAN  CLASS="textbf">-T</SPAN> <SPAN  CLASS="textit">xxx</SPAN>  </STRONG></DT>
<DD>
<BR>Size threshold before approximative computation. The nearest-neighbour approach 
will be used by default if the length of the sequence is inferior to this threshold,
otherwise it is the approximative approach which will be used by default.
</DD>
<DT><STRONG><SPAN  CLASS="textbf">-v</SPAN></STRONG></DT>
<DD>
<BR>Activates the verbose mode, issuing a lot more information about the current run  (try it once 
to see if you can get something interesting).
</DD>
<DT><STRONG><SPAN  CLASS="textbf">-nnpath</SPAN> <SPAN  CLASS="textit">folder_pathway</SPAN></STRONG></DT>
<DD>
<BR>  Change the default pathway (Data) where to find the default calorimetric tables (thermodynamic parameters).
  The program will look for the file in a directory specified during the installation.
  However, if an environment variable NN_PATH is defined, melting will search in this one first.
</DD>
<DT><STRONG><SPAN  CLASS="textbf">-O</SPAN> <SPAN  CLASS="textit">output_file</SPAN></STRONG></DT>
<DD>
<BR>  The output is directed to this file instead of the standard output. The name of the file must be specified.
</DD>
<DT><STRONG><SPAN  CLASS="textbf">-self</SPAN></STRONG></DT>
<DD>
<BR>  To precise that the sequence entered with the option <SPAN  CLASS="textbf">-S</SPAN> is self complementary. No complementary sequence is mandatory. 
  The program automatically can detect a self complementary sequence for perfect matching sequences or sequences with dangling ends. 
  In these cases, the option <SPAN  CLASS="textbf">-self</SPAN> is not necessary. Otherwise we need to precise that the sequences are self complementary with this option. 
  examples: 

<P>
<PRE>
  Situation 1 : The sequence ATCGCGAT is self 
                complementary.
</PRE>

<P>
The option <SPAN  CLASS="textbf">-self</SPAN> is not necessary because the program can automatically detect it.

<P>
<PRE>
  Situation 2 : The sequence -TCGCGAT is self 
                complementary with a single 
		dangling end.
</PRE>

<P>
The option <SPAN  CLASS="textbf">-self</SPAN> is not necessary because the program 
  can automatically detect it.

<P>
<PRE>  
  Situation 3 :  If the sequence ATCCCGAT is self 
                 complementary with a single mismatch 
		(C/C)
</PRE>

<P>
The option <SPAN  CLASS="textbf">-self</SPAN> is necessary to precise the self 
  complementarity because the program can't detect it.

<P>
</DD>
<DT><STRONG><SPAN  CLASS="textbf">-F</SPAN> <SPAN  CLASS="textit">factor</SPAN>  </STRONG></DT>
<DD>
<BR>  This is the correction factor used to modulate the effect of the nucleic acid concentration in the computation of the melting temperature. 
  See section ALGORITHM for details. If the sequences are automatically recognized as self complementary sequences or if the option <SPAN  CLASS="textbf">-self</SPAN>
  is used, the factor correction is automatically 1. Otherwise F is 4 if the both strands are present in equivalent amount and 2 if one strand is in excess. 
  The default factor value is 4.
</DD>
</DL>

<P>

<H2><A NAME="SECTION00044000000000000000">
Set of thermodynamic parameters and methods (models)</A>
</H2>

<P>
By default, the approximative mode is used for oligonucleotides longer than 60 bases (the default threshold value), otherwise the nearest 
neighbor model is used. 
<DL>
<DT><STRONG><SPAN  CLASS="textbf">-am</SPAN> <SPAN  CLASS="textit">method_name</SPAN></STRONG></DT>
<DD>
<BR>  Forces to use a specific approximative formula, based on G+C content. You can use one of the following :

<P>
DNA <SMALL>DUPLEXES</SMALL>
    <DL COMPACT>
<DT><SPAN  CLASS="textit">ahs01</SPAN></DT>
<DD>(from Ahsen et al. 2001)
    
</DD>
<DT><SPAN  CLASS="textit">che93</SPAN></DT>
<DD>(from Marmur, Chester and al. 1962, 1993)
    
</DD>
<DT><SPAN  CLASS="textit">che93corr</SPAN></DT>
<DD>(from Ahsen et al. 2001 and from Marmur, Chester and al. 1962, 1993)
    
</DD>
<DT><SPAN  CLASS="textit">schdot</SPAN></DT>
<DD>(Marmur-Schildkraut-Doty formula)
    
</DD>
<DT><SPAN  CLASS="textit">owe69</SPAN></DT>
<DD>(from Owen et al. 1969)
    
</DD>
<DT><SPAN  CLASS="textit">san98</SPAN></DT>
<DD>(from Santalucia et al. 1998)
    
</DD>
<DT><SPAN  CLASS="textit">wetdna91</SPAN></DT>
<DD>(from Wetmur 1991)  (by default)
    
</DD>
</DL>
  RNA <SMALL>DUPLEXES</SMALL>
    <DL COMPACT>
<DT><SPAN  CLASS="textit">wetrna91</SPAN></DT>
<DD>(from Wetmur 1991)  (by default)
    
</DD>
</DL>
  DNA/RNA <SMALL>DUPLEXES</SMALL>
    <DL COMPACT>
<DT><SPAN  CLASS="textit">wetdnarna91</SPAN></DT>
<DD>(from Wetmur 1991)  (by default)
    
</DD>
</DL>
  If there is no formula name after the option <SPAN  CLASS="textbf">-am</SPAN>, we will compute the melting temperature with the default approximative formula.
  This option has to be used with caution. Note that such a calcul is increasingly incorrect when the length of  the duplex 
  decreases. Moreover, it does not take into account nucleic acid concentration, which is a strong mistake.
  examples :

<P>
<PRE>
   command line 1 : "-am"
</PRE>	  
  if you want to force the approximative approach with the 
  default formula.

<P>
<PRE>
   command line 2 : "-am ahs01"
</PRE>

<P>
if you want to use the approximative formula from 
  Ahsen et al. 2001.

<P>
</DD>
<DT><STRONG><SPAN  CLASS="textbf">-nn</SPAN> <SPAN  CLASS="textit">method_name</SPAN></STRONG></DT>
<DD>
<BR>  Forces to use a specific nearest neighbor model. You can use one of the following :

<P>
DNA <SMALL>DUPLEXES</SMALL>
    <DL COMPACT>
<DT><SPAN  CLASS="textit">all97</SPAN></DT>
<DD>(from Allawi and Santalucia 1997)
    
</DD>
<DT><SPAN  CLASS="textit">bre86</SPAN></DT>
<DD>(from Breslauer et al. 1986)
    
</DD>
<DT><SPAN  CLASS="textit">san04</SPAN></DT>
<DD>(from Santalucia 2004)  (by default)
    
</DD>
<DT><SPAN  CLASS="textit">san96</SPAN></DT>
<DD>(from Santalucia et al. 1996)
    
</DD>
<DT><SPAN  CLASS="textit">sug96</SPAN></DT>
<DD>(from Sugimoto et al 1996)
    
</DD>
<DT><SPAN  CLASS="textit">tan04</SPAN></DT>
<DD>(from Tanaka et al. 2004)		 
    
</DD>
</DL>
  RNA <SMALL>DUPLEXES</SMALL>
    <DL COMPACT>
<DT><SPAN  CLASS="textit">fre86</SPAN></DT>
<DD>(from Freier al. 1986)
    
</DD>
<DT><SPAN  CLASS="textit">xia98</SPAN></DT>
<DD>(from Xia et al. 1998)  (by default)		 
    
</DD>
</DL>
  DNA/RNA <SMALL>DUPLEXES</SMALL>
    <DL COMPACT>
<DT><SPAN  CLASS="textit">sug95</SPAN></DT>
<DD>(from Sugimoto et al. 1995)  (by default)
    
</DD>
</DL>
  <SMALL>M</SMALL>RNA/RNA <SMALL>DUPLEXES</SMALL>
    <DL COMPACT>
<DT><SPAN  CLASS="textit">tur06</SPAN></DT>
<DD>(from Turner et al. 2006)  (by default)
    
</DD>
</DL>
  If there is no formula name after the option <SPAN  CLASS="textbf">-nn</SPAN>, we will compute the melting temperature with the default nearest neighbor model. 
  Each nearest neighbor model uses a specific xml file containing the thermodynamic values. If you want to use another file, write the file name or the file pathway preceded by ':' (-nn [optionalname:optionalfile]).
  examples:

<P>
<PRE>
  Command line 1 : "-nn"
</PRE>
  if you want to force the nearest neighbor computation with the default model.

<P>
<PRE>
  Command line 2 : "-nn tan04"
</PRE>
  if you want to use the nearest neighbor model from Tanaka et al. 2004 with the 
  thermodynamic parameters in the default xml file.

<P>
<PRE>
  Command line 3 : "-nn tan04:fileName"
</PRE>
  if you want to use the nearest neighbor model from Tanaka et al. 2004 with the 
  thermodynamic parameters in the file fileName.

<P>
<PRE>
  Command line 4 : "-nn :fileName"
</PRE>
  if you want to use the default nearest neighbor model with the thermodynamic parameters in the file fileName.

<P>
</DD>
<DT><STRONG><SPAN  CLASS="textbf">-sinMM</SPAN> <SPAN  CLASS="textit">method_name</SPAN></STRONG></DT>
<DD>
<BR>  Forces to use a specific nearest neighbor model to compute the contribution of single mismatch to the thermodynamic of helix-coil transition. 
  You can use one of the following :

<P>
DNA <SMALL>DUPLEXES</SMALL>
    <DL COMPACT>
<DT><SPAN  CLASS="textit">allsanpey</SPAN></DT>
<DD>(from Allawi, Santalucia and Peyret 1997, 1998 and 1999)  (by default) 
    
</DD>
</DL>
  RNA <SMALL>DUPLEXES</SMALL>
    <DL COMPACT>
<DT><SPAN  CLASS="textit">tur06</SPAN></DT>
<DD>(from Turner et al. 2006)
    
</DD>
<DT><SPAN  CLASS="textit">zno07</SPAN></DT>
<DD>(from Znosko et al. 2007)  (by default)
    
</DD>
<DT><SPAN  CLASS="textit">zno08</SPAN></DT>
<DD>(from Znosko et al. 2008)		 		 
    
</DD>
</DL>
  To change the file containing the thermodynamic parameters for single mismatch computation, the same syntax as the one for the <SPAN  CLASS="textbf">-nn</SPAN> option is used.
  Single mismatches are not taken into account by the approximative mode.
</DD>
<DT><STRONG><SPAN  CLASS="textbf">-GU</SPAN> <SPAN  CLASS="textit">method_name</SPAN></STRONG></DT>
<DD>
<BR>  Forces to use a specific nearest neighbor model to compute the contribution of GU base pairs to the thermodynamic of helix-coil transition. 
  You can use one of the following :

<P>
RNA <SMALL>DUPLEXES</SMALL>
    <DL COMPACT>
<DT><SPAN  CLASS="textit">tur99</SPAN></DT>
<DD>(from Turner et al. 1999) (by default)		 		 
    
</DD>
</DL>
  To change the file containing the thermodynamic parameters for GU base pair computation, the same syntax as the one for the <SPAN  CLASS="textbf">-nn</SPAN> option is used.
  GU base pairs are not taken into account by the approximative mode.
</DD>
<DT><STRONG><SPAN  CLASS="textbf">-tanMM</SPAN> <SPAN  CLASS="textit">method_name</SPAN></STRONG></DT>
<DD>
<BR>  Forces to use a specific nearest neighbor model to compute the contribution of tandem mismatches to the thermodynamic of helix-coil transition. 
  You can use one of the following :

<P>
DNA <SMALL>DUPLEXES</SMALL>
    <DL COMPACT>
<DT><SPAN  CLASS="textit">allsanpey</SPAN></DT>
<DD>(from Allawi, Santalucia and Peyret 1997, 1998 and 1999)  (by default) 
    
</DD>
</DL>
  RNA <SMALL>DUPLEXES</SMALL>
    <DL COMPACT>
<DT><SPAN  CLASS="textit">tur99</SPAN></DT>
<DD>(from Turner et al. 1999) (by default)		 		 
    
</DD>
</DL>
  To change the file containing the thermodynamic parameters for tandem mismatch computation, the same syntax as the one for the <SPAN  CLASS="textbf">-nn</SPAN> option is used.
  Tandem mismatches are not taken into account by the approximative mode. Note that not all the mismatched Crick's pairs have been investigated. 
</DD>
<DT><STRONG><SPAN  CLASS="textbf">-intLP</SPAN> <SPAN  CLASS="textit">method_name</SPAN></STRONG></DT>
<DD>
<BR>  Forces to use a specific nearest neighbor model to compute the contribution of internal loop to the thermodynamic of helix-coil transition. 
  You can use one of the following :

<P>
DNA <SMALL>DUPLEXES</SMALL>
    <DL COMPACT>
<DT><SPAN  CLASS="textit">san04</SPAN></DT>
<DD>(from Santalucia 2004)  (by default) 
    
</DD>
</DL>
  RNA <SMALL>DUPLEXES</SMALL>
    <DL COMPACT>
<DT><SPAN  CLASS="textit">tur06</SPAN></DT>
<DD>(from Turner et al. 2006) (by default)
    
</DD>
<DT><SPAN  CLASS="textit">zno07</SPAN></DT>
<DD>(from Znosko et al. 2007, only for 1x2 loop)  
    
</DD>
</DL>
  To change the file containing the thermodynamic parameters for internal loop computation, the same syntax as the one for the <SPAN  CLASS="textbf">-nn</SPAN> option is used.
  Internal loops are not taken into account by the approximative mode.   
</DD>
<DT><STRONG><SPAN  CLASS="textbf">-sinDE</SPAN> <SPAN  CLASS="textit">method_name</SPAN></STRONG></DT>
<DD>
<BR>  Forces to use a specific nearest neighbor model to compute the contribution of single dangling end to the thermodynamic of helix-coil transition. 
  You can use one of the following :

<P>
DNA <SMALL>DUPLEXES</SMALL>
    <DL COMPACT>
<DT><SPAN  CLASS="textit">bom00</SPAN></DT>
<DD>(from Bommarito et al. 2000)  (by default) 
    
</DD>
<DT><SPAN  CLASS="textit">sugdna02</SPAN></DT>
<DD>(from Sugimoto et al. 2002, only for polyA dangling ends)      
    
</DD>
</DL>
  RNA <SMALL>DUPLEXES</SMALL>
    <DL COMPACT>
<DT><SPAN  CLASS="textit">sugrna02</SPAN></DT>
<DD>(from Sugimoto et al. 2002, only for polyA dangling ends)
    
</DD>
<DT><SPAN  CLASS="textit">ser08</SPAN></DT>
<DD>(from Serra et al. 2008)  (by default) 		  
    
</DD>
</DL>
  To change the file containing the thermodynamic parameters for single dangling end computation, the same syntax as the one for the <SPAN  CLASS="textbf">-nn</SPAN> option is used.
  Single dangling ends are not taken into account by the approximative mode.   
</DD>
<DT><STRONG><SPAN  CLASS="textbf">-secDE</SPAN> <SPAN  CLASS="textit">method_name</SPAN></STRONG></DT>
<DD>
<BR>  Forces to use a specific nearest neighbor model to compute the contribution of double dangling end to the thermodynamic of helix-coil transition. 
  You can use one of the following :

<P>
DNA <SMALL>DUPLEXES</SMALL>
    <DL COMPACT>
<DT><SPAN  CLASS="textit">sugdna02</SPAN></DT>
<DD>(from Sugimoto et al. 2002, only for polyA dangling ends) (by default)     
    
</DD>
</DL>
  RNA <SMALL>DUPLEXES</SMALL>
    <DL COMPACT>
<DT><SPAN  CLASS="textit">sugrna02</SPAN></DT>
<DD>(from Sugimoto et al. 2002, only for polyA dangling ends)
    
</DD>
<DT><SPAN  CLASS="textit">ser05</SPAN></DT>
<DD>(from Serra et al. 2005) 	
    
</DD>
<DT><SPAN  CLASS="textit">ser06</SPAN></DT>
<DD>(from Serra et al. 2006) (by default) 			 
    
</DD>
</DL>
  To change the file containing the thermodynamic parameters for double dangling end computation, the same syntax as the one for the <SPAN  CLASS="textbf">-nn</SPAN> option is used.
  Double dangling ends are not taken into account by the approximative mode.  
</DD>
<DT><STRONG><SPAN  CLASS="textbf">-longDE</SPAN> <SPAN  CLASS="textit">method_name</SPAN></STRONG></DT>
<DD>
<BR>  Forces to use a specific nearest neighbor model to compute the contribution of long dangling end to the thermodynamic of helix-coil transition. 
  You can use one of the following :

<P>
DNA <SMALL>DUPLEXES</SMALL>
    <DL COMPACT>
<DT><SPAN  CLASS="textit">sugdna02</SPAN></DT>
<DD>(from Sugimoto et al. 2002, only for polyA dangling ends) (by default)     
    
</DD>
</DL>
  RNA <SMALL>DUPLEXES</SMALL>
    <DL COMPACT>
<DT><SPAN  CLASS="textit">sugrna02</SPAN></DT>
<DD>(from Sugimoto et al. 2002, only for polyA dangling ends)
    
</DD>
</DL>
  To change the file containing the thermodynamic parameters for long dangling end computation, the same syntax as the one for the <SPAN  CLASS="textbf">-nn</SPAN> option is used.
  Long dangling ends are not taken into account by the approximative mode.  
</DD>
<DT><STRONG><SPAN  CLASS="textbf">-sinBU</SPAN> <SPAN  CLASS="textit">method_name</SPAN></STRONG></DT>
<DD>
<BR>  Forces to use a specific nearest neighbor model to compute the contribution of single bulge loop to the thermodynamic of helix-coil transition. 
  You can use one of the following :

<P>
DNA <SMALL>DUPLEXES</SMALL>
    <DL COMPACT>
<DT><SPAN  CLASS="textit">san04</SPAN></DT>
<DD>(from Santalucia 2004) 
    
</DD>
<DT><SPAN  CLASS="textit">tan04</SPAN></DT>
<DD>(from Tanaka et al. 2004)  (by default) 	    
    
</DD>
</DL>
  RNA <SMALL>DUPLEXES</SMALL>
    <DL COMPACT>
<DT><SPAN  CLASS="textit">ser07</SPAN></DT>
<DD>(from Serra et al. 2007)
    
</DD>
<DT><SPAN  CLASS="textit">tur06</SPAN></DT>
<DD>(from Turner et al. 1999 and 2006)  (by default) 			 			 
    
</DD>
</DL>
  To change the file containing the thermodynamic parameters for single bulge loop computation, the same syntax as the one for the <SPAN  CLASS="textbf">-nn</SPAN> option is used.
  Single bulge loops are not taken into account by the approximative mode. 
</DD>
<DT><STRONG><SPAN  CLASS="textbf">-lonBU</SPAN> <SPAN  CLASS="textit">method_name</SPAN></STRONG></DT>
<DD>
<BR>  Forces to use a specific nearest neighbor model to compute the contribution of long bulge loop to the thermodynamic of helix-coil transition. 
  You can use one of the following :

<P>
DNA <SMALL>DUPLEXES</SMALL>
    <DL COMPACT>
<DT><SPAN  CLASS="textit">san04</SPAN></DT>
<DD>(from Santalucia 2004) (by default)
    
</DD>
</DL>
  RNA <SMALL>DUPLEXES</SMALL>
    <DL COMPACT>
<DT><SPAN  CLASS="textit">tur06</SPAN></DT>
<DD>(from Turner et al. 1999 and 2006)  (by default) 			 			 
    
</DD>
</DL>
  To change the file containing the thermodynamic parameters for long bulge loop computation, the same syntax as the one for the <SPAN  CLASS="textbf">-nn</SPAN> option is used.
  Long bulge loops are not taken into account by the approximative mode.
</DD>
<DT><STRONG><SPAN  CLASS="textbf">-CNG</SPAN> <SPAN  CLASS="textit">method_name</SPAN></STRONG></DT>
<DD>
<BR>  Forces to use a specific nearest neighbor model to compute the contribution of CNG repeats to the thermodynamic of helix-coil transition.
  N represents a single mismatch of type N/N. 
  You can use one of the following :

<P>
RNA <SMALL>DUPLEXES</SMALL>
    <DL COMPACT>
<DT><SPAN  CLASS="textit">bro05</SPAN></DT>
<DD>(from Broda et al. 2005) (by default)			 			 
    
</DD>
</DL>
  To change the file containing the thermodynamic parameters for CNG repeats computation, the same syntax as the one for the <SPAN  CLASS="textbf">-nn</SPAN> option is used.
  CNG repeats are not taken into account by the approximative mode. 
  Be aware : Melting can compute the contribution of CNG repeats to the thermodynamic of helix-coil transition for only 2 to 7 CNG repeats.
</DD>
<DT><STRONG><SPAN  CLASS="textbf">-ino</SPAN> <SPAN  CLASS="textit">method_name</SPAN></STRONG></DT>
<DD>
<BR>  Forces to use a specific nearest neighbor model to compute the contribution of inosine bases (I) to the thermodynamic of helix-coil transition. 
  You can use one of the following :

<P>
DNA <SMALL>DUPLEXES</SMALL>
    <DL COMPACT>
<DT><SPAN  CLASS="textit">san05</SPAN></DT>
<DD>(from Santalucia et al. 2005)  (by default)
    
</DD>
</DL>
  RNA <SMALL>DUPLEXES</SMALL>
    <DL COMPACT>
<DT><SPAN  CLASS="textit">zno07</SPAN></DT>
<DD>(from Znosco et al. 2007, only IU base pairs)  (by default)  			 			 
    
</DD>
</DL>
  To change the file containing the thermodynamic parameters for inosine bases computation, the same syntax as the one for the <SPAN  CLASS="textbf">-nn</SPAN> option is used.
  Inosine bases (I) are not taken into account by the approximative mode. 
</DD>
<DT><STRONG><SPAN  CLASS="textbf">-ha</SPAN> <SPAN  CLASS="textit">method_name</SPAN></STRONG></DT>
<DD>
<BR>  Forces to use a specific nearest neighbor model to compute the contribution of hydroxyadenine bases (A*) to the thermodynamic of helix-coil transition. 
  You can use one of the following :

<P>
DNA <SMALL>DUPLEXES</SMALL>
    <DL COMPACT>
<DT><SPAN  CLASS="textit">sug01</SPAN></DT>
<DD>(from Sugimoto et al. 2001) (by default)
    
</DD>
</DL>
  To change the file containing the thermodynamic parameters for hydroxyadenine bases computation, the same syntax as the one for the <SPAN  CLASS="textbf">-nn</SPAN> option is used.
  Hydroxyadenine bases (A*) are not taken into account by the approximative mode.
</DD>
<DT><STRONG><SPAN  CLASS="textbf">-azo</SPAN> <SPAN  CLASS="textit">method_name</SPAN></STRONG></DT>
<DD>
<BR>  Forces to use a specific nearest neighbor model to compute the contribution of azobenzenes (X_T for trans azobenzenes and X_C for cis azobenzenes) to the thermodynamic 
  of helix-coil transition. 
  You can use one of the following :

<P>
DNA <SMALL>DUPLEXES</SMALL>
    <DL COMPACT>
<DT><SPAN  CLASS="textit">asa05</SPAN></DT>
<DD>(from Asanuma et al. 2005)(by default)
    
</DD>
</DL>
  To change the file containing the thermodynamic parameters for azobenzene computation, the same syntax as the one for the <SPAN  CLASS="textbf">-nn</SPAN> option is used.
  Azobenzenes (X_T for trans azobenzenes and X_C for cis azobenzenes) are not taken into account by the approximative mode.
</DD>
<DT><STRONG><SPAN  CLASS="textbf">-lck</SPAN> <SPAN  CLASS="textit">method_name</SPAN></STRONG></DT>
<DD>
<BR>  Forces to use a specific nearest neighbor model to compute the contribution of locked nucleic acids (AL, GL, TL and CL) to the thermodynamic 
  of helix-coil transition. 
  You can use one of the following :

<P>
DNA <SMALL>DUPLEXES</SMALL>
    <DL COMPACT>
<DT><SPAN  CLASS="textit">mct04</SPAN></DT>
<DD>(from McTigue et al. 2004) (by default)
    
</DD>
</DL>
  To change the file containing the thermodynamic parameters for locked nucleic acids computation, the same syntax as the one for the <SPAN  CLASS="textbf">-nn</SPAN> option is used.
  Locked nucleic acids (AL, GL, TL and CL) are not taken into account by the approximative mode.
</DD>
<DT><STRONG><SPAN  CLASS="textbf">-ion</SPAN> <SPAN  CLASS="textit">method_name</SPAN></STRONG></DT>
<DD>
<BR>  Forces to use a specific ion correction. You can use one of the following corrections : 

<P>
<SPAN  CLASS="textbf">Sodium corrections</SPAN>

<P>
DNA <SMALL>DUPLEXES</SMALL>
      <DL COMPACT>
<DT><SPAN  CLASS="textit">ahs01</SPAN></DT>
<DD>(from Ahsen et al. 2001)
      
</DD>
<DT><SPAN  CLASS="textit">kam71</SPAN></DT>
<DD>(from Frank Kamenetskii et al 2001)
      
</DD>
<DT><SPAN  CLASS="textit">owc1904</SPAN></DT>
<DD>(equation 19 from Owczarzy et al. 2004)
      
</DD>
<DT><SPAN  CLASS="textit">owc2004</SPAN></DT>
<DD>(equation 20 from Owczarzy et al. 2004)
      
</DD>
<DT><SPAN  CLASS="textit">owc2104</SPAN></DT>
<DD>(equation 21 from Owczarzy et al. 2004)
      
</DD>
<DT><SPAN  CLASS="textit">owc2204</SPAN></DT>
<DD>(equation 21 from Owczarzy et al. 2004)  (by default)
      
</DD>
<DT><SPAN  CLASS="textit">san96</SPAN></DT>
<DD>(from Santalucia et al. 1996)
      
</DD>
<DT><SPAN  CLASS="textit">san04</SPAN></DT>
<DD>(from Santalucia et al. 1998, 2004)
      
</DD>
<DT><SPAN  CLASS="textit">schlif</SPAN></DT>
<DD>(from Schildkraut and Lifson 1965)
      
</DD>
<DT><SPAN  CLASS="textit">tanna06</SPAN></DT>
<DD>(from Zhi-Jie Tan et al. 2006)
      
</DD>
<DT><SPAN  CLASS="textit">wetdna91</SPAN></DT>
<DD>(from wetmur 1991)	 
      
</DD>
</DL>
    RNA <SMALL>DUPLEXES OR M</SMALL>RNA/RNA <SMALL>DUPLEXES</SMALL>
      <DL COMPACT>
<DT><SPAN  CLASS="textit">tanna07</SPAN></DT>
<DD>(from Zhi-Jie Tan et al. 2007)  (by default)
      
</DD>
<DT><SPAN  CLASS="textit">wetrna91</SPAN></DT>
<DD>(from wetmur 1991)	 
      
</DD>
</DL>
    DNA/RNA <SMALL>DUPLEXES</SMALL>
      <DL COMPACT>
<DT><SPAN  CLASS="textit">wetdnarna91</SPAN></DT>
<DD>(from wetmur 1991)	 
      
</DD>
</DL>

<P>
<SPAN  CLASS="textbf">Magnesium corrections</SPAN>

<P>
DNA <SMALL>DUPLEXES</SMALL>
      <DL COMPACT>
<DT><SPAN  CLASS="textit">owcmg08</SPAN></DT>
<DD>(from Owczarzy et al. 2008)  (by default)
      
</DD>
<DT><SPAN  CLASS="textit">tanmg06</SPAN></DT>
<DD>(from Zhi-Jie Tan et al. 2006)	  
      
</DD>
</DL>
    RNA <SMALL>DUPLEXES OR M</SMALL>RNA/RNA <SMALL>DUPLEXES</SMALL>
      <DL COMPACT>
<DT><SPAN  CLASS="textit">tanmg07</SPAN></DT>
<DD>(from Zhi-Jie Tan et al. 2007)  (by default)
      
</DD>
</DL>

<P>
<SPAN  CLASS="textbf">Mixed Na Mg corrections</SPAN>

<P>
DNA <SMALL>DUPLEXES</SMALL>
      <DL COMPACT>
<DT><SPAN  CLASS="textit">owcmix08</SPAN></DT>
<DD>(from Owczarzy et al. 2008)  (by default)
      
</DD>
<DT><SPAN  CLASS="textit">tanmix07</SPAN></DT>
<DD>(from Zhi-Jie Tan et al. 2007)	 	  
      
</DD>
</DL>
    RNA <SMALL>DUPLEXES OR M</SMALL>RNA/RNA <SMALL>DUPLEXES</SMALL>
      <DL COMPACT>
<DT><SPAN  CLASS="textit">tanmix07</SPAN></DT>
<DD>(from Zhi-Jie Tan et al. 2007)  (by default)
      
</DD>
</DL>
  The effect of ions on  thermodynamic  stability  of nucleic  acid duplexes is complex, and the correcting 
  functions are  at  best rough  approximations.
  By default, the program use the algorithm from Owczarzy et al 2008 : ratio =
  <!-- MATH
 $[\mbox{Mg}^{0.5}]$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">Mg<IMG
 WIDTH="26" HEIGHT="34" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img9.png"
 ALT="$ ^{0.5}]$"></SPAN> and monovalent = Na + Tris + K

<P>
if monovalent = 0, a magnesium correction is used.

<P>
if ratio &lt; 0.22, a sodium correction is used.

<P>
if 0.22 &lt;= ratio &lt; 6, a mixed Na Mg correction is used.

<P>
if ratio &gt;= 6, a magnesium correction is used.

<P>
example :

<P>
<PRE>
  Command line : "-ion owcmg08"
</PRE>
  if you want to force the use of the magnesium correction from Owczarzy et al 2008. This correction will be used 
  independently of the cations present in the solution.
</DD>
<DT><STRONG><SPAN  CLASS="textbf">-naeq</SPAN> <SPAN  CLASS="textit">method_name</SPAN></STRONG></DT>
<DD>
<BR>  Forces to use a specific ion correction which gives a sodium equivalent concentration if other cations are present.
  You can use one of the following :

<P>
DNA <SMALL>DUPLEXES</SMALL>
    <DL COMPACT>
<DT><SPAN  CLASS="textit">ahs01</SPAN></DT>
<DD>(from Ahsen et al 2001)  (by default)
    
</DD>
<DT><SPAN  CLASS="textit">mit96</SPAN></DT>
<DD>(from Mitsuhashi et al. 1996)
    
</DD>
<DT><SPAN  CLASS="textit">pey00</SPAN></DT>
<DD>(from Peyret 2000)		 
    
</DD>
</DL>
  For the other types of hybridization, the DNA default correction is used but there is no guaranty of accuracy.
  If there are other cations when an approximative approach is used, a sodium equivalence is automatically computed.
  The correcting functions are  at  best rough  approximations.
  example :

<P>
<PRE>
  Command line 1 : "-naeq ahs01"
</PRE>
  if you want to force the use of the sodium equivalence from Ahsen et al 2001. This sodium equivalence 
  will be used in case of approximative approach. In case of nearest neighbor approach, the sodium equivalence 
  will be used only if a sodium correction is selected by the user.

<P>
<PRE>
  Command line 2 : "-naeq ahs01 -ion san04"
</PRE>
  it means that the sodium equivalence computed by the method ahs01 (from Ahsen et al 2001) will be combined with the 
  sodium correction san04 (from Santalucia 2004).

<P>
</DD>
<DT><STRONG><SPAN  CLASS="textbf">-DMSO</SPAN> <SPAN  CLASS="textit">method_name</SPAN></STRONG></DT>
<DD>
<BR>  Forces to use a specific DMSO correction (DMSO is always in %).
  You can use one of the following :

<P>
DNA <SMALL>DUPLEXES</SMALL>
    <DL COMPACT>
<DT><SPAN  CLASS="textit">ahs01</SPAN></DT>
<DD>(from Ahsen et al 2001)  (by default)
    
</DD>
<DT><SPAN  CLASS="textit">mus81</SPAN></DT>
<DD>(from Musielski et al. 1981)
    
</DD>
<DT><SPAN  CLASS="textit">cul76</SPAN></DT>
<DD>(from Cullen et al. 1976)	
    
</DD>
<DT><SPAN  CLASS="textit">esc80</SPAN></DT>
<DD>(from Escara et al. 1980)		 	 
    
</DD>
</DL>
  For the other types of hybridization, the DNA default correction is used but there is no guaranty of accuracy.
  If there are DMSO when an approximative approach is used, a DMSO correction is automatically computed.
  The correcting functions are  at  best rough  approximations.
  example :

<P>
<PRE>
  Command line : "-DMSO ahs01"
</PRE> 
  if you want to force the use of the DMSO correction from Ahsen et al 2001. This DMSO correction will be used if there is 
  DMSO present in the solutions in case of nearest neighbor approach and approximative approach.
</DD>
<DT><STRONG><SPAN  CLASS="textbf">-for</SPAN> <SPAN  CLASS="textit">method_name</SPAN></STRONG></DT>
<DD>
<BR>  Forces to use a specific formamide correction.
  You can use one of the following :

<P>
DNA <SMALL>DUPLEXES</SMALL>
    <DL COMPACT>
<DT><SPAN  CLASS="textit">bla96</SPAN></DT>
<DD>(from Blake et al 1996) with formamide concentration in mol/L  (by default)
    
</DD>
<DT><SPAN  CLASS="textit">lincorr</SPAN></DT>
<DD>(linear correction) with a % of formamide volume		  	 
    
</DD>
</DL>
  For the other types of hybridization, the DNA default correction is used but there is no guaranty of accuracy.
  If there are formamide when an approximative approach is used, a formamide correction is automatically computed.
  The correcting functions are  at  best rough  approximations.
  example :

<P>
<PRE>
  Command line : "-for lincorr"
</PRE>
  if you want to force the use of the linear formamide correction. This formamide correction will be used if there is formamide 
  present in the solutions in case of nearest neighbor approach and approximative approach. 
</DD>
</DL>

<P>

<H1><A NAME="SECTION00050000000000000000">
Algorithm </A>
</H1>

<P>

<H2><A NAME="SECTION00051000000000000000">
Thermodynamics of helix-coil transition of nucleic acid</A>
</H2>  

<P>
The nearest-neighbor approach is based on the fact that the helix-coil
transition works as a zipper. After an initial attachment, the hybridisation
propagates laterally. This program first computes the hybridisation enthalpy 
and entropy for each structure in the duplex. (see later for the different possible
structures recognized by Melting). If the sequences are self complementary, a 
symmetry correction will be added to the initiation energy.
<P><!-- MATH
 \begin{displaymath}
\begin{array}[t]{ccc}
  \Delta{}H&=&\delta{}h_\mathrm{initiation}+\sum \delta{}h_\mathrm{structure}\\
  \Delta{}S&=&\delta{}s_\mathrm{initiation}+\sum \delta{}s_\mathrm{structure}
  \end{array}
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="268" HEIGHT="58" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img10.png"
 ALT="$\displaystyle \begin{array}[t]{ccc}
\Delta{}H&amp;=&amp;\delta{}h_\mathrm{initiation}+\...
...S&amp;=&amp;\delta{}s_\mathrm{initiation}+\sum \delta{}s_\mathrm{structure}
\end{array}$">
</DIV><P></P> 

<P>
<SPAN  CLASS="textbf">Example</SPAN> : 

<P>
<SPAN  CLASS="textit">Sequence with a single mismatch</SPAN> 

<DIV CLASS="alltt" ALIGN="LEFT">
<TT>
 ATC<SPAN &nbsp;CLASS="textbf">G</SPAN>GCTA
<BR>TAG<SPAN &nbsp;CLASS="textbf">A</SPAN>CGAT
<BR></TT>
</DIV>

<P><!-- MATH
 \begin{displaymath}
\begin{array}[t]{ccc}
  \Delta{}H&=&\delta{}h_\mathrm{initiation}+\delta{}h_\mathrm{structure1}+\delta{}h_\mathrm{structure2}+\delta{}h_\mathrm{structure3}\\
  \Delta{}S&=&\delta{}s_\mathrm{initiation}+\delta{}s_\mathrm{structure1}+\delta{}s_\mathrm{structure2}+\delta{}s_\mathrm{structure3}
  \end{array}
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="442" HEIGHT="58" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img11.png"
 ALT="$\displaystyle \begin{array}[t]{ccc}
\Delta{}H&amp;=&amp;\delta{}h_\mathrm{initiation}+\...
...cture1}+\delta{}s_\mathrm{structure2}+\delta{}s_\mathrm{structure3}
\end{array}$">
</DIV><P></P>

<P>
where :

<P>
structure1 = perfectly matching sequences ATC/TAG

<P>
structure2 = single mismatch G/A

<P>
structure 3 = perfectly matching sequences GCTA/CGAT

<P>

<H3><A NAME="SECTION00051100000000000000">
Perfectly matching sequences</A>
</H3>

<P>
The hybridization process depends on the adjacent nucleotides on each strand (the Crick's pairs).  
Two duplexes with the same base pairs could have different stabilities, and on the contrary, two 
duplexes with different sequences but identical sets of Crick's pairs will have the same
thermodynamics properties.  This program first computes the hybridisation enthalpy 
and entropy from the elementary parameters of each Crick's pair.
<P><!-- MATH
 \begin{displaymath}
\begin{array}[t]{ccc}
  \Delta{}h_\mathrm{perfectly-matching}&=&\sum \delta{}h_\mathrm{Crick's pair}\\
  \Delta{}s_\mathrm{perfectly-matching}&=&\sum \delta{}s_\mathrm{Crick's pair}
  \end{array}
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="296" HEIGHT="58" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img12.png"
 ALT="$\displaystyle \begin{array}[t]{ccc}
\Delta{}h_\mathrm{perfectly-matching}&amp;=&amp;\su...
..._\mathrm{perfectly-matching}&amp;=&amp;\sum \delta{}s_\mathrm{Crick's pair}
\end{array}$">
</DIV><P></P>
The initiation computation is not the same for each following model. 

<P>
<BR><P></P>
<DIV ALIGN="CENTER">
<TABLE CELLPADDING=3 BORDER="1">
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">model</SPAN></TH>
<TH ALIGN="CENTER"><SPAN  CLASS="textbf">limits</SPAN></TH>
<TH ALIGN="CENTER"><SPAN  CLASS="textbf">Article</SPAN></TH>
</TR>
<TR><TD ALIGN="CENTER">all97</TD>
<TD ALIGN="CENTER">DNA</TD>
<TD ALIGN="CENTER">Allawi and SantaLucia (1997),Biochemistry 36 : 10581-10594</TD>
</TR>
<TR><TD ALIGN="CENTER">bre86</TD>
<TD ALIGN="CENTER">DNA</TD>
<TD ALIGN="CENTER">Breslauer et al. (1986), Proc Natl Acad Sci USA 83 : 3746-3750</TD>
</TR>
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">san04</SPAN></TH>
<TD ALIGN="CENTER">DNA</TD>
<TD ALIGN="CENTER">Santalucia et al (2004, Annu. Rev. Biophys. Biomol. Struct 33 : 415-440</TD>
</TR>
<TR><TD ALIGN="CENTER">san96</TD>
<TD ALIGN="CENTER">DNA</TD>
<TD ALIGN="CENTER">SantaLucia et al.(1996), Biochemistry 35 : 3555-3562</TD>
</TR>
<TR><TD ALIGN="CENTER">sug96</TD>
<TD ALIGN="CENTER">DNA</TD>
<TD ALIGN="CENTER">Sugimoto et al. (1996), Nuc Acids Res 24 : 4501-4505</TD>
</TR>
<TR><TD ALIGN="CENTER">tan04</TD>
<TD ALIGN="CENTER">DNA</TD>
<TD ALIGN="CENTER">Tanaka Fumiaki et al (2004), Biochemistry 43 : 7143-7150</TD>
</TR>
<TR><TD ALIGN="CENTER">fre86</TD>
<TD ALIGN="CENTER">RNA</TD>
<TD ALIGN="CENTER">Freier et al (1986), Proc Natl Acad Sci USA 83: 9373-9377</TD>
</TR>
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">xia98</SPAN></TH>
<TD ALIGN="CENTER">RNA</TD>
<TD ALIGN="CENTER">Xia et al (1998), Biochemistry 37: 14719-14735</TD>
</TR>
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">sug95</SPAN></TH>
<TD ALIGN="CENTER">DNA/RNA</TD>
<TD ALIGN="CENTER">SantaLucia et al.(1996), Biochemistry 35 : 3555-3562</TD>
</TR>
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">tur06</SPAN></TH>
<TD ALIGN="CENTER">DNA </br> A sodium correction (san04) is automatically applied to convert the entropy (Na = 0.1M) into the entropy (Na = 1M)</TD>
<TD ALIGN="CENTER">Turner et al (2006), Nucleic acids research 34: 3609-3614</TD>
</TR>
</TABLE>
</DIV><BR>

<P>
<SPAN  CLASS="textbf">Example</SPAN> :
<P></P><DIV ALIGN="CENTER"><!-- MATH
 \begin{multline*}
\Delta H {\mbox{\texttt{AGCGA}} \choose \mbox{\texttt{TCGCT}}} =
\Delta H {\mbox{\texttt{AG}} \choose \mbox{\texttt{TC}} } + 
\Delta H {\mbox{\texttt{GC}} \choose \mbox{\texttt{CG}} } +
\Delta H {\mbox{\texttt{CG}} \choose \mbox{\texttt{GC}} } +
\Delta H {\mbox{\texttt{GA}} \choose \mbox{\texttt{CT}} }
\end{multline*}
 -->
<IMG
 WIDTH="520" HEIGHT="49" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img13.png"
 ALT="\begin{multline*}
\Delta H {\mbox{\texttt{AGCGA}} \choose \mbox{\texttt{TCGCT}}}...
...}} } +
\Delta H {\mbox{\texttt{GA}} \choose \mbox{\texttt{CT}} }
\end{multline*}"></DIV>
<BR CLEAR="ALL">
<P><P></P>

<P>
(The same computation is performed for <SPAN CLASS="MATH"><IMG
 WIDTH="28" HEIGHT="14" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img14.png"
 ALT="$ \Delta S$"></SPAN>)

<P>

<DIV ALIGN="CENTER"><A NAME="1985"></A>
<TABLE>
<CAPTION ALIGN="BOTTOM"><STRONG>Figure:</STRONG>
Comparison of experimental and computed Tm for various sets of
  DNA nearest-neighbor parameters. <!-- MATH
 $[\mbox{Na}^+] = 1$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">Na<IMG
 WIDTH="49" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img15.png"
 ALT="$ ^+] = 1$"></SPAN>&nbsp;M, <!-- MATH
 $[\mbox{nucleic acid}] = 4\cdot{}10^{-4}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">nucleic acid<IMG
 WIDTH="83" HEIGHT="34" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img16.png"
 ALT="$ ] = 4\cdot{}10^{-4}$"></SPAN>&nbsp;M</CAPTION>
<TR><TD><IMG
  WIDTH="581" HEIGHT="406" ALIGN="BOTTOM" BORDER="0"
 SRC="html/DNAPerfectlyMatching.png"
 ALT="Image DNAPerfectlyMatching"></TD></TR>
</TABLE>
</DIV>

<P>

<DIV ALIGN="CENTER"><A NAME="1986"></A>
<TABLE>
<CAPTION ALIGN="BOTTOM"><STRONG>Figure:</STRONG>
Comparison of experimental and computed Tm for various sets of
  RNA nearest-neighbor parameters. <!-- MATH
 $[\mbox{Na}^+] = 1$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">Na<IMG
 WIDTH="49" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img15.png"
 ALT="$ ^+] = 1$"></SPAN>&nbsp;M, <!-- MATH
 $[\mbox{nucleic acid}] = 2\cdot{}10^{-4}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">nucleic acid<IMG
 WIDTH="83" HEIGHT="34" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img17.png"
 ALT="$ ] = 2\cdot{}10^{-4}$"></SPAN>&nbsp;M</CAPTION>
<TR><TD><IMG
  WIDTH="568" HEIGHT="318" ALIGN="BOTTOM" BORDER="0"
 SRC="html/RNAPerfectlyMatching.png"
 ALT="Image RNAPerfectlyMatching"></TD></TR>
</TABLE>
</DIV>

<P>

<DIV ALIGN="CENTER"><A NAME="1987"></A>
<TABLE>
<CAPTION ALIGN="BOTTOM"><STRONG>Figure:</STRONG>
Comparison of experimental and computed Tm for various sets of
  DNA/RNA nearest-neighbor parameters. <!-- MATH
 $[\mbox{Na}^+] = 1$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">Na<IMG
 WIDTH="49" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img15.png"
 ALT="$ ^+] = 1$"></SPAN>&nbsp;M, <!-- MATH
 $[\mbox{nucleic acid}] = 1\cdot{}10^{-4}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">nucleic acid<IMG
 WIDTH="83" HEIGHT="34" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img18.png"
 ALT="$ ] = 1\cdot{}10^{-4}$"></SPAN>&nbsp;M</CAPTION>
<TR><TD><IMG
  WIDTH="568" HEIGHT="353" ALIGN="BOTTOM" BORDER="0"
 SRC="html/DNARNA.png"
 ALT="Image DNARNA"></TD></TR>
</TABLE>
</DIV>

<P>

<DIV ALIGN="CENTER"><A NAME="1988"></A>
<TABLE>
<CAPTION ALIGN="BOTTOM"><STRONG>Figure:</STRONG>
Comparison of experimental and computed Tm for various sets of
  2-O-methyl RNA nearest-neighbor parameters. <!-- MATH
 $[\mbox{Na}^+] = 1$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">Na<IMG
 WIDTH="49" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img15.png"
 ALT="$ ^+] = 1$"></SPAN>&nbsp;M, <!-- MATH
 $[\mbox{nucleic acid}] = 1\cdot{}10^{-4}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">nucleic acid<IMG
 WIDTH="83" HEIGHT="34" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img18.png"
 ALT="$ ] = 1\cdot{}10^{-4}$"></SPAN>&nbsp;M</CAPTION>
<TR><TD><IMG
  WIDTH="581" HEIGHT="406" ALIGN="BOTTOM" BORDER="0"
 SRC="html/mRNARNA.png"
 ALT="Image mRNARNA"></TD></TR>
</TABLE>
</DIV>

<P>

<H3><A NAME="SECTION00051200000000000000">
Sequences composed of CNG repeats</A>
</H3>

<P>
If the sequence (sens 5'3') is a sequence of type <TT>G(CNG)xC</TT> where x is the number of CNG repeats in
the sequence and N a unique nucleic acid which will get bound to itself, we can use specific
experimental parameters to compute the enthalpy and entropy of the duplex formation. These parameters can be used
only for sequences composed from 2 to 7 CNG repeats and the initiation is already included.

<P>
<P><!-- MATH
 \begin{displaymath}
\begin{array}[t]{ccc}
  \Delta{}H&=&\Delta{}h_\mathrm{sequence-of-type-\texttt{G(CNG)xC}}\\
  \end{array}
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="283" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img19.png"
 ALT="$\displaystyle \begin{array}[t]{ccc}
\Delta{}H&amp;=&amp;\Delta{}h_\mathrm{sequence-of-type-\texttt{G(CNG)xC}}\\
\end{array}$">
</DIV><P></P>
For further information, see the referenced article.

<P>
<BR><P></P>
<DIV ALIGN="CENTER">
<TABLE CELLPADDING=3 BORDER="1">
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">model</SPAN></TH>
<TH ALIGN="CENTER"><SPAN  CLASS="textbf">limits</SPAN></TH>
<TH ALIGN="CENTER"><SPAN  CLASS="textbf">Article</SPAN></TH>
</TR>
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">bro05</SPAN></TH>
<TD ALIGN="CENTER">RNA <br/>Self complementary sequences <br/>2 to 7 CNG repeats</TD>
<TD ALIGN="CENTER">Broda et al (2005), Biochemistry 44: 10873-10882</TD>
</TR>
</TABLE>
</DIV><BR>

<P>
<SPAN  CLASS="textbf">Example</SPAN> :

<DIV CLASS="alltt" ALIGN="LEFT">
<TT>
 G<SPAN &nbsp;CLASS="textbf">CAGCAGCAG</SPAN>C
<BR>C<SPAN &nbsp;CLASS="textbf">GACGACGAC</SPAN>G
<BR></TT>
</DIV>

<P>
<P></P><DIV ALIGN="CENTER"><!-- MATH
 \begin{multline*}
\Delta H {\mbox{\texttt{GCAGCAGCAGC}} \choose \mbox{\texttt{CGACGACGACG}}} =
\Delta H {(\mbox{3-CAG-repeats})}
\end{multline*}
 -->
<IMG
 WIDTH="521" HEIGHT="49" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img20.png"
 ALT="\begin{multline*}
\Delta H {\mbox{\texttt{GCAGCAGCAGC}} \choose \mbox{\texttt{CGACGACGACG}}} =
\Delta H {(\mbox{3-CAG-repeats})}
\end{multline*}"></DIV>
<BR CLEAR="ALL">
<P><P></P>

<P>
(The same computation is performed for <SPAN CLASS="MATH"><IMG
 WIDTH="28" HEIGHT="14" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img14.png"
 ALT="$ \Delta S$"></SPAN>)

<P>

<DIV ALIGN="CENTER"><A NAME="1993"></A>
<TABLE>
<CAPTION ALIGN="BOTTOM"><STRONG>Figure:</STRONG>
Comparison of experimental and computed Tm for various sets of
 RNA sequences composed of CNG repeats. <!-- MATH
 $[\mbox{Na}^+] = 1$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">Na<IMG
 WIDTH="49" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img15.png"
 ALT="$ ^+] = 1$"></SPAN>&nbsp;M, <!-- MATH
 $[\mbox{nucleic acid}] = 1\cdot{}10^{-4}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">nucleic acid<IMG
 WIDTH="83" HEIGHT="34" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img18.png"
 ALT="$ ] = 1\cdot{}10^{-4}$"></SPAN>&nbsp;M</CAPTION>
<TR><TD><IMG
  WIDTH="497" HEIGHT="350" ALIGN="BOTTOM" BORDER="0"
 SRC="html/CNG.png"
 ALT="Image CNG"></TD></TR>
</TABLE>
</DIV>

<P>
Be aware : The results for sequences composed of 4 or 5 CCG repeats is not reliable. (the figure shows two values
far from the expected temperature). This might be due to a majority of hairpin loop formation. See the article
above for further informations.

<P>

<H3><A NAME="SECTION00051300000000000000">
Single mismatch effect</A>
</H3>

<P>
The single mismatches are taken into account but the two first and positions cannot
be mismatched. in such a case, the result is unpredictable, and all cases are
possible. for instance (see Allawi and SanLucia 1997), the duplex

<DIV CLASS="alltt" ALIGN="LEFT">
<TT>
 A &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;T &nbsp;
<BR>&nbsp;<SPAN ID="txt572">G</SPAN>TGAGCTCA<SPAN ID="txt573">T</SPAN> &nbsp;
<BR>&nbsp;<SPAN ID="txt574">T</SPAN>ACTCGAGT<SPAN ID="txt575">G</SPAN> &nbsp;
<BR>T &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;A &nbsp;&nbsp;
<BR></TT>
</DIV>

<P>
is more stable than 

<P>

<DIV CLASS="alltt" ALIGN="LEFT">
<TT>
 A<SPAN ID="txt578">G</SPAN>TGAGCTCA<SPAN ID="txt579">T</SPAN>T 
<BR>T<SPAN ID="txt580">T</SPAN>ACTCGAGT<SPAN ID="txt581">G</SPAN>A 
<BR></TT>
</DIV>

<P>
For DNA duplexes, this program computes the hybridisation enthalpy and entropy from the elementary 
parameters of each Crick's pair containing the single mismatch. 
<P><!-- MATH
 \begin{displaymath}
\begin{array}[t]{ccc}
  \Delta{}h_\mathrm{single-mismatch}&=&\sum \delta{}h_\mathrm{Crick's-pair-containing-the-mismatch}\\
  \end{array}
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="449" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img21.png"
 ALT="$\displaystyle \begin{array}[t]{ccc}
\Delta{}h_\mathrm{single-mismatch}&amp;=&amp;\sum \delta{}h_\mathrm{Crick's-pair-containing-the-mismatch}\\
\end{array}$">
</DIV><P></P>

<P>
<SPAN  CLASS="textbf">Example</SPAN> :
<P></P><DIV ALIGN="CENTER"><!-- MATH
 \begin{multline*}
\Delta H {\mbox{\texttt{ATC}} \choose \mbox{\texttt{TCG}}} =
\Delta H {\mbox{\texttt{AT}} \choose \mbox{\texttt{TC}} } + 
\Delta H {\mbox{\texttt{TC}} \choose \mbox{\texttt{CG}} } \\
\end{multline*}
 -->
<IMG
 WIDTH="257" HEIGHT="75" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img22.png"
 ALT="\begin{multline*}
\Delta H {\mbox{\texttt{ATC}} \choose \mbox{\texttt{TCG}}} =
...
...+
\Delta H {\mbox{\texttt{TC}} \choose \mbox{\texttt{CG}} } \\
\end{multline*}"></DIV>
<BR CLEAR="ALL">
<P><P></P>

<P>
(The same computation is performed for <SPAN CLASS="MATH"><IMG
 WIDTH="28" HEIGHT="14" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img14.png"
 ALT="$ \Delta S$"></SPAN>)

<P>

<DIV ALIGN="CENTER"><A NAME="2000"></A>
<TABLE>
<CAPTION ALIGN="BOTTOM"><STRONG>Figure:</STRONG>
Comparison of experimental and computed Tm for various sets of
 DNA sequences containing one single mismatch. <!-- MATH
 $[\mbox{Na}^+] = 1$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">Na<IMG
 WIDTH="49" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img15.png"
 ALT="$ ^+] = 1$"></SPAN>&nbsp;M, <!-- MATH
 $[\mbox{nucleic acid}] = 4\cdot{}10^{-4}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">nucleic acid<IMG
 WIDTH="83" HEIGHT="34" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img16.png"
 ALT="$ ] = 4\cdot{}10^{-4}$"></SPAN>&nbsp;M</CAPTION>
<TR><TD><IMG
  WIDTH="488" HEIGHT="322" ALIGN="BOTTOM" BORDER="0"
 SRC="html/DNASingleMismatch.png"
 ALT="Image DNASingleMismatch"></TD></TR>
</TABLE>
</DIV>

<P>
For RNA duplexes, the different models to computes the thermodynamic contribution of single mismatch to the helix coil 
stability are more complex.

<P>

<H4><A NAME="SECTION00051310000000000000">
<SPAN  CLASS="textbf">Model from Amber R. Davis and Brent M Znosco, 2007-2008</SPAN></A>
</H4>

<P>
<P></P><DIV ALIGN="CENTER"><!-- MATH
 \begin{multline*}
\Delta h {(\mbox{single-mismatch})} =
\delta{}h_\mathrm{mismatch-nucleotides} +
\delta{}h_\mathrm{mismatch-NN-interaction} +
\delta{}h_\mathrm{AU/GU} \\
\end{multline*}
 -->
<IMG
 WIDTH="557" HEIGHT="58" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img23.png"
 ALT="\begin{multline*}
\Delta h {(\mbox{single-mismatch})} =
\delta{}h_\mathrm{misma...
..._\mathrm{mismatch-NN-interaction} +
\delta{}h_\mathrm{AU/GU} \\
\end{multline*}"></DIV>
<BR CLEAR="ALL">
<P><P></P>

<P>
Where :

<P>
<!-- MATH
 $\delta{}h_\mathrm{mismatch-nucleotides}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="147" HEIGHT="30" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img24.png"
 ALT="$ \delta{}h_\mathrm{mismatch-nucleotides}$"></SPAN> accounts for the identity of the single mismatch nucleotides.

<P>
<!-- MATH
 $\delta{}h_\mathrm{mismatch-NN interaction}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="164" HEIGHT="30" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img25.png"
 ALT="$ \delta{}h_\mathrm{mismatch-NN interaction}$"></SPAN> accounts for the interaction between the mismatch nucleotides and 
the nearest neighbors. (R purine, Y pyrimidine)

<P>
<!-- MATH
 $\delta{}h_\mathrm{AU/GU}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="66" HEIGHT="30" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img26.png"
 ALT="$ \delta{}h_\mathrm{AU/GU}$"></SPAN> accounts for AU or GU nearest neighbors.

<P>
<SPAN  CLASS="textbf">Example</SPAN> :
<P></P><DIV ALIGN="CENTER"><!-- MATH
 \begin{multline*}
\Delta H {\mbox{\texttt{AUC}} \choose \mbox{\texttt{UUG}}} =
\Delta H {\mbox{\texttt{U}} \choose \mbox{\texttt{U}} } + 
1 \times \Delta H {\mbox{\texttt{AU}} } +
\Delta H {\mbox{\texttt{RYY}} \choose \mbox{\texttt{YYR}} } \\
\end{multline*}
 -->
<IMG
 WIDTH="351" HEIGHT="75" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img27.png"
 ALT="\begin{multline*}
\Delta H {\mbox{\texttt{AUC}} \choose \mbox{\texttt{UUG}}} =
...
...
\Delta H {\mbox{\texttt{RYY}} \choose \mbox{\texttt{YYR}} } \\
\end{multline*}"></DIV>
<BR CLEAR="ALL">
<P><P></P>

<P>
(The same computation is performed for <SPAN CLASS="MATH"><IMG
 WIDTH="28" HEIGHT="14" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img14.png"
 ALT="$ \Delta S$"></SPAN>)

<P>

<H4><A NAME="SECTION00051320000000000000">
<SPAN  CLASS="textbf">Model from Zhi Johm Lu, Douglas H. Turner and David H. Mathews, 2006</SPAN></A>
</H4>

<P>
<P></P><DIV ALIGN="CENTER"><!-- MATH
 \begin{multline*}
\Delta h {(\mbox{single-mismatch})} =
\delta{}h_\mathrm{initiation-loop-of-2} +
\delta{}h_\mathrm{per-AU/GU} +
\delta{}h_\mathrm{GG} +
\delta{}h_\mathrm{RU/YU}\\
\end{multline*}
 -->
<IMG
 WIDTH="537" HEIGHT="58" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img28.png"
 ALT="\begin{multline*}
\Delta h {(\mbox{single-mismatch})} =
\delta{}h_\mathrm{initia...
...per-AU/GU} +
\delta{}h_\mathrm{GG} +
\delta{}h_\mathrm{RU/YU}\\
\end{multline*}"></DIV>
<BR CLEAR="ALL">
<P><P></P>

<P>
Where :

<P>
<!-- MATH
 $\delta{}h_\mathrm{initiation-loop-of-2}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="144" HEIGHT="30" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img29.png"
 ALT="$ \delta{}h_\mathrm{initiation-loop-of-2}$"></SPAN> accounts for the initiation of a single non canonical pair.

<P>
<!-- MATH
 $\delta{}h_\mathrm{GG}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="42" HEIGHT="30" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img30.png"
 ALT="$ \delta{}h_\mathrm{GG}$"></SPAN> accounts for a GG single mismatch.

<P>
<!-- MATH
 $\delta{}h_\mathrm{RU/YU}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="66" HEIGHT="30" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img31.png"
 ALT="$ \delta{}h_\mathrm{RU/YU}$"></SPAN> accounts for a 5'RU/3'YU stack with R a purine and Y a pyrimidine.

<P>
<!-- MATH
 $\delta{}h_\mathrm{per-AU/GU}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="94" HEIGHT="30" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img32.png"
 ALT="$ \delta{}h_\mathrm{per-AU/GU}$"></SPAN> accounts for AU or GU nearest neighbors.

<P>
<SPAN  CLASS="textbf">Example</SPAN> :
<P></P><DIV ALIGN="CENTER"><!-- MATH
 \begin{multline*}
\Delta H {\mbox{\texttt{AUC}} \choose \mbox{\texttt{UUG}}} =
\Delta H {\mbox{initiation-loop-of-2}} +
1 \times \Delta H {\mbox{\texttt{per-AU}} } +
\Delta H {\mbox{\texttt{RU}} \choose \mbox{\texttt{YU}} } \\
\end{multline*}
 -->
<IMG
 WIDTH="469" HEIGHT="75" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img33.png"
 ALT="\begin{multline*}
\Delta H {\mbox{\texttt{AUC}} \choose \mbox{\texttt{UUG}}} =
\...
... +
\Delta H {\mbox{\texttt{RU}} \choose \mbox{\texttt{YU}} } \\
\end{multline*}"></DIV>
<BR CLEAR="ALL">
<P><P></P>

<P>
(The same computation is performed for <SPAN CLASS="MATH"><IMG
 WIDTH="28" HEIGHT="14" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img14.png"
 ALT="$ \Delta S$"></SPAN>)

<P>
For further information, see the referenced articles.

<P>
<BR><P></P>
<DIV ALIGN="CENTER">
<TABLE CELLPADDING=3 BORDER="1">
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">model</SPAN></TH>
<TH ALIGN="CENTER"><SPAN  CLASS="textbf">limits</SPAN></TH>
<TH ALIGN="CENTER"><SPAN  CLASS="textbf">Article</SPAN></TH>
</TR>
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">allsanpey</SPAN></TH>
<TD ALIGN="CENTER">DNA</TD>
<TD ALIGN="CENTER">Allawi and SantaLucia (1997), Biochemistry 36: 10581-10594 <br/> Allawi and SantaLucia (1998), Biochemistry 37: 2170-2179 <br/> Allawi and SantaLucia (1998), Nuc Acids Res 26: 2694-2701 <br/> Allawi and SantaLucia (1998), Biochemistry 37: 9435-9444 <br/> Peyret et al. (1999), Biochemistry 38: 3468-3477</TD>
</TR>
<TR><TD ALIGN="CENTER">tur06</TD>
<TD ALIGN="CENTER">RNA</TD>
<TD ALIGN="CENTER">Douglas M Turner et al (2006), Nucleic Acids Research 34: 4912-4924</TD>
</TR>
<TR><TD ALIGN="CENTER">zno07</TD>
<TD ALIGN="CENTER">RNA</TD>
<TD ALIGN="CENTER">Brent M Znosko et al (2007), Biochemistry 46: 13425-13436</TD>
</TR>
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">zno08</SPAN></TH>
<TD ALIGN="CENTER">RNA <br/> at least one adjacent GU base pair</TD>
<TD ALIGN="CENTER">Brent M Znosko et al (2008), Biochemistry 47: 10178-10187</TD>
</TR>
</TABLE>
</DIV><BR>

<P>

<DIV ALIGN="CENTER"><A NAME="2018"></A>
<TABLE>
<CAPTION ALIGN="BOTTOM"><STRONG>Figure:</STRONG>
Comparison of experimental and computed Tm for various sets of
 RNA sequences containing one single mismatch. <!-- MATH
 $[\mbox{Na}^+] = 1$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">Na<IMG
 WIDTH="49" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img15.png"
 ALT="$ ^+] = 1$"></SPAN>&nbsp;M, <!-- MATH
 $[\mbox{nucleic acid}] = 1\cdot{}10^{-4}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">nucleic acid<IMG
 WIDTH="83" HEIGHT="34" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img18.png"
 ALT="$ ] = 1\cdot{}10^{-4}$"></SPAN>&nbsp;M</CAPTION>
<TR><TD><IMG
  WIDTH="518" HEIGHT="322" ALIGN="BOTTOM" BORDER="0"
 SRC="html/RNASingleMismatch.png"
 ALT="Image RNASingleMismatch"></TD></TR>
</TABLE>
</DIV>

<P>

<H3><A NAME="SECTION00051400000000000000">
Tandem mismatches effect</A>
</H3>

<P>
The tandem mismatches (two adjacent mismatches) are taken into account but the two first and positions cannot
be mismatched. Moreover the thermodynamic parameters are still not available for every possible cases.
In such a case, the program, unable to compute any relevant result, will quit with a warning.

<P>
For DNA duplexes, this program computes the hybridisation enthalpy and entropy from the elementary 
parameters of each Crick's pair containing the mismatch(es). 
<P></P><DIV ALIGN="CENTER"><!-- MATH
 \begin{multline*}
\Delta{}h_\mathrm{tandem-mismatch} =
\delta{}h_\mathrm{Crick's-pair-containing-tandem-mismatch} \\+
\sum \delta{}h_\mathrm{Crick's-pair-containing-single-mismatch}
\end{multline*}
 -->
<IMG
 WIDTH="522" HEIGHT="66" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img34.png"
 ALT="\begin{multline*}
\Delta{}h_\mathrm{tandem-mismatch} =
\delta{}h_\mathrm{Crick's...
...
\sum \delta{}h_\mathrm{Crick's-pair-containing-single-mismatch}
\end{multline*}"></DIV>
<BR CLEAR="ALL">
<P><P></P>

<P>
<SPAN  CLASS="textbf">Example</SPAN> :
<P></P><DIV ALIGN="CENTER"><!-- MATH
 \begin{multline*}
\Delta H {\mbox{\texttt{ATGC}} \choose \mbox{\texttt{TCAG}}} =
\Delta H {\mbox{\texttt{AT}} \choose \mbox{\texttt{TC}} } + 
\Delta H {\mbox{\texttt{TG}} \choose \mbox{\texttt{CA}} } + 
\Delta H {\mbox{\texttt{GC}} \choose \mbox{\texttt{AG}} } \\
\end{multline*}
 -->
<IMG
 WIDTH="356" HEIGHT="75" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img35.png"
 ALT="\begin{multline*}
\Delta H {\mbox{\texttt{ATGC}} \choose \mbox{\texttt{TCAG}}} =...
...+
\Delta H {\mbox{\texttt{GC}} \choose \mbox{\texttt{AG}} } \\
\end{multline*}"></DIV>
<BR CLEAR="ALL">
<P><P></P>

<P>
(The same computation is performed for <SPAN CLASS="MATH"><IMG
 WIDTH="28" HEIGHT="14" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img14.png"
 ALT="$ \Delta S$"></SPAN>) 

<P>

<DIV ALIGN="CENTER"><A NAME="2027"></A>
<TABLE>
<CAPTION ALIGN="BOTTOM"><STRONG>Figure:</STRONG>
Comparison of experimental and computed Tm for various sets of
 DNA sequences containing one tandem mismatch. <!-- MATH
 $[\mbox{Na}^+] = 1$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">Na<IMG
 WIDTH="49" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img15.png"
 ALT="$ ^+] = 1$"></SPAN>&nbsp;M, <!-- MATH
 $[\mbox{nucleic acid}] = 4\cdot{}10^{-4}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">nucleic acid<IMG
 WIDTH="83" HEIGHT="34" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img16.png"
 ALT="$ ] = 4\cdot{}10^{-4}$"></SPAN>&nbsp;M</CAPTION>
<TR><TD><IMG
  WIDTH="485" HEIGHT="306" ALIGN="BOTTOM" BORDER="0"
 SRC="html/DNATandemMismatch.png"
 ALT="Image DNATandemMismatch"></TD></TR>
</TABLE>
</DIV>

<P>
For RNA duplexes, the different models to computes the thermodynamic contribution of tandem mismatch to the helix coil 
stability are more complex.

<P>

<H4><A NAME="SECTION00051410000000000000">
Symmetric tandem mismatches : <SPAN  CLASS="textbf">Model from Zhi Johm Lu, Douglas H. Turner and David H. Mathews, 2006</SPAN></A>
</H4>

<P>
<P></P><DIV ALIGN="CENTER"><!-- MATH
 \begin{multline*}
\Delta h {(\mbox{tandem-mismatch})} =
\delta{}h_\mathrm{tandem-mismatch+closing-base-pairs} \\
\end{multline*}
 -->
<IMG
 WIDTH="409" HEIGHT="58" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img36.png"
 ALT="\begin{multline*}
\Delta h {(\mbox{tandem-mismatch})} =
\delta{}h_\mathrm{tandem-mismatch+closing-base-pairs} \\
\end{multline*}"></DIV>
<BR CLEAR="ALL">
<P><P></P>

<P>
Where :

<P>
<!-- MATH
 $\delta{}h_\mathrm{mismatch-nucleotides}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="147" HEIGHT="30" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img24.png"
 ALT="$ \delta{}h_\mathrm{mismatch-nucleotides}$"></SPAN> accounts for the identity of the double mismatch nucleotides and the identity of the base pairs
adjacent to the tandem mismatches.

<P>
<SPAN  CLASS="textbf">Example</SPAN> :
<P></P><DIV ALIGN="CENTER"><!-- MATH
 \begin{multline*}
\Delta H {\mbox{\texttt{G AC C}} \choose \mbox{\texttt{C CA G}}} =
\Delta H {\mbox{\texttt{AC}} \choose \mbox{\texttt{CA}}} -adjacent-to-GC\\
\end{multline*}
 -->
<IMG
 WIDTH="356" HEIGHT="75" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img37.png"
 ALT="\begin{multline*}
\Delta H {\mbox{\texttt{G AC C}} \choose \mbox{\texttt{C CA G}...
...mbox{\texttt{AC}} \choose \mbox{\texttt{CA}}} -adjacent-to-GC\\
\end{multline*}"></DIV>
<BR CLEAR="ALL">
<P><P></P>

<P>
(The same computation is performed for <SPAN CLASS="MATH"><IMG
 WIDTH="28" HEIGHT="14" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img14.png"
 ALT="$ \Delta S$"></SPAN>)    

<P>

<H4><A NAME="SECTION00051420000000000000">
Asymmetric tandem mismatches : <SPAN  CLASS="textbf">Model from Zhi Johm Lu, Douglas H. Turner and David H. Mathews, 2006</SPAN></A>
</H4>

<P>
<P></P><DIV ALIGN="CENTER"><!-- MATH
 \begin{multline*}
\Delta h {(\mbox{tandem-mismatch})} =
( \delta{}h_\mathrm{symmetric-duplex-1} +
\frac{\delta{}h_\mathrm{symmetric-duplex-2} )}{2} \\+
\delta{}h_\mathrm{GG} +
\delta{}h_\mathrm{p}\\
\end{multline*}
 -->
<IMG
 WIDTH="480" HEIGHT="94" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img38.png"
 ALT="\begin{multline*}
\Delta h {(\mbox{tandem-mismatch})} =
( \delta{}h_\mathrm{symm...
...ex-2} )}{2} \\ +
\delta{}h_\mathrm{GG} +
\delta{}h_\mathrm{p}\\
\end{multline*}"></DIV>
<BR CLEAR="ALL">
<P><P></P>

<P>
Where :

<P>
<!-- MATH
 $\delta{}h_\mathrm{symmetric-duplex-1}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="143" HEIGHT="30" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img39.png"
 ALT="$ \delta{}h_\mathrm{symmetric-duplex-1}$"></SPAN> accounts for the enthalpy of a symmetric tandem mismatch composed of
the first closing base pair and the first mismatch nucleotides.

<P>
<!-- MATH
 $\delta{}h_\mathrm{symmetric-duplex-2}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="143" HEIGHT="30" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img40.png"
 ALT="$ \delta{}h_\mathrm{symmetric-duplex-2}$"></SPAN> accounts for the enthalpy of a symmetric tandem mismatch composed of
the second closing base pair and the second mismatch nucleotides.

<P>
<!-- MATH
 $\delta{}h_\mathrm{GG}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="42" HEIGHT="30" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img30.png"
 ALT="$ \delta{}h_\mathrm{GG}$"></SPAN> accounts for a GG pair adjacent to a AA pair or any non canonical pair containing a pyrimidine.

<P>
<!-- MATH
 $\delta{}h_\mathrm{p}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="29" HEIGHT="30" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img41.png"
 ALT="$ \delta{}h_\mathrm{p}$"></SPAN> accounts for an AG or GA pairs adjacent to a UC, CC or CU pair and a UU pair adjacent to an AA pair .

<P>
<SPAN  CLASS="textbf">Example</SPAN> :
<P></P><DIV ALIGN="CENTER"><!-- MATH
 \begin{multline*}
\Delta H {\mbox{\texttt{A GC C}} \choose \mbox{\texttt{U AU G}}} =
( \Delta H {\mbox{\texttt{A GA U}} \choose \mbox{\texttt{U AG A}}} +
\Delta H {\mbox{\texttt{G UC C}} \choose \mbox{\texttt{C CU G}}} +
\Delta H {\mbox{\texttt{GA}} -adjacent-to- \mbox{\texttt{CU}} } \\
\end{multline*}
 -->
<IMG
 WIDTH="575" HEIGHT="75" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img42.png"
 ALT="\begin{multline*}
\Delta H {\mbox{\texttt{A GC C}} \choose \mbox{\texttt{U AU G}...
...lta H {\mbox{\texttt{GA}} -adjacent-to- \mbox{\texttt{CU}} } \\
\end{multline*}"></DIV>
<BR CLEAR="ALL">
<P><P></P>

<P>
(The same computation is performed for <SPAN CLASS="MATH"><IMG
 WIDTH="28" HEIGHT="14" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img14.png"
 ALT="$ \Delta S$"></SPAN>)

<P>
For further information, see the referenced articles.

<P>
<BR><P></P>
<DIV ALIGN="CENTER">
<TABLE CELLPADDING=3 BORDER="1">
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">model</SPAN></TH>
<TH ALIGN="CENTER"><SPAN  CLASS="textbf">limits</SPAN></TH>
<TH ALIGN="CENTER"><SPAN  CLASS="textbf">Article</SPAN></TH>
</TR>
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">allsanpey</SPAN></TH>
<TD ALIGN="CENTER">DNA <br/> only GT mismatches and TA/TG mismatches</TD>
<TD ALIGN="CENTER">Allawi and SantaLucia (1997), Biochemistry 36: 10581-10594 <br/> Allawi and SantaLucia (1998), Biochemistry 37: 2170-2179 <br/> Allawi and SantaLucia (1998), Nuc Acids Res 26: 2694-2701 <br/> Allawi and SantaLucia (1998),Biochemistry 37: 9435-9444 <br/> Peyret et al. (1999), Biochemistry 38: 3468-3477 </TD>
</TR>
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">tur99</SPAN></TH>
<TD ALIGN="CENTER">RNA <br/> no adjacent GU or UG base pairs</TD>
<TD ALIGN="CENTER">Douglas M Turner et al (1999), J.Mol.Biol.  288: 911-940</TD>
</TR>
</TABLE>
</DIV><BR>

<P>

<DIV ALIGN="CENTER"><A NAME="2045"></A>
<TABLE>
<CAPTION ALIGN="BOTTOM"><STRONG>Figure:</STRONG>
Comparison of experimental and computed Tm for various sets of
 RNA sequences containing one tandem mismatch. <!-- MATH
 $[\mbox{Na}^+] = 1$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">Na<IMG
 WIDTH="49" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img15.png"
 ALT="$ ^+] = 1$"></SPAN>&nbsp;M, <!-- MATH
 $[\mbox{nucleic acid}] = 1\cdot{}10^{-4}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">nucleic acid<IMG
 WIDTH="83" HEIGHT="34" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img18.png"
 ALT="$ ] = 1\cdot{}10^{-4}$"></SPAN>&nbsp;M</CAPTION>
<TR><TD><IMG
  WIDTH="457" HEIGHT="308" ALIGN="BOTTOM" BORDER="0"
 SRC="html/RNATandemMismatch.png"
 ALT="Image RNATandemMismatch"></TD></TR>
</TABLE>
</DIV>

<P>

<H3><A NAME="SECTION00051500000000000000">
Internal loop effect</A>
</H3>

<P>
The internal loops (more than two adjacent mismatches) are taken into account but the two first and positions cannot
be mismatched. Moreover the thermodynamic parameters are still not available for every possible cases.
In such a case, the program, unable to compute any relevant result, will quit with a warning.
Moreover, the thermodynamics of the nucleic acids within the internal loop are salt 
independent and no salt correction will be applied to it. However, the thermodynamics
of the terminal mismatches are salt dependent and a salt correction will be applied
to them.
The thermodynamic model for DNA and RNA duplexes are similar.

<P>

<H4><A NAME="SECTION00051510000000000000">
DNA duplexes :<SPAN  CLASS="textbf">Model from John Santalucia, Jr. and Donald Hicks, 2004</SPAN></A>
</H4> 

<P>
<P></P><DIV ALIGN="CENTER"><!-- MATH
 \begin{multline*}
\Delta h {(\mbox{internal-loop(n)})} =
\delta{}h_\mathrm{asymmetry} +
\delta{}h_\mathrm{left-terminal-mismatch} \\+
\delta{}h_\mathrm{right-terminal-mismatch}\\
\Delta s {(\mbox{internal-loop(n)})} =
\delta{}s_\mathrm{loop(n)} +
\delta{}s_\mathrm{asymmetry} +
\delta{}s_\mathrm{left-terminal-mismatch} \\+
\delta{}s_\mathrm{right-terminal-mismatch}\\
\end{multline*}
 -->
<IMG
 WIDTH="501" HEIGHT="130" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img43.png"
 ALT="\begin{multline*}
\Delta h {(\mbox{internal-loop(n)})} =
\delta{}h_\mathrm{asymm...
...nal-mismatch} \\ +
\delta{}s_\mathrm{right-terminal-mismatch}\\
\end{multline*}"></DIV>
<BR CLEAR="ALL">
<P><P></P>

<P>
Where :

<P>
<!-- MATH
 $\delta{}h_\mathrm{internal-loop(n)}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="116" HEIGHT="30" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img44.png"
 ALT="$ \delta{}h_\mathrm{internal-loop(n)}$"></SPAN> accounts for the internal loop of n nucleotides.

<P>
<!-- MATH
 $\delta{}h_\mathrm{asymmetry}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="84" HEIGHT="30" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img45.png"
 ALT="$ \delta{}h_\mathrm{asymmetry}$"></SPAN> accounts for the internal loop asymmetry (when the number of
nucleic acid within the internal loop is higher in one of the strand).

<P>
<!-- MATH
 $\delta{}h_\mathrm{left-terminal-mismatch}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="160" HEIGHT="30" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img46.png"
 ALT="$ \delta{}h_\mathrm{left-terminal-mismatch}$"></SPAN> accounts for the identity of the first mismatch 
nucleotides of the loop.

<P>
<!-- MATH
 $\delta{}h_\mathrm{right-terminal-mismatch}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="168" HEIGHT="30" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img47.png"
 ALT="$ \delta{}h_\mathrm{right-terminal-mismatch}$"></SPAN> accounts for the identity of the last mismatch 
nucleotides of the loop.

<P>
<SPAN  CLASS="textbf">Example</SPAN> : Symmetric internal loop
<P></P><DIV ALIGN="CENTER"><!-- MATH
 \begin{multline*}
\Delta H {\mbox{\texttt{G ACCG C}} \choose \mbox{\texttt{C CATA G}}} =
\Delta H {\mbox{\texttt{GA}} \choose \mbox{\texttt{CC}}} +
\Delta H {\mbox{\texttt{GC}} \choose \mbox{\texttt{AG}}}\\
\Delta S {\mbox{\texttt{G ACCG C}} \choose \mbox{\texttt{C CATA G}}} = 
\Delta S {\mbox{loop of 8}} +
\Delta S {\mbox{\texttt{GA}} \choose \mbox{\texttt{CC}}} +
\Delta S {\mbox{\texttt{GC}} \choose \mbox{\texttt{AG}}}\\
\end{multline*}
 -->
<IMG
 WIDTH="457" HEIGHT="119" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img48.png"
 ALT="\begin{multline*}
\Delta H {\mbox{\texttt{G ACCG C}} \choose \mbox{\texttt{C CAT...
...}} +
\Delta S {\mbox{\texttt{GC}} \choose \mbox{\texttt{AG}}}\\
\end{multline*}"></DIV>
<BR CLEAR="ALL">
<P><P></P>

<P>

<H4><A NAME="SECTION00051520000000000000">
RNA duplexes :<SPAN  CLASS="textbf">Model from Zhi Johm Lu, Douglas H. Turner and David H. Mathews, 2006</SPAN></A>
</H4>

<P>
<P></P><DIV ALIGN="CENTER"><!-- MATH
 \begin{multline*}
\Delta h {(\mbox{internal-loop(n)})} =
\delta{}h_\mathrm{initiation-loop(n)} +
\delta{}h_\mathrm{per-AU/GU} +
(n1 - n2) \delta{}h_\mathrm{asymmetry} \\+
\delta{}h_\mathrm{first-non-canonical-pairs}\\
\end{multline*}
 -->
<IMG
 WIDTH="537" HEIGHT="82" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img49.png"
 ALT="\begin{multline*}
\Delta h {(\mbox{internal-loop(n)})} =
\delta{}h_\mathrm{initi...
...{asymmetry} \\ +
\delta{}h_\mathrm{first-non-canonical-pairs}\\
\end{multline*}"></DIV>
<BR CLEAR="ALL">
<P><P></P>

<P>
Where :

<P>
<!-- MATH
 $\delta{}h_\mathrm{initiation-loop(n)}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="124" HEIGHT="30" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img50.png"
 ALT="$ \delta{}h_\mathrm{initiation-loop(n)}$"></SPAN> accounts for the internal loop of n nucleotides.

<P>
<!-- MATH
 $\delta{}h_\mathrm{asymmetry}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="84" HEIGHT="30" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img45.png"
 ALT="$ \delta{}h_\mathrm{asymmetry}$"></SPAN> accounts for the internal loop asymmetry (when the number of
there is an unequal numbers of nucleotides on each side) with n1 and n2 the
number of nucleotides on each strand..

<P>
<!-- MATH
 $\delta{}h_\mathrm{per_AU/GU}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="84" HEIGHT="30" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img51.png"
 ALT="$ \delta{}h_\mathrm{per_AU/GU}$"></SPAN> accounts for each AU or GU base pair adjacent
to the internal loop.

<P>
<!-- MATH
 $\delta{}h_\mathrm{first-non-canonical-pairs}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="173" HEIGHT="30" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img52.png"
 ALT="$ \delta{}h_\mathrm{first-non-canonical-pairs}$"></SPAN> accounts for each sequence
specific first mismatch (bonus). It is not applied to loops of the form 1 &#215;(n-1) with
n &gt; 2.

<P>
<SPAN  CLASS="textbf">Example</SPAN> : asymmetric internal loop
<P></P><DIV ALIGN="CENTER"><!-- MATH
 \begin{multline*}
\Delta H {\mbox{\texttt{A ACCG C}} \choose \mbox{\texttt{U C-UA G}}} =
\Delta H {\mbox{loop initiation(7)}} +
1 \times \Delta H {\mbox{\texttt{per-AU}}} +
(4 - 3)\Delta H {\mbox{asymmetry}}\\
\end{multline*}
 -->
<IMG
 WIDTH="553" HEIGHT="75" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img53.png"
 ALT="\begin{multline*}
\Delta H {\mbox{\texttt{A ACCG C}} \choose \mbox{\texttt{U C-U...
...{\mbox{\texttt{per-AU}}} +
(4 - 3)\Delta H {\mbox{asymmetry}}\\
\end{multline*}"></DIV>
<BR CLEAR="ALL">
<P><P></P>

<P>
(The same computation is performed for <SPAN CLASS="MATH"><IMG
 WIDTH="28" HEIGHT="14" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img14.png"
 ALT="$ \Delta S$"></SPAN>)    

<P>
For further information, see the referenced articles.

<P>
<BR><P></P>
<DIV ALIGN="CENTER">
<TABLE CELLPADDING=3 BORDER="1">
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">model</SPAN></TH>
<TH ALIGN="CENTER"><SPAN  CLASS="textbf">limits</SPAN></TH>
<TH ALIGN="CENTER"><SPAN  CLASS="textbf">Article</SPAN></TH>
</TR>
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">san04</SPAN></TH>
<TD ALIGN="CENTER">DNA <br/> missing asymmetry penalty <br/> not tested with experimental results</TD>
<TD ALIGN="CENTER">Santalucia et al (2004), Annu. Rev. Biophys. Biomol. Struct 33 : 415-440</TD>
</TR>
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">tur06</SPAN></TH>
<TD ALIGN="CENTER">RNA <br/> not tested with experimental results</TD>
<TD ALIGN="CENTER">Douglas M Turner et al (2006), Nucleic Acids Research 34: 4912-4924</TD>
</TR>
</TABLE>
</DIV><BR>

<P>

<DIV ALIGN="CENTER"><A NAME="2069"></A>
<TABLE>
<CAPTION ALIGN="BOTTOM"><STRONG>Figure:</STRONG>
Comparison of experimental and computed Tm for various sets of
 RNA sequences containing one 1x2 internal loop. <!-- MATH
 $[\mbox{Na}^+] = 1$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">Na<IMG
 WIDTH="49" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img15.png"
 ALT="$ ^+] = 1$"></SPAN>&nbsp;M, <!-- MATH
 $[\mbox{nucleic acid}] = 1\cdot{}10^{-4}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">nucleic acid<IMG
 WIDTH="83" HEIGHT="34" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img18.png"
 ALT="$ ] = 1\cdot{}10^{-4}$"></SPAN>&nbsp;M</CAPTION>
<TR><TD><IMG
  WIDTH="493" HEIGHT="347" ALIGN="BOTTOM" BORDER="0"
 SRC="html/1x2InternalLoop.png"
 ALT="Image 1x2InternalLoop"></TD></TR>
</TABLE>
</DIV>

<P>

<H3><A NAME="SECTION00051600000000000000">
GU wobble base pairs effect</A>
</H3>

<P>
The wobble GU base pairs are taken into account. This pairing is a non-Watson-Crick base pairing between two nucleotides 
in RNA molecules, but the thermodynamic stability of a wobble base pair is comparable to that of a Watson-Crick base pair.
Melting can also compute the thermodynamic of patterns with several adjacent GU base pairs.
This program computes the hybridisation enthalpy and entropy from the elementary 
parameters of each Crick's pair containing the GU base pairs.

<P>
<P><!-- MATH
 \begin{displaymath}
\begin{array}[t]{ccc}
  \Delta{}h_\mathrm{pattern-composed-of-GU-base-pairs}&=&\sum \delta{}h_\mathrm{Crick's pair-containing-GU-base-pairs}\\
  \end{array}
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="569" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img54.png"
 ALT="$\displaystyle \begin{array}[t]{ccc}
\Delta{}h_\mathrm{pattern-composed-of-GU-ba...
...&amp;=&amp;\sum \delta{}h_\mathrm{Crick's pair-containing-GU-base-pairs}\\
\end{array}$">
</DIV><P></P>

<P>
<SPAN  CLASS="textbf">Examples</SPAN> : One GU base pair
<P></P><DIV ALIGN="CENTER"><!-- MATH
 \begin{multline*}
\Delta H {\mbox{\texttt{GUC}} \choose \mbox{\texttt{CGG}}} =
\Delta H {\mbox{\texttt{GU}} \choose \mbox{\texttt{CG}}} +
\Delta H {\mbox{\texttt{UC}} \choose \mbox{\texttt{GG}}}\\
\end{multline*}
 -->
<IMG
 WIDTH="257" HEIGHT="75" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img55.png"
 ALT="\begin{multline*}
\Delta H {\mbox{\texttt{GUC}} \choose \mbox{\texttt{CGG}}} =
...
...}} +
\Delta H {\mbox{\texttt{UC}} \choose \mbox{\texttt{GG}}}\\
\end{multline*}"></DIV>
<BR CLEAR="ALL">
<P><P></P>

<P>
<SPAN  CLASS="textbf">Examples</SPAN> : Two adjacent GU base pairs
<P></P><DIV ALIGN="CENTER"><!-- MATH
 \begin{multline*}
\Delta H {\mbox{\texttt{GUGC}} \choose \mbox{\texttt{CGUG}}} =
\Delta H {\mbox{\texttt{GU}} \choose \mbox{\texttt{CG}}} +
\Delta H {\mbox{\texttt{UG}} \choose \mbox{\texttt{GU}}} +
\Delta H {\mbox{\texttt{UC}} \choose \mbox{\texttt{GG}}}\\
\end{multline*}
 -->
<IMG
 WIDTH="356" HEIGHT="75" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img56.png"
 ALT="\begin{multline*}
\Delta H {\mbox{\texttt{GUGC}} \choose \mbox{\texttt{CGUG}}} =...
...}} +
\Delta H {\mbox{\texttt{UC}} \choose \mbox{\texttt{GG}}}\\
\end{multline*}"></DIV>
<BR CLEAR="ALL">
<P><P></P>

<P>
(The same computation is performed for <SPAN CLASS="MATH"><IMG
 WIDTH="28" HEIGHT="14" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img14.png"
 ALT="$ \Delta S$"></SPAN>) 

<P>
For further information, see the referenced articles.

<P>
<BR><P></P>
<DIV ALIGN="CENTER">
<TABLE CELLPADDING=3 BORDER="1">
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">model</SPAN></TH>
<TH ALIGN="CENTER"><SPAN  CLASS="textbf">limits</SPAN></TH>
<TH ALIGN="CENTER"><SPAN  CLASS="textbf">Article</SPAN></TH>
</TR>
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">tur99</SPAN></TH>
<TD ALIGN="CENTER">RNA</TD>
<TD ALIGN="CENTER">Douglas M Turner et al (1999), J.Mol.Biol.  288: 911-940</TD>
</TR>
</TABLE>
</DIV><BR>

<P>

<DIV ALIGN="CENTER"><A NAME="2084"></A>
<TABLE>
<CAPTION ALIGN="BOTTOM"><STRONG>Figure:</STRONG>
Comparison of experimental and computed Tm for various sets of
 RNA sequences containing GU base pairs. <!-- MATH
 $[\mbox{Na}^+] = 1$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">Na<IMG
 WIDTH="49" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img15.png"
 ALT="$ ^+] = 1$"></SPAN>&nbsp;M, <!-- MATH
 $[\mbox{nucleic acid}] = 1\cdot{}10^{-4}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">nucleic acid<IMG
 WIDTH="83" HEIGHT="34" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img18.png"
 ALT="$ ] = 1\cdot{}10^{-4}$"></SPAN>&nbsp;M</CAPTION>
<TR><TD><IMG
  WIDTH="508" HEIGHT="322" ALIGN="BOTTOM" BORDER="0"
 SRC="html/GUBasePairs.png"
 ALT="Image GUBasePairs"></TD></TR>
</TABLE>
</DIV>

<P>

<H3><A NAME="SECTION00051700000000000000">
Single dangling end effect</A>
</H3>

<P>
The single dangling ends, that is the unmatched terminal nucleotides, can be taken into
account, but all the thermodynamic parameters are not available. In such a case, 
the result is unpredictable, and all cases are possible. 

<P>
For DNA and RNA duplexes, this program computes the hybridisation enthalpy and entropy from the elementary 
parameters of the Crick's pair containing the single dangling end. 
<P><!-- MATH
 \begin{displaymath}
\begin{array}[t]{ccc}
  \Delta{}h_\mathrm{single-dangling-end}&=&\delta{}h_\mathrm{Crick's-pair-containing-the-dangling-end}\\
  \end{array}
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="477" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img57.png"
 ALT="$\displaystyle \begin{array}[t]{ccc}
\Delta{}h_\mathrm{single-dangling-end}&amp;=&amp;\delta{}h_\mathrm{Crick's-pair-containing-the-dangling-end}\\
\end{array}$">
</DIV><P></P>

<P>
<SPAN  CLASS="textbf">Example</SPAN> :
If the duplex is :

<DIV CLASS="alltt" ALIGN="LEFT">
<TT>
 GCTAG<SPAN &nbsp;CLASS="textbf">-</SPAN>
<BR>CGATC<SPAN &nbsp;CLASS="textbf">A</SPAN>
<BR></TT>
</DIV>

<P></P><DIV ALIGN="CENTER"><!-- MATH
 \begin{multline*}
\Delta H {\mbox{\texttt{GCTAG-}} \choose \mbox{\texttt{CGATCCA}}} =
\Delta H {\mbox{perfectly-matching-sequence}} +
\Delta H {\mbox{single-dangling-end}} \\
\Delta H {\mbox{\texttt{GCTAG-}} \choose \mbox{\texttt{CGATCCA}}} =
\Delta H {\mbox{\texttt{GCTAG}} \choose \mbox{\texttt{CGATCC}}} +
\Delta H {\mbox{\texttt{G-}} \choose \mbox{\texttt{CA}}} \\
\end{multline*}
 -->
<IMG
 WIDTH="525" HEIGHT="119" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img58.png"
 ALT="\begin{multline*}
\Delta H {\mbox{\texttt{GCTAG-}} \choose \mbox{\texttt{CGATCCA...
...} +
\Delta H {\mbox{\texttt{G-}} \choose \mbox{\texttt{CA}}} \\
\end{multline*}"></DIV>
<BR CLEAR="ALL">
<P><P></P>

<P>
(The same computation is performed for <SPAN CLASS="MATH"><IMG
 WIDTH="28" HEIGHT="14" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img14.png"
 ALT="$ \Delta S$"></SPAN>)

<P>
For further information, see the referenced articles.

<P>
<BR><P></P>
<DIV ALIGN="CENTER">
<TABLE CELLPADDING=3 BORDER="1">
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">model</SPAN></TH>
<TH ALIGN="CENTER"><SPAN  CLASS="textbf">limits</SPAN></TH>
<TH ALIGN="CENTER"><SPAN  CLASS="textbf">Article</SPAN></TH>
</TR>
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">bom00</SPAN></TH>
<TD ALIGN="CENTER">DNA</TD>
<TD ALIGN="CENTER">Bommarito et al. (2000), Nuc Acids Res 28: 1929-1934</TD>
<TR><TD ALIGN="CENTER">sugdna02</TD>
<TD ALIGN="CENTER">DNA <br/> only terminal poly A <br/> self complementary sequences</TD>
<TD ALIGN="CENTER">Sugimoto et al. (2002), J. Am. Chem. Soc. 124: 10367-10372</TD>
</TR>
<TR><TD ALIGN="CENTER">sugrna02</TD>
<TD ALIGN="CENTER">RNA <br/> only terminal poly A <br/> self complementary sequences</TD>
<TD ALIGN="CENTER">Sugimoto et al. (2002), J. Am. Chem. Soc. 124: 10367-10372</TD>
</TR>
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">ser08</SPAN></TH>
<TD ALIGN="CENTER">RNA <br/> only 3' UA, GU and UG terminal base pairs <br/> only 5' UG and GU terminal base pairs</TD>
<TD ALIGN="CENTER">Martin J Serra et al. (2006), Nucleic Acids research 34: 3338-3344 <br/> Martin J Serra et al. (2008), Nucleic Acids research 36: 5652-5659</TD>
</TR>
</TABLE>
</DIV><BR>

<P>

<DIV ALIGN="CENTER"><A NAME="2095"></A>
<TABLE>
<CAPTION ALIGN="BOTTOM"><STRONG>Figure:</STRONG>
Comparison of experimental and computed Tm for various sets of
 DNA sequences containing single dangling ends. <!-- MATH
 $[\mbox{Na}^+] = 1$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">Na<IMG
 WIDTH="49" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img15.png"
 ALT="$ ^+] = 1$"></SPAN>&nbsp;M, <!-- MATH
 $[\mbox{nucleic acid}] = 1\cdot{}10^{-4}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">nucleic acid<IMG
 WIDTH="83" HEIGHT="34" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img18.png"
 ALT="$ ] = 1\cdot{}10^{-4}$"></SPAN>&nbsp;M</CAPTION>
<TR><TD><IMG
  WIDTH="486" HEIGHT="310" ALIGN="BOTTOM" BORDER="0"
 SRC="html/DNASingleDanglingEnd.png"
 ALT="Image DNASingleDanglingEnd"></TD></TR>
</TABLE>
</DIV>

<P>

<DIV ALIGN="CENTER"><A NAME="2096"></A>
<TABLE>
<CAPTION ALIGN="BOTTOM"><STRONG>Figure:</STRONG>
Comparison of experimental and computed Tm for various sets of
 RNA sequences containing single dangling ends. <!-- MATH
 $[\mbox{Na}^+] = 1$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">Na<IMG
 WIDTH="49" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img15.png"
 ALT="$ ^+] = 1$"></SPAN>&nbsp;M, <!-- MATH
 $[\mbox{nucleic acid}] = 1\cdot{}10^{-4}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">nucleic acid<IMG
 WIDTH="83" HEIGHT="34" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img18.png"
 ALT="$ ] = 1\cdot{}10^{-4}$"></SPAN>&nbsp;M</CAPTION>
<TR><TD><IMG
  WIDTH="442" HEIGHT="293" ALIGN="BOTTOM" BORDER="0"
 SRC="html/RNASingleDanglingEnd.png"
 ALT="Image RNASingleDanglingEnd"></TD></TR>
</TABLE>
</DIV>

<P>

<H3><A NAME="SECTION00051800000000000000">
Double dangling end effect</A>
</H3>

<P>
The double dangling ends, that is the two adjacent unmatched terminal nucleotides, can be taken into
account (mostly for RNA sequences). This program computes the hybridisation enthalpy and entropy in two times :
First, it computes the energy from the single dangling end as if the duplex contained
only a single danging end and then, it adds a bonus for the second dangling end if it is necessary. 
<P></P><DIV ALIGN="CENTER"><!-- MATH
 \begin{multline*}
\Delta{}h_\mathrm{double-dangling-end} =
\delta{}h_\mathrm{single-dangling-end} \\+
\delta{}h_\mathrm{bonus-second-dangling-end}\\
\end{multline*}
 -->
<IMG
 WIDTH="362" HEIGHT="82" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img59.png"
 ALT="\begin{multline*}
\Delta{}h_\mathrm{double-dangling-end} =
\delta{}h_\mathrm{sin...
...gling-end} \\ +
\delta{}h_\mathrm{bonus-second-dangling-end}\\
\end{multline*}"></DIV>
<BR CLEAR="ALL">
<P><P></P>

<P>
<SPAN  CLASS="textbf">Example</SPAN> :

<P>
<P></P><DIV ALIGN="CENTER"><!-- MATH
 \begin{multline*}
\Delta H {\mbox{\texttt{UAC}} \choose \mbox{\texttt{A- -}}} =
\Delta H {\mbox{UA}} +
\Delta H {\mbox{bonus-pyrimidine-purine-pyrimidine}} \\
\end{multline*}
 -->
<IMG
 WIDTH="431" HEIGHT="75" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img60.png"
 ALT="\begin{multline*}
\Delta H {\mbox{\texttt{UAC}} \choose \mbox{\texttt{A--}}} =
\...
...{UA}} +
\Delta H {\mbox{bonus-pyrimidine-purine-pyrimidine}} \\
\end{multline*}"></DIV>
<BR CLEAR="ALL">
<P><P></P>

<P>
(The same computation is performed for <SPAN CLASS="MATH"><IMG
 WIDTH="28" HEIGHT="14" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img14.png"
 ALT="$ \Delta S$"></SPAN>)

<P>
For further information, see the referenced articles.

<P>
<BR><P></P>
<DIV ALIGN="CENTER">
<TABLE CELLPADDING=3 BORDER="1">
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">model</SPAN></TH>
<TH ALIGN="CENTER"><SPAN  CLASS="textbf">limits</SPAN></TH>
<TH ALIGN="CENTER"><SPAN  CLASS="textbf">Article</SPAN></TH>
</TR>
<TR><TD ALIGN="CENTER">sugdna02</TD>
<TD ALIGN="CENTER">DNA <br/> only terminal poly A <br/> self complementary sequences</TD>
<TD ALIGN="CENTER">Sugimoto et al. (2002), J. Am. Chem. Soc. 124: 10367-10372</TD>
</TR>
<TR><TD ALIGN="CENTER">sugrna02</TD>
<TD ALIGN="CENTER">RNA <br/> only terminal poly A <br/> self complementary sequences</TD>
<TD ALIGN="CENTER">Sugimoto et al. (2002), J. Am. Chem. Soc. 124: 10367-10372</TD>
</TR>
<TR><TD ALIGN="CENTER">ser05</TD>
<TD ALIGN="CENTER">RNA <br/> depends on the available thermodynamic parameters for single dangling end</TD>
<TD ALIGN="CENTER">Martin J Serra et al. (2005), RNA 11: 512-516</TD>
</TR>
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">ser06</SPAN></TH>
<TD ALIGN="CENTER">RNA</TD>
<TD ALIGN="CENTER">Martin J Serra et al. (2006),Nucleic Acids research 34: 3338-3344</TD>
</TR>
</TABLE>
</DIV><BR>

<P>

<DIV ALIGN="CENTER"><A NAME="2101"></A>
<TABLE>
<CAPTION ALIGN="BOTTOM"><STRONG>Figure:</STRONG>
Comparison of experimental and computed Tm for various sets of
 RNA sequences containing double dangling ends. <!-- MATH
 $[\mbox{Na}^+] = 1$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">Na<IMG
 WIDTH="49" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img15.png"
 ALT="$ ^+] = 1$"></SPAN>&nbsp;M, <!-- MATH
 $[\mbox{nucleic acid}] = 1\cdot{}10^{-4}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">nucleic acid<IMG
 WIDTH="83" HEIGHT="34" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img18.png"
 ALT="$ ] = 1\cdot{}10^{-4}$"></SPAN>&nbsp;M</CAPTION>
<TR><TD><IMG
  WIDTH="443" HEIGHT="354" ALIGN="BOTTOM" BORDER="0"
 SRC="html/RNASecondDanglingEnd.png"
 ALT="Image RNASecondDanglingEnd"></TD></TR>
</TABLE>
</DIV>

<P>

<H3><A NAME="SECTION00041900000000000000">
Long dangling end effect (poly A)</A>
</H3>

<P>
The long dangling ends, that is all the adjacent unmatched terminal nucleotides, can be taken into
account (only for polyA dangling ends for the moment). It is possible to compute the thermodynamic 
form one to four poly A dangling end. This program computes the hybridisation enthalpy 
and entropy from the parameters of the long dangling end with the adjacent terminal base pair.  
<P></P><DIV ALIGN="CENTER"><!-- MATH
 \begin{multline*}
\Delta{}h_\mathrm{long-dangling-end} =
\delta{}h_\mathrm{adjacent-terminal-base-pair+polyA} \\
\end{multline*}
 -->
<IMG
 WIDTH="383" HEIGHT="58" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img61.png"
 ALT="\begin{multline*}
\Delta{}h_\mathrm{long-dangling-end} =
\delta{}h_\mathrm{adjacent-terminal-base-pair+polyA} \\
\end{multline*}"></DIV>
<BR CLEAR="ALL">
<P><P></P>

<P>
<SPAN  CLASS="textbf">Example</SPAN> :

<P>
If the duplex is :

<DIV CLASS="alltt" ALIGN="LEFT">
<TT>
 GCTAG<SPAN &nbsp;CLASS="textbf">--</SPAN>
<BR>CGATC<SPAN &nbsp;CLASS="textbf">AAA</SPAN>
<BR></TT>
</DIV>

<P></P><DIV ALIGN="CENTER"><!-- MATH
 \begin{multline*}
\Delta H {\mbox{\texttt{GCTAG- - -}} \choose \mbox{\texttt{CGATCCAAA}}} =
\Delta H {\mbox{perfectly-matching-sequence}} +
\Delta H {\mbox{long-dangling-end}} \\
\Delta H {\mbox{\texttt{GCTAG- - -}} \choose \mbox{\texttt{CGATCCAAA}}} =
\Delta H {\mbox{\texttt{GCTAG}} \choose \mbox{\texttt{CGATCC}}} +
\Delta H {\mbox{\texttt{G- - -}} \choose \mbox{\texttt{CAAA}}} \\
\end{multline*}
 -->
<IMG
 WIDTH="535" HEIGHT="119" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img62.png"
 ALT="\begin{multline*}
\Delta H {\mbox{\texttt{GCTAG---}} \choose \mbox{\texttt{CGATC...
...\Delta H {\mbox{\texttt{G---}} \choose \mbox{\texttt{CAAA}}} \\
\end{multline*}"></DIV>
<BR CLEAR="ALL">
<P><P></P>

<P>
(The same computation is performed for <SPAN CLASS="MATH"><IMG
 WIDTH="28" HEIGHT="14" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img14.png"
 ALT="$ \Delta S$"></SPAN>)

<P>
For further information, see the referenced articles.

<P>
<BR><P></P>
<DIV ALIGN="CENTER">
<TABLE CELLPADDING=3 BORDER="1">
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">model</SPAN></TH>
<TH ALIGN="CENTER"><SPAN  CLASS="textbf">limits</SPAN></TH>
<TH ALIGN="CENTER"><SPAN  CLASS="textbf">Article</SPAN></TH>
</TR>
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">sugdna02</SPAN></TH>
<TD ALIGN="CENTER">DNA <br/> only terminal poly A <br/> self complementary sequences</TD>
<TD ALIGN="CENTER">Sugimoto et al. (2002), J. Am. Chem. Soc. 124: 10367-10372</TD>
</TR>
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">sugrna02</SPAN></TH>
<TD ALIGN="CENTER">RNA <br/> only terminal poly A <br/> self complementary sequences</TD>
<TD ALIGN="CENTER">Sugimoto et al. (2002), J. Am. Chem. Soc. 124: 10367-10372</TD>
</TR>
</TABLE>
</DIV><BR>

<P>

<DIV ALIGN="CENTER"><A NAME="2112"></A>
<TABLE>
<CAPTION ALIGN="BOTTOM"><STRONG>Figure:</STRONG>
Comparison of experimental and computed Tm for various sets of DNA sequences containing long polyA dangling ends. <!-- MATH
 $[\mbox{Na}^+] = 1$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">Na<IMG
 WIDTH="49" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img15.png"
 ALT="$ ^+] = 1$"></SPAN>&nbsp;M, <!-- MATH
 $[\mbox{nucleic acid}] = 1\cdot{}10^{-4}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">nucleic acid<IMG
 WIDTH="83" HEIGHT="34" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img18.png"
 ALT="$ ] = 1\cdot{}10^{-4}$"></SPAN>&nbsp;M</CAPTION>
<TR><TD><IMG
  WIDTH="578" HEIGHT="354" ALIGN="BOTTOM" BORDER="0"
 SRC="html/DNALongDanglingEnd.png"
 ALT="Image DNALongDanglingEnd"></TD></TR>
</TABLE>
</DIV>

<P>

<DIV ALIGN="CENTER"><A NAME="2113"></A>
<TABLE>
<CAPTION ALIGN="BOTTOM"><STRONG>Figure:</STRONG>
Comparison of experimental and computed Tm for various sets of RNA sequences containing long polyA dangling ends. <!-- MATH
 $[\mbox{Na}^+] = 1$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">Na<IMG
 WIDTH="49" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img15.png"
 ALT="$ ^+] = 1$"></SPAN>&nbsp;M, <!-- MATH
 $[\mbox{nucleic acid}] = 1\cdot{}10^{-4}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">nucleic acid<IMG
 WIDTH="83" HEIGHT="34" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img18.png"
 ALT="$ ] = 1\cdot{}10^{-4}$"></SPAN>&nbsp;M</CAPTION>
<TR><TD><IMG
  WIDTH="500" HEIGHT="321" ALIGN="BOTTOM" BORDER="0"
 SRC="html/RNALongDanglingEnd.png"
 ALT="Image RNALongDanglingEnd"></TD></TR>
</TABLE>
</DIV>

<P>

<H3><A NAME="SECTION000511000000000000000">
Single bulge loop effect</A>
</H3>

<P>
The single bulge loops, that is the single unmatched internal nucleotides, can be taken into
account. , but all the thermodynamic parameters are not available. In such a case, 
the result is unpredictable, and all cases are possible. 
There are several different models to compute the thermodynamic of single bulge loop:

<P>

<H4><A NAME="SECTION000511010000000000000">
DNA and RNA duplexes :<SPAN  CLASS="textbf">nearest neighbor model "NNN"</SPAN></A>
</H4> 

<P>
<P></P><DIV ALIGN="CENTER"><!-- MATH
 \begin{multline*}
\Delta h {(\mbox{single-bulge-loop})} =
\delta{}h_\mathrm{unpaired-nucleotid+adjacent-base-pairs} \\
\end{multline*}
 -->
<IMG
 WIDTH="420" HEIGHT="58" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img63.png"
 ALT="\begin{multline*}
\Delta h {(\mbox{single-bulge-loop})} =
\delta{}h_\mathrm{unpaired-nucleotid+adjacent-base-pairs} \\
\end{multline*}"></DIV>
<BR CLEAR="ALL">
<P><P></P>

<P>
<SPAN  CLASS="textbf">Example</SPAN> :
If the duplex is :

<DIV CLASS="alltt" ALIGN="LEFT">
<TT>
 GCT<SPAN &nbsp;CLASS="textbf">T</SPAN>AGGC
<BR>CGA<SPAN &nbsp;CLASS="textbf">-</SPAN>TCCG
<BR></TT>
</DIV>

<P></P><DIV ALIGN="CENTER"><!-- MATH
 \begin{multline*}
\Delta H {\mbox{\texttt{GCTTAGGC}} \choose \mbox{\texttt{CGA-TCCG}}} =
\Delta H {\mbox{perfectly-matching-sequence-1}} +
\Delta H {\mbox{single-bulge-loop}} \\+
\Delta H {\mbox{perfectly-matching-sequence-2}}\\
\Delta H {\mbox{\texttt{GCTTAGGC}} \choose \mbox{\texttt{CGA-TCCG}}} =
\Delta H {\mbox{\texttt{GCT}} \choose \mbox{\texttt{CGA}}} +
\Delta H {\mbox{\texttt{TTA}} \choose \mbox{\texttt{A-T}}} +
\Delta H {\mbox{\texttt{AGGC}} \choose \mbox{\texttt{TCCG}}}\\
\end{multline*}
 -->
<IMG
 WIDTH="533" HEIGHT="145" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img64.png"
 ALT="\begin{multline*}
\Delta H {\mbox{\texttt{GCTTAGGC}} \choose \mbox{\texttt{CGA-T...
...
\Delta H {\mbox{\texttt{AGGC}} \choose \mbox{\texttt{TCCG}}}\\
\end{multline*}"></DIV>
<BR CLEAR="ALL">
<P><P></P>

<P>
(The same computation is performed for <SPAN CLASS="MATH"><IMG
 WIDTH="28" HEIGHT="14" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img14.png"
 ALT="$ \Delta S$"></SPAN>)   

<P>
However, some types of single bulge loop can't be only modelled with a NNN nearest 
 neighbor model and the following models can give more reliable and accurate results (mostly
 for RNA single bulge loops.)  

<P>

<H4><A NAME="SECTION000511020000000000000">
DNA duplexes :<SPAN  CLASS="textbf">Model from John Santalucia, Jr. and Donald Hicks, 2004</SPAN></A>
</H4> 

<P>
<P></P><DIV ALIGN="CENTER"><!-- MATH
 \begin{multline*}
\Delta h {(\mbox{single-bulge-loop})} =
\delta{}h_\mathrm{intervening-NN} +
\delta{}h_\mathrm{closing-AT-penalty}\\
\Delta s {(\mbox{single-bulge-loop})} =
\delta{}s_\mathrm{bulge-loop-of-1} +
\delta{}s_\mathrm{intervening-NN} +
\delta{}s_\mathrm{closing-AT-penalty}\\
\end{multline*}
 -->
<IMG
 WIDTH="553" HEIGHT="82" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img65.png"
 ALT="\begin{multline*}
\Delta h {(\mbox{single-bulge-loop})} =
\delta{}h_\mathrm{inte...
...athrm{intervening-NN} +
\delta{}s_\mathrm{closing-AT-penalty}\\
\end{multline*}"></DIV>
<BR CLEAR="ALL">
<P><P></P>

<P>
Where :

<P>
<!-- MATH
 $\delta{}h_\mathrm{bulge-loop-of-1}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="123" HEIGHT="30" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img66.png"
 ALT="$ \delta{}h_\mathrm{bulge-loop-of-1}$"></SPAN> accounts for the bulge loop of 1 nucleotide.

<P>
<!-- MATH
 $\delta{}h_\mathrm{intervening-NN}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="113" HEIGHT="30" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img67.png"
 ALT="$ \delta{}h_\mathrm{intervening-NN}$"></SPAN> accounts for the intervening base pair stack.

<P>
<!-- MATH
 $\delta{}h_\mathrm{closing-AT-penalty}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="139" HEIGHT="30" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img68.png"
 ALT="$ \delta{}h_\mathrm{closing-AT-penalty}$"></SPAN> accounts for each AT base pair adjacent
to the single bulge loop.

<P>
<SPAN  CLASS="textbf">Example</SPAN> :

<P>
<P></P><DIV ALIGN="CENTER"><!-- MATH
 \begin{multline*}
\Delta H {\mbox{\texttt{GAC}} \choose \mbox{\texttt{C-G}}} =
\Delta H {\mbox{\texttt{GC}} \choose \mbox{\texttt{CG}}} \\
\Delta S {\mbox{\texttt{GAC}} \choose \mbox{\texttt{C-G}}} =
\Delta S {\mbox{bulge-loop-of-1}} +
\Delta S {\mbox{\texttt{GC}} \choose \mbox{\texttt{CG}}} \\
\end{multline*}
 -->
<IMG
 WIDTH="411" HEIGHT="119" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img69.png"
 ALT="\begin{multline*}
\Delta H {\mbox{\texttt{GAC}} \choose \mbox{\texttt{C-G}}} =
\...
...} +
\Delta S {\mbox{\texttt{GC}} \choose \mbox{\texttt{CG}}} \\
\end{multline*}"></DIV>
<BR CLEAR="ALL">
<P><P></P>

<P>
(The same computation is performed for <SPAN CLASS="MATH"><IMG
 WIDTH="28" HEIGHT="14" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img14.png"
 ALT="$ \Delta S$"></SPAN>) 

<P>

<H4><A NAME="SECTION000511030000000000000">
RNA duplexes :<SPAN  CLASS="textbf">Model from Zhi Johm Lu, Douglas H. Turner and David H. Mathews, 2006</SPAN></A>
</H4> 

<P>
<P></P><DIV ALIGN="CENTER"><!-- MATH
 \begin{multline*}
\Delta h {(\mbox{single-bulge-loop})} =
\delta{}h_\mathrm{initiation-bulge-loop-of-1} +
\delta{}h_\mathrm{intervening-NN} \\
\end{multline*}
 -->
<IMG
 WIDTH="473" HEIGHT="58" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img70.png"
 ALT="\begin{multline*}
\Delta h {(\mbox{single-bulge-loop})} =
\delta{}h_\mathrm{initiation-bulge-loop-of-1} +
\delta{}h_\mathrm{intervening-NN} \\
\end{multline*}"></DIV>
<BR CLEAR="ALL">
<P><P></P>

<P>
Where :

<P>
<!-- MATH
 $\delta{}h_\mathrm{initiation-bulge-loop-of-1}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="184" HEIGHT="30" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img71.png"
 ALT="$ \delta{}h_\mathrm{initiation-bulge-loop-of-1}$"></SPAN> accounts for the initiation of  bulge loop of 1 nucleotide.

<P>
<!-- MATH
 $\delta{}h_\mathrm{intervening-NN}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="113" HEIGHT="30" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img67.png"
 ALT="$ \delta{}h_\mathrm{intervening-NN}$"></SPAN> accounts for the intervening base pair stack.

<P>
<SPAN  CLASS="textbf">Example</SPAN> :

<P>
<P></P><DIV ALIGN="CENTER"><!-- MATH
 \begin{multline*}
\Delta H {\mbox{\texttt{GAC}} \choose \mbox{\texttt{C-G}}} =
\Delta H {\mbox{initiation-bulge-loop-of-1}} +
\Delta H {\mbox{\texttt{GC}} \choose \mbox{\texttt{CG}}} \\
\end{multline*}
 -->
<IMG
 WIDTH="378" HEIGHT="75" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img72.png"
 ALT="\begin{multline*}
\Delta H {\mbox{\texttt{GAC}} \choose \mbox{\texttt{C-G}}} =
\...
...} +
\Delta H {\mbox{\texttt{GC}} \choose \mbox{\texttt{CG}}} \\
\end{multline*}"></DIV>
<BR CLEAR="ALL">
<P><P></P>

<P>
(The same computation is performed for <SPAN CLASS="MATH"><IMG
 WIDTH="28" HEIGHT="14" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img14.png"
 ALT="$ \Delta S$"></SPAN>) 
For further information, see the referenced articles.

<P>
<BR><P></P>
<DIV ALIGN="CENTER">
<TABLE CELLPADDING=3 BORDER="1">
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">model</SPAN></TH>
<TH ALIGN="CENTER"><SPAN  CLASS="textbf">limits</SPAN></TH>
<TH ALIGN="CENTER"><SPAN  CLASS="textbf">Article</SPAN></TH>
</TR>
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">tan04</SPAN></TH>
<TD ALIGN="CENTER">DNA</TD>
<TD ALIGN="CENTER">Tanaka Fumiaki et al (2004), Biochemistry 43 : 7143-7150</TD>
</TR>
<TR><TD ALIGN="CENTER">san04</TD>
<TD ALIGN="CENTER">DNA <br/> missing closing AT penalty</TD>
<TD ALIGN="CENTER">Santalucia et al (2004), Annu. Rev. Biophys. Biomol. Struct 33 : 415-440</TD>
</TR>
<TR><TD ALIGN="CENTER">ser07</TD>
<TD ALIGN="CENTER">RNA </br> less reliable results <br/> some missing parameters</TD>
<TD ALIGN="CENTER">Martin J Serra et al (2007), Biochemistry 46 : 15123-15135</TD>
</TR>
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">tur06</SPAN></TH>
<TD ALIGN="CENTER">RNA</TD>
<TD ALIGN="CENTER">Douglas M Turner et al (2006), Nucleic Acids Research 34: 4912-4924</TD>
</TR>
</TABLE>
</DIV><BR>

<P>

<DIV ALIGN="CENTER"><A NAME="2148"></A>
<TABLE>
<CAPTION ALIGN="BOTTOM"><STRONG>Figure:</STRONG>
Comparison of experimental and computed Tm for various sets of DNA sequences containing one single bulge loop. <!-- MATH
 $[\mbox{Na}^+] = 1$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">Na<IMG
 WIDTH="49" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img15.png"
 ALT="$ ^+] = 1$"></SPAN>&nbsp;M, <!-- MATH
 $[\mbox{nucleic acid}] = 1\cdot{}10^{-4}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">nucleic acid<IMG
 WIDTH="83" HEIGHT="34" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img18.png"
 ALT="$ ] = 1\cdot{}10^{-4}$"></SPAN>&nbsp;M</CAPTION>
<TR><TD><IMG
  WIDTH="585" HEIGHT="425" ALIGN="BOTTOM" BORDER="0"
 SRC="html/DNASingleBulgeLoop.png"
 ALT="Image DNASingleBulgeLoop"></TD></TR>
</TABLE>
</DIV>

<P>

<DIV ALIGN="CENTER"><A NAME="2149"></A>
<TABLE>
<CAPTION ALIGN="BOTTOM"><STRONG>Figure:</STRONG>
Comparison of experimental and computed Tm for various sets of RNA sequences containing one single bulge loop. <!-- MATH
 $[\mbox{Na}^+] = 1$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">Na<IMG
 WIDTH="49" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img15.png"
 ALT="$ ^+] = 1$"></SPAN>&nbsp;M, <!-- MATH
 $[\mbox{nucleic acid}] = 1\cdot{}10^{-4}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">nucleic acid<IMG
 WIDTH="83" HEIGHT="34" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img18.png"
 ALT="$ ] = 1\cdot{}10^{-4}$"></SPAN>&nbsp;M</CAPTION>
<TR><TD><IMG
  WIDTH="578" HEIGHT="354" ALIGN="BOTTOM" BORDER="0"
 SRC="html/RNASingleBulgeLoop.png"
 ALT="Image RNASingleBulgeLoop"></TD></TR>
</TABLE>
</DIV>

<P>

<H3><A NAME="SECTION000511100000000000000">
long bulge loop effect</A>
</H3>

<P>
The long bulge loops, that is all the adjacent unmatched internal nucleotides, can be taken into
account. , but all the thermodynamic parameters are not available. In such a case, 
the result is unpredictable, and all cases are possible. 
The RNA and DNA thermodynamic models are similar : 

<P>

<H4><A NAME="SECTION000511110000000000000">
DNA duplexes :<SPAN  CLASS="textbf">Model from John Santalucia, Jr. and Donald Hicks, 2004</SPAN></A>
</H4> 

<P>
<P></P><DIV ALIGN="CENTER"><!-- MATH
 \begin{multline*}
\Delta h {(\mbox{long-bulge-loop})} =
\delta{}h_\mathrm{closing-AT-penalty}\\
\Delta s {(\mbox{single-bulge-loop})} =
\delta{}s_\mathrm{bulge-loop-of-n} +
\delta{}s_\mathrm{closing-AT-penalty}\\
\end{multline*}
 -->
<IMG
 WIDTH="478" HEIGHT="82" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img73.png"
 ALT="\begin{multline*}
\Delta h {(\mbox{long-bulge-loop})} =
\delta{}h_\mathrm{closin...
...thrm{bulge-loop-of-n} +
\delta{}s_\mathrm{closing-AT-penalty}\\
\end{multline*}"></DIV>
<BR CLEAR="ALL">
<P><P></P>

<P>
Where :

<P>
<!-- MATH
 $\delta{}h_\mathrm{bulge-loop-of-n}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="123" HEIGHT="30" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img74.png"
 ALT="$ \delta{}h_\mathrm{bulge-loop-of-n}$"></SPAN> accounts for the bulge loop of n nucleotides.

<P>
<!-- MATH
 $\delta{}h_\mathrm{closing-AT-penalty}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="139" HEIGHT="30" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img68.png"
 ALT="$ \delta{}h_\mathrm{closing-AT-penalty}$"></SPAN> accounts for each AT base pair adjacent
to the long bulge loop.

<P>
<SPAN  CLASS="textbf">Example</SPAN> :

<P>
<P></P><DIV ALIGN="CENTER"><!-- MATH
 \begin{multline*}
\Delta H {\mbox{\texttt{GACGC}} \choose \mbox{\texttt{C- - -G}}} =
0
\Delta S {\mbox{\texttt{GACGC}} \choose \mbox{\texttt{C- - -G}}} =
\Delta S {\mbox{bulge-loop-of-3}} \\
\end{multline*}
 -->
<IMG
 WIDTH="367" HEIGHT="75" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img75.png"
 ALT="\begin{multline*}
\Delta H {\mbox{\texttt{GACGC}} \choose \mbox{\texttt{C---G}}}...
...e \mbox{\texttt{C---G}}} =
\Delta S {\mbox{bulge-loop-of-3}} \\
\end{multline*}"></DIV>
<BR CLEAR="ALL">
<P><P></P>

<P>

<H4><A NAME="SECTION000511120000000000000">
RNA duplexes : <SPAN  CLASS="textbf">Model from Zhi Johm Lu, Douglas H. Turner and David H. Mathews, 2006</SPAN></A>
</H4> 

<P>
<P></P><DIV ALIGN="CENTER"><!-- MATH
 \begin{multline*}
\Delta h {(\mbox{long-bulge-loop})} =
\delta{}h_\mathrm{initiation-bulge-loop-of-n} \\+
\delta{}h_\mathrm{per-AU/GU-penalty}\\
\end{multline*}
 -->
<IMG
 WIDTH="340" HEIGHT="82" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img76.png"
 ALT="\begin{multline*}
\Delta h {(\mbox{long-bulge-loop})} =
\delta{}h_\mathrm{initiation-bulge-loop-of-n} \\ +
\delta{}h_\mathrm{per-AU/GU-penalty}\\
\end{multline*}"></DIV>
<BR CLEAR="ALL">
<P><P></P>

<P>
Where :

<P>
<!-- MATH
 $\delta{}h_\mathrm{initiation-bulge-loop-of-n}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="185" HEIGHT="30" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img77.png"
 ALT="$ \delta{}h_\mathrm{initiation-bulge-loop-of-n}$"></SPAN> accounts for the initiation of the bulge loop of n nucleotides.

<P>
<!-- MATH
 $\delta{}h_\mathrm{per-AU/GU-penalty}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="146" HEIGHT="30" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img78.png"
 ALT="$ \delta{}h_\mathrm{per-AU/GU-penalty}$"></SPAN> accounts for each AU or GU base pair adjacent
to the long bulge loop.

<P>
<SPAN  CLASS="textbf">Example</SPAN> :

<P>
<P></P><DIV ALIGN="CENTER"><!-- MATH
 \begin{multline*}
\Delta H {\mbox{\texttt{AACGC}} \choose \mbox{\texttt{U- - -G}}} =
\Delta H {\mbox{initiation-bulge-loop-of-3}} +
1 \times \Delta H {\mbox{per-AU-penalty}} \\
\end{multline*}
 -->
<IMG
 WIDTH="484" HEIGHT="75" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img79.png"
 ALT="\begin{multline*}
\Delta H {\mbox{\texttt{AACGC}} \choose \mbox{\texttt{U---G}}}...
...ulge-loop-of-3}} +
1 \times \Delta H {\mbox{per-AU-penalty}} \\
\end{multline*}"></DIV>
<BR CLEAR="ALL">
<P><P></P>

<P>
(The same computation is performed for <SPAN CLASS="MATH"><IMG
 WIDTH="28" HEIGHT="14" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img14.png"
 ALT="$ \Delta S$"></SPAN>) 

<P>
For further information, see the referenced articles.

<P>
<BR><P></P>
<DIV ALIGN="CENTER">
<TABLE CELLPADDING=3 BORDER="1">
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">model</SPAN></TH>
<TH ALIGN="CENTER"><SPAN  CLASS="textbf">limits</SPAN></TH>
<TH ALIGN="CENTER"><SPAN  CLASS="textbf">Article</SPAN></TH>
</TR>
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">san04</SPAN></TH>
<TD ALIGN="CENTER">DNA <br/> missing closing AT penalty <br/> not tested with experimental results</TD>
<TD ALIGN="CENTER">Santalucia et al (2004), Annu. Rev. Biophys. Biomol. Struct 33 : 415-440</TD>
</TR>
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">tur06</SPAN></TH>
<TD ALIGN="CENTER">RNA <br/> not tested with experimental results</TD>
<TD ALIGN="CENTER">Douglas M Turner et al (2006), Nucleic Acids Research 34: 4912-4924</TD>
</TR>
</TABLE>
</DIV><BR>

<P>

<H3><A NAME="SECTION000511200000000000000">
Inosine bases effect</A>
</H3>

<P>
The inosine bases (I) are taken into account, but all the thermodynamic parameters are not available. 
In such a case, the result is unpredictable, and all cases are possible, so the
program quit with a warning. For the RNA duplexes, only the thermodynamic parameters
for IU base pairs are available for the moment.
This program computes the hybridisation enthalpy and entropy from the elementary 
parameters of each Crick's pair containing the inosine base.

<P>
<P><!-- MATH
 \begin{displaymath}
\begin{array}[t]{ccc}
  \Delta{}h_\mathrm{pattern-containing-inosine-bases}&=&\sum
  \delta{}h_\mathrm{Crick's pair-containing-inosine-bases}\\
  \end{array}
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="537" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img80.png"
 ALT="$\displaystyle \begin{array}[t]{ccc}
\Delta{}h_\mathrm{pattern-containing-inosin...
...&amp;=&amp;\sum
\delta{}h_\mathrm{Crick's pair-containing-inosine-bases}\\
\end{array}$">
</DIV><P></P>

<P>
<SPAN  CLASS="textbf">Examples</SPAN> : One inosine base
<P></P><DIV ALIGN="CENTER"><!-- MATH
 \begin{multline*}
\Delta H {\mbox{\texttt{AIC}} \choose \mbox{\texttt{TAG}}} =
\Delta H {\mbox{\texttt{AI}} \choose \mbox{\texttt{TA}}} +
\Delta H {\mbox{\texttt{IC}} \choose \mbox{\texttt{AG}}}\\
\end{multline*}
 -->
<IMG
 WIDTH="257" HEIGHT="75" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img81.png"
 ALT="\begin{multline*}
\Delta H {\mbox{\texttt{AIC}} \choose \mbox{\texttt{TAG}}} =
...
...}} +
\Delta H {\mbox{\texttt{IC}} \choose \mbox{\texttt{AG}}}\\
\end{multline*}"></DIV>
<BR CLEAR="ALL">
<P><P></P>

<P>
<SPAN  CLASS="textbf">Examples</SPAN> : Two adjacent base pairs containing inosine
<P></P><DIV ALIGN="CENTER"><!-- MATH
 \begin{multline*}
\Delta H {\mbox{\texttt{GIAC}} \choose \mbox{\texttt{CAIG}}} =
\Delta H {\mbox{\texttt{GI}} \choose \mbox{\texttt{CA}}} +
\Delta H {\mbox{\texttt{IA}} \choose \mbox{\texttt{AI}}} +
\Delta H {\mbox{\texttt{AC}} \choose \mbox{\texttt{IG}}}\\
\end{multline*}
 -->
<IMG
 WIDTH="356" HEIGHT="75" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img82.png"
 ALT="\begin{multline*}
\Delta H {\mbox{\texttt{GIAC}} \choose \mbox{\texttt{CAIG}}} =...
...}} +
\Delta H {\mbox{\texttt{AC}} \choose \mbox{\texttt{IG}}}\\
\end{multline*}"></DIV>
<BR CLEAR="ALL">
<P><P></P>

<P>
(The same computation is performed for <SPAN CLASS="MATH"><IMG
 WIDTH="28" HEIGHT="14" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img14.png"
 ALT="$ \Delta S$"></SPAN>) 

<P>
For further information, see the referenced articles.

<P>
<BR><P></P>
<DIV ALIGN="CENTER">
<TABLE CELLPADDING=3 BORDER="1">
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">model</SPAN></TH>
<TH ALIGN="CENTER"><SPAN  CLASS="textbf">limits</SPAN></TH>
<TH ALIGN="CENTER"><SPAN  CLASS="textbf">Article</SPAN></TH>
</TR>
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">san05</SPAN></TH>
<TD ALIGN="CENTER">DNA <br/>missing parameters for tandem base pairs containing inosine bases</TD>
<TD ALIGN="CENTER">Santalucia et al.(2005), Nucleic acids research 33 : 6258-6267</TD>
</TR>
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">zno07</SPAN></TH>
<TD ALIGN="CENTER">RNA <br/> only IU base pairs</TD>
<TD ALIGN="CENTER">Brent M Znosko et al. (2005), Biochemistry 46 : 4625-4634 </TD>
</TR>
</TABLE>
</DIV><BR>

<P>

<DIV ALIGN="CENTER"><A NAME="2178"></A>
<TABLE>
<CAPTION ALIGN="BOTTOM"><STRONG>Figure:</STRONG>
Comparison of experimental and computed Tm for various sets of
 DNA sequences containing inosine. <!-- MATH
 $[\mbox{Na}^+] = 1$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">Na<IMG
 WIDTH="49" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img15.png"
 ALT="$ ^+] = 1$"></SPAN>&nbsp;M, <!-- MATH
 $[\mbox{nucleic acid}] = 1\cdot{}10^{-4}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">nucleic acid<IMG
 WIDTH="83" HEIGHT="34" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img18.png"
 ALT="$ ] = 1\cdot{}10^{-4}$"></SPAN>&nbsp;M</CAPTION>
<TR><TD><IMG
  WIDTH="569" HEIGHT="335" ALIGN="BOTTOM" BORDER="0"
 SRC="html/DNAInosine.png"
 ALT="Image DNAInosine"></TD></TR>
</TABLE>
</DIV>

<P>

<DIV ALIGN="CENTER"><A NAME="2179"></A>
<TABLE>
<CAPTION ALIGN="BOTTOM"><STRONG>Figure:</STRONG>
Comparison of experimental and computed Tm for various sets of
 RNA sequences containing inosine. <!-- MATH
 $[\mbox{Na}^+] = 1$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">Na<IMG
 WIDTH="49" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img15.png"
 ALT="$ ^+] = 1$"></SPAN>&nbsp;M, <!-- MATH
 $[\mbox{nucleic acid}] = 1\cdot{}10^{-4}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">nucleic acid<IMG
 WIDTH="83" HEIGHT="34" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img18.png"
 ALT="$ ] = 1\cdot{}10^{-4}$"></SPAN>&nbsp;M</CAPTION>
<TR><TD><IMG
  WIDTH="569" HEIGHT="346" ALIGN="BOTTOM" BORDER="0"
 SRC="html/RNAInosine.png"
 ALT="Image RNAInosine"></TD></TR>
</TABLE>
</DIV>

<P>

<H3><A NAME="SECTION000511300000000000000">
Azobenzenes effect</A>
</H3>

<P>
The trans azobenzenes (X_T) and cis azobenzenes (X_C) in DNA duplexes are taken 
into account. Be aware : when the number of cis azobenzenes increases in the sequence, 
the predictions are less accurate and less reliable.

<P>
<P></P><DIV ALIGN="CENTER"><!-- MATH
 \begin{multline*}
\Delta{}h_\mathrm{pattern-containing-azobenzene} =
  \delta{}h_\mathrm{Crick's pair-containing-azobenzene+adjacent-base-pairs}\\
\end{multline*}
 -->
<IMG
 WIDTH="569" HEIGHT="58" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img83.png"
 ALT="\begin{multline*}
\Delta{}h_\mathrm{pattern-containing-azobenzene} =
\delta{}h_\mathrm{Crick's pair-containing-azobenzene+adjacent-base-pairs}\\
\end{multline*}"></DIV>
<BR CLEAR="ALL">
<P><P></P>

<P>
<SPAN  CLASS="textbf">Example</SPAN> :
If the duplex is :

<DIV CLASS="alltt" ALIGN="LEFT">
<TT>
 GCT<SPAN &nbsp;CLASS="textbf">X_C</SPAN>AGGC
<BR>CGATCCG
<BR></TT>
</DIV>

<P></P><DIV ALIGN="CENTER"><!-- MATH
 \begin{multline*}
\Delta H {\mbox{\texttt{GCTX\_CAGGC}} \choose \mbox{\texttt{CGATCCG}}} =
\Delta H {\mbox{perfectly-matching-sequence-1}} +
\Delta H {\mbox{azobenzene}} \\+
\Delta H {\mbox{perfectly-matching-sequence-2}}\\
\Delta H {\mbox{\texttt{GCTX\_CAGGC}} \choose \mbox{\texttt{CGATCCG}}} =
\Delta H {\mbox{\texttt{GCT}} \choose \mbox{\texttt{CGA}}} +
\Delta H {\mbox{\texttt{TX\_CA}} \choose \mbox{\texttt{AT}}} +
\Delta H {\mbox{\texttt{AGGC}} \choose \mbox{\texttt{TCCG}}}\\
\end{multline*}
 -->
<IMG
 WIDTH="514" HEIGHT="145" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img84.png"
 ALT="\begin{multline*}
\Delta H {\mbox{\texttt{GCTX\_CAGGC}} \choose \mbox{\texttt{CG...
...
\Delta H {\mbox{\texttt{AGGC}} \choose \mbox{\texttt{TCCG}}}\\
\end{multline*}"></DIV>
<BR CLEAR="ALL">
<P><P></P>

<P>
(The same computation is performed for <SPAN CLASS="MATH"><IMG
 WIDTH="28" HEIGHT="14" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img14.png"
 ALT="$ \Delta S$"></SPAN>) 

<P>
For further information, see the referenced articles.

<P>
<BR><P></P>
<DIV ALIGN="CENTER">
<TABLE CELLPADDING=3 BORDER="1">
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">model</SPAN></TH>
<TH ALIGN="CENTER"><SPAN  CLASS="textbf">limits</SPAN></TH>
<TH ALIGN="CENTER"><SPAN  CLASS="textbf">Article</SPAN></TH>
</TR>
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">asa05</SPAN></TH>
<TD ALIGN="CENTER">DNA <br/> less reliable results when the number of cis azobenzene increases</TD>
<TD ALIGN="CENTER">Asanuma et al. (2005), Nucleic acids Symposium Series 49 : 35-36</TD>
</TR>
</TABLE>
</DIV><BR>

<P>

<DIV ALIGN="CENTER"><A NAME="2193"></A>
<TABLE>
<CAPTION ALIGN="BOTTOM"><STRONG>Figure:</STRONG>
Comparison of experimental and computed Tm for various sets of
 DNA sequences containing azobenzene. <!-- MATH
 $[\mbox{Na}^+] = 1$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">Na<IMG
 WIDTH="49" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img15.png"
 ALT="$ ^+] = 1$"></SPAN>&nbsp;M, <!-- MATH
 $[\mbox{nucleic acid}] = 2\cdot{}10^{-6}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">nucleic acid<IMG
 WIDTH="83" HEIGHT="34" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img85.png"
 ALT="$ ] = 2\cdot{}10^{-6}$"></SPAN>&nbsp;M</CAPTION>
<TR><TD><IMG
  WIDTH="471" HEIGHT="339" ALIGN="BOTTOM" BORDER="0"
 SRC="html/Azobenzene.png"
 ALT="Image Azobenzene"></TD></TR>
</TABLE>
</DIV>

<P>

<H3><A NAME="SECTION000411400000000000000">
2-Hydroxyadenine bases effect</A>
</H3>

<P>
The 2-hydroxyadenine bases (A*) in DNA duplexes are taken into account, but only in this two different
sequence contexts : 5' GA*C 3' and 5' TA*A 3'. The program computes the enthalpy and the entropy in two
times : first it computes the enthalpy and entropy of the two Crick's pairs containing the hydroxyadenine
as if the base pair containing the hydroxyadenine was a simple AT base pair, and then it computes the hydroxyadenine
increments.

<P>
<P></P><DIV ALIGN="CENTER"><!-- MATH
 \begin{multline*}
\Delta{}h_\mathrm{pattern-containing-hydroxyadenine} =
\sum \delta{}h_\mathrm{Crick's pair-with-AT-base-pair} \\+
\delta{}h_\mathrm{hydroxyadenine-increment}
\end{multline*}
 -->
<IMG
 WIDTH="521" HEIGHT="63" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img86.png"
 ALT="\begin{multline*}
\Delta{}h_\mathrm{pattern-containing-hydroxyadenine} =
\sum \d...
...h-AT-base-pair} \\ +
\delta{}h_\mathrm{hydroxyadenine-increment}
\end{multline*}"></DIV>
<BR CLEAR="ALL">
<P><P></P>

<P>
<SPAN  CLASS="textbf">Examples</SPAN>
<P></P><DIV ALIGN="CENTER"><!-- MATH
 \begin{multline*}
\Delta H {\mbox{\texttt{GA*C}} \choose \mbox{\texttt{CCG}}} =
\Delta H {\mbox{\texttt{GA}} \choose \mbox{\texttt{CT}}} +
\Delta H {\mbox{\texttt{AC}} \choose \mbox{\texttt{CG}}} \\+
\Delta H {\mbox{increment-for-GA*C/CCG}}\\
\end{multline*}
 -->
<IMG
 WIDTH="370" HEIGHT="99" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img87.png"
 ALT="\begin{multline*}
\Delta H {\mbox{\texttt{GA*C}} \choose \mbox{\texttt{CCG}}} = ...
...x{\texttt{CG}}} \\ +
\Delta H {\mbox{increment-for-GA*C/CCG}}\\
\end{multline*}"></DIV>
<BR CLEAR="ALL">
<P><P></P>

<P>
(The same computation is performed for <SPAN CLASS="MATH"><IMG
 WIDTH="28" HEIGHT="14" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img14.png"
 ALT="$ \Delta S$"></SPAN>) 

<P>
For further information, see the referenced articles.

<P>
<BR><P></P>
<DIV ALIGN="CENTER">
<TABLE CELLPADDING=3 BORDER="1">
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">model</SPAN></TH>
<TH ALIGN="CENTER"><SPAN  CLASS="textbf">limits</SPAN></TH>
<TH ALIGN="CENTER"><SPAN  CLASS="textbf">Article</SPAN></TH>
</TR>
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">sug01</SPAN></TH>
<TD ALIGN="CENTER">DNA <br/> only 5' GA*C 3'and 5' TA*A 3' contexts</TD>
<TD ALIGN="CENTER">Sugimoto et al.(2001), Nucleic acids research 29 : 3289-3296</TD>
</TR>
</TABLE>
</DIV><BR>

<P>

<DIV ALIGN="CENTER"><A NAME="2201"></A>
<TABLE>
<CAPTION ALIGN="BOTTOM"><STRONG>Figure:</STRONG>
Comparison of experimental and computed Tm for various sets of
 DNA sequences containing hydroxyadenine. <!-- MATH
 $[\mbox{Na}^+] = 1$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">Na<IMG
 WIDTH="49" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img15.png"
 ALT="$ ^+] = 1$"></SPAN>&nbsp;M, <!-- MATH
 $[\mbox{nucleic acid}] = 1\cdot{}10^{-4}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">nucleic acid<IMG
 WIDTH="83" HEIGHT="34" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img18.png"
 ALT="$ ] = 1\cdot{}10^{-4}$"></SPAN>&nbsp;M</CAPTION>
<TR><TD><IMG
  WIDTH="410" HEIGHT="314" ALIGN="BOTTOM" BORDER="0"
 SRC="html/Hydroxyadenine.png"
 ALT="Image Hydroxyadenine"></TD></TR>
</TABLE>
</DIV>

<P>

<H3><A NAME="SECTION000511500000000000000">
Locked nucleic acids effect</A>
</H3>

<P>
The locked nucleic acids (AL, GL, CL, TL) in DNA duplexes are taken into account.
The program computes the enthalpy and the entropy in two times : first it computes the enthalpy and entropy of 
the two Crick's pairs containing the locked nucleic acid as if the locked nucleic acid was a simple nucleic acid,
and then it computes the locked nucleic acid increments for each Crick's base pair containing the locked nucleic acid.

<P>
<P></P><DIV ALIGN="CENTER"><!-- MATH
 \begin{multline*}
\Delta{}h_\mathrm{pattern-containing-Locked-Nucleic-Acid} =
\sum \delta{}h_\mathrm{Crick's pair-without-Locked-Nucleic-Acid} \\+
\sum \delta{}h_\mathrm{increment-Crick's pair-with-Locked-Nucleic-Acid}
\end{multline*}
 -->
<IMG
 WIDTH="578" HEIGHT="69" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img88.png"
 ALT="\begin{multline*}
\Delta{}h_\mathrm{pattern-containing-Locked-Nucleic-Acid} =
\s...
...elta{}h_\mathrm{increment-Crick's pair-with-Locked-Nucleic-Acid}
\end{multline*}"></DIV>
<BR CLEAR="ALL">
<P><P></P>

<P>
<SPAN  CLASS="textbf">Examples</SPAN>
<P></P><DIV ALIGN="CENTER"><!-- MATH
 \begin{multline*}
\Delta H {\mbox{\texttt{GALC}} \choose \mbox{\texttt{CTG}}} =
\Delta H {\mbox{\texttt{GA}} \choose \mbox{\texttt{CT}}} +
\Delta H {\mbox{\texttt{AC}} \choose \mbox{\texttt{CG}}} \\+
\Delta H {\mbox{increment-for-GAL/CT}} +
\Delta H {\mbox{increment-for-ALC/TG}}\\
\end{multline*}
 -->
<IMG
 WIDTH="458" HEIGHT="99" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img89.png"
 ALT="\begin{multline*}
\Delta H {\mbox{\texttt{GALC}} \choose \mbox{\texttt{CTG}}} = ...
...crement-for-GAL/CT}} +
\Delta H {\mbox{increment-for-ALC/TG}}\\
\end{multline*}"></DIV>
<BR CLEAR="ALL">
<P><P></P>

<P>
(The same computation is performed for <SPAN CLASS="MATH"><IMG
 WIDTH="28" HEIGHT="14" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img14.png"
 ALT="$ \Delta S$"></SPAN>) 

<P>
For further information, see the referenced articles.

<P>
<BR><P></P>
<DIV ALIGN="CENTER">
<TABLE CELLPADDING=3 BORDER="1">
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">model</SPAN></TH>
<TH ALIGN="CENTER"><SPAN  CLASS="textbf">limits</SPAN></TH>
<TH ALIGN="CENTER"><SPAN  CLASS="textbf">Article</SPAN></TH>
</TR>
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">mct04</SPAN></TH>
<TD ALIGN="CENTER">DNA</TD>
<TD ALIGN="CENTER">McTigue et al.(2004), Biochemistry 43 : 5388-5405</TD>
</TR>
</TABLE>
</DIV><BR>

<P>

<DIV ALIGN="CENTER"><A NAME="2210"></A>
<TABLE>
<CAPTION ALIGN="BOTTOM"><STRONG>Figure:</STRONG>
Comparison of experimental and computed Tm for various sets of
 DNA sequences containing Locked Nucleic Acids. <!-- MATH
 $[\mbox{Na}^+] = 1$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">Na<IMG
 WIDTH="49" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img15.png"
 ALT="$ ^+] = 1$"></SPAN>&nbsp;M, <!-- MATH
 $[\mbox{nucleic acid}] = 5\cdot{}10^{-6}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">nucleic acid<IMG
 WIDTH="83" HEIGHT="34" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img90.png"
 ALT="$ ] = 5\cdot{}10^{-6}$"></SPAN>&nbsp;M</CAPTION>
<TR><TD><IMG
  WIDTH="569" HEIGHT="397" ALIGN="BOTTOM" BORDER="0"
 SRC="html/LockedNucleicAcid.png"
 ALT="Image LockedNucleicAcid"></TD></TR>
</TABLE>
</DIV>

<P>

<H2><A NAME="SECTION00052000000000000000">
The melting temperature </A>
</H2>  

<P>
Then the melting temperature is computed by the following formula: 

<P>
<TABLE CELLPADDING=3>
<TR><TD ALIGN="RIGHT">Tm</TD>
<TD ALIGN="CENTER">=</TD>
<TD ALIGN="LEFT" VALIGN="TOP" WIDTH=115><!-- MATH
 \begin{math}
\frac{\Delta{}H}{\Delta{}S + R \ln (C_T/F)} - 273.15
\end{math}
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="167" HEIGHT="36" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img91.png"
 ALT="$ \frac{\Delta{}H}{\Delta{}S + R \ln (C_T/F)} - 273.15 $"></SPAN></TD>
<TD ALIGN="LEFT" VALIGN="TOP" WIDTH=158>&nbsp;</TD>
</TR>
<TR><TD ALIGN="RIGHT">&nbsp;</TD>
<TD ALIGN="CENTER">&nbsp;</TD>
<TD ALIGN="LEFT" VALIGN="TOP" WIDTH=115>&nbsp;</TD>
<TD ALIGN="LEFT" VALIGN="TOP" WIDTH=158>&nbsp;</TD>
</TR>
<TR><TD ALIGN="RIGHT">&nbsp;</TD>
<TD ALIGN="CENTER">&nbsp;</TD>
<TD ALIGN="LEFT" VALIGN="TOP" WIDTH=115><SMALL CLASS="FOOTNOTESIZE"><SPAN  CLASS="textit">Tm</SPAN> in K (for [Na<SPAN CLASS="MATH"><IMG
 WIDTH="15" HEIGHT="19" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img92.png"
 ALT="$ ^+$"></SPAN>] = 1&nbsp;M)
</SMALL></TD>
<TD ALIGN="LEFT" VALIGN="TOP" WIDTH=158>&nbsp;</TD>
</TR>
</TABLE>

<P>
In case of self complementary sequences, if the sequence (5' 3') is a sequence of type G(CNG)xC 
and x &gt; 4, the sequence mainly turns into hairpin loops and this program will compute the melting
temperature with this formula :

<P>
<TABLE CELLPADDING=3>
<TR><TD ALIGN="RIGHT">Tm</TD>
<TD ALIGN="CENTER">=</TD>
<TD ALIGN="LEFT" VALIGN="TOP" WIDTH=115><!-- MATH
 \begin{math}
\frac{\Delta{}H}{\Delta{}S} - 273.15
\end{math}
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="94" HEIGHT="36" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img93.png"
 ALT="$ \frac{\Delta{}H}{\Delta{}S} - 273.15 $"></SPAN></TD>
<TD ALIGN="LEFT" VALIGN="TOP" WIDTH=158>&nbsp;</TD>
</TR>
<TR><TD ALIGN="RIGHT">&nbsp;</TD>
<TD ALIGN="CENTER">&nbsp;</TD>
<TD ALIGN="LEFT" VALIGN="TOP" WIDTH=115>&nbsp;</TD>
<TD ALIGN="LEFT" VALIGN="TOP" WIDTH=158>&nbsp;</TD>
</TR>
<TR><TD ALIGN="RIGHT">&nbsp;</TD>
<TD ALIGN="CENTER">&nbsp;</TD>
<TD ALIGN="LEFT" VALIGN="TOP" WIDTH=115><SMALL CLASS="FOOTNOTESIZE"><SPAN  CLASS="textit">Tm</SPAN> in K 
</SMALL></TD>
<TD ALIGN="LEFT" VALIGN="TOP" WIDTH=158>&nbsp;</TD>
</TR>
</TABLE>

<P>
Moreover, no ion correction will be applied to this formula.

<P>

<H2><A NAME="SECTION00053000000000000000">
Correction for the concentration of nucleic acid </A>
</H2>  

<P>
<SPAN  CLASS="textit">F</SPAN> is 1 in the case of self-complementarity oligonucleotides. If the
ODNs are not self-complementary, <SPAN  CLASS="textit">F</SPAN> is 4 if both strands are present in
equivalent amount and <SPAN  CLASS="textit">F</SPAN> is 2 if one strand is in excess (for instance
in <SMALL>PCR</SMALL> experiments).  Actually in the latter case, the formula would
have to use the difference of concentrations rather than the total
concentration. But if the excess is sufficient, the total concentration can be
assumed to be identical to the concentration of the strand in excess. That is,
if one strand is in excess, the actual formula is effectively <!-- MATH
 $(C_{\mbox{max}} -
C_{\mbox{min}})/2$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="129" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img94.png"
 ALT="$ (C_{\mbox{max}} -
C_{\mbox{min}})/2$"></SPAN> but if <!-- MATH
 $C_{\mbox{max}} \gg C_{\mbox{min}}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="106" HEIGHT="30" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img95.png"
 ALT="$ C_{\mbox{max}} \gg C_{\mbox{min}}$"></SPAN>, <!-- MATH
 $C_{\mbox{max}}
- C_{\mbox{min}}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="100" HEIGHT="30" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img96.png"
 ALT="$ C_{\mbox{max}}
- C_{\mbox{min}}$"></SPAN> is close to the total concentration <SPAN CLASS="MATH"><IMG
 WIDTH="26" HEIGHT="30" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img97.png"
 ALT="$ C_T$"></SPAN>.  If <!-- MATH
 $C_{\mbox{max}}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="44" HEIGHT="30" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img98.png"
 ALT="$ C_{\mbox{max}}$"></SPAN> is close
to <!-- MATH
 $C_{\mbox{min}}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="41" HEIGHT="30" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img99.png"
 ALT="$ C_{\mbox{min}}$"></SPAN>, <!-- MATH
 $(C_{\mbox{max}} - C_{\mbox{min}})/2$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="129" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img94.png"
 ALT="$ (C_{\mbox{max}} -
C_{\mbox{min}})/2$"></SPAN> is equivalent to <SPAN CLASS="MATH"><IMG
 WIDTH="42" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img100.png"
 ALT="$ C_T/4$"></SPAN>, which is the default
correction.

<P>
<SPAN  CLASS="textit">F</SPAN> is 4 by default but note that <SMALL>MELTING</SMALL> can detect self complementary sequences 
for perfectly matching sequences even though there is(are) dangling end(s). In this case, the program will 
automatically change <SPAN  CLASS="textit">F</SPAN> to 1. In addition to that, the computation takes an entropic term to correct 
for self-complementarity.
In case of other self complementary sequences which doesn't match perfetcly, the option <SPAN  CLASS="textit">-self</SPAN> must be
used to inform the program of the self complementarity.

<P>

<DIV ALIGN="CENTER"><A NAME="2227"></A>
<TABLE>
<CAPTION ALIGN="BOTTOM"><STRONG>Figure:</STRONG>
Comparison of experimental and computed Tm for various sets of
 DNA self complementary sequences. <!-- MATH
 $[\mbox{Na}^+] = 1$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">Na<IMG
 WIDTH="49" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img15.png"
 ALT="$ ^+] = 1$"></SPAN>&nbsp;M, <!-- MATH
 $[\mbox{nucleic acid}] = 1\cdot{}10^{-4}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">nucleic acid<IMG
 WIDTH="83" HEIGHT="34" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img18.png"
 ALT="$ ] = 1\cdot{}10^{-4}$"></SPAN>&nbsp;M</CAPTION>
<TR><TD><IMG
  WIDTH="646" HEIGHT="369" ALIGN="BOTTOM" BORDER="0"
 SRC="html/DNASelfComplementarity.png"
 ALT="Image DNASelfComplementarity"></TD></TR>
</TABLE>
</DIV>

<P>

<DIV ALIGN="CENTER"><A NAME="2228"></A>
<TABLE>
<CAPTION ALIGN="BOTTOM"><STRONG>Figure:</STRONG>
Comparison of experimental and computed Tm for various sets of
 RNA self complementary sequences. <!-- MATH
 $[\mbox{Na}^+] = 1$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">Na<IMG
 WIDTH="49" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img15.png"
 ALT="$ ^+] = 1$"></SPAN>&nbsp;M, <!-- MATH
 $[\mbox{nucleic acid}] = 1\cdot{}10^{-4}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">nucleic acid<IMG
 WIDTH="83" HEIGHT="34" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img18.png"
 ALT="$ ] = 1\cdot{}10^{-4}$"></SPAN>&nbsp;M</CAPTION>
<TR><TD><IMG
  WIDTH="658" HEIGHT="471" ALIGN="BOTTOM" BORDER="0"
 SRC="html/RNASelfComplementarity.png"
 ALT="Image RNASelfComplementarity"></TD></TR>
</TABLE>
</DIV>

<P>

<H2><A NAME="SECTION00054000000000000000">
Correction for the concentration of cations</A>
</H2>  

<P>
After the program computed the melting temperature for <!-- MATH
 $[\mbox{Na}^+]$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">Na<IMG
 WIDTH="19" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img8.png"
 ALT="$ ^+]$"></SPAN>=1, an ion correction wille be applied
either directly on the computed melting temperature or on the computed entropy. In the last case, the melting
temperature is computed using the first formula of the <SPAN  CLASS="textit">Melting temperature</SPAN> section.
We must enter at least one of the following ion concentrations : <!-- MATH
 $[\mbox{Na}^+]$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">Na<IMG
 WIDTH="19" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img8.png"
 ALT="$ ^+]$"></SPAN>, <!-- MATH
 $[\mbox{K}^+]$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">K<IMG
 WIDTH="19" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img8.png"
 ALT="$ ^+]$"></SPAN>, <!-- MATH
 $[\mbox{Tris}^+]$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">Tris<IMG
 WIDTH="19" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img8.png"
 ALT="$ ^+]$"></SPAN> or
<!-- MATH
 $[\mbox{Mg}^{2+}]$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">Mg<IMG
 WIDTH="26" HEIGHT="34" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img101.png"
 ALT="$ ^{2+}]$"></SPAN> and several ion corrections are proposed  (see the reference table to have more information):

<P>

<H3><A NAME="SECTION00054100000000000000">
Sodium corrections</A>
</H3>
 
<UL>
<LI><SPAN  CLASS="textit">ahs01</SPAN>
 <P><!-- MATH
 \begin{displaymath}
\Delta{}S=\Delta{}S_{[\mbox{Na}^+]=1\;\mathrm{M}}+0.847 \times (N - 1) \times \log [\mbox{Na}^+]
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="347" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img102.png"
 ALT="$\displaystyle \Delta{}S=\Delta{}S_{[\mbox{Na}^+]=1\;\mathrm{M}}+0.847 \times (N - 1) \times \log [\mbox{Na}^+]
$">
</DIV><P></P>
Where <SPAN  CLASS="textit">N</SPAN> is the length of the duplex.
</LI>
<LI><SPAN  CLASS="textit">kam71</SPAN>
 <P><!-- MATH
 \begin{displaymath}
Tm=Tm_{[\mbox{Na}^+]=1\;\mathrm{M}}+(7.95 - 3.06 \times \Chi_{GC}) \times \ln [\mbox{Na}^+]
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="373" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img103.png"
 ALT="$\displaystyle Tm=Tm_{[\mbox{Na}^+]=1\;\mathrm{M}}+(7.95 - 3.06 \times \Chi_{GC}) \times \ln [\mbox{Na}^+]
$">
</DIV><P></P>
Where <SPAN  CLASS="textit"><SPAN CLASS="MATH"><IMG
 WIDTH="37" HEIGHT="31" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img104.png"
 ALT="$ \Chi_{GC}$"></SPAN></SPAN> is the frequence of GC base pairs in the duplex.
</LI>
<LI><SPAN  CLASS="textit">marschdot</SPAN>
 <P><!-- MATH
 \begin{displaymath}
Tm=Tm_{[\mbox{Na}^+]=1\;\mathrm{M}}+ (8.75 - 2.83 \times \Chi_{GC}) \times \ln [\mbox{Na}^+]
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="373" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img105.png"
 ALT="$\displaystyle Tm=Tm_{[\mbox{Na}^+]=1\;\mathrm{M}}+ (8.75 - 2.83 \times \Chi_{GC}) \times \ln [\mbox{Na}^+]
$">
</DIV><P></P>
Where <SPAN  CLASS="textit"><SPAN CLASS="MATH"><IMG
 WIDTH="37" HEIGHT="31" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img104.png"
 ALT="$ \Chi_{GC}$"></SPAN></SPAN> is the frequence of GC base pairs in the duplex.
</LI>
<LI><SPAN  CLASS="textit">owc1904</SPAN>
 <P><!-- MATH
 \begin{displaymath}
Tm=Tm_{[\mbox{Na}^+]=1\;\mathrm{M}}+ (-3.22 \times \Chi_{GC} - 6.39) \times \ln [\mbox{Na}^+]
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="386" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img106.png"
 ALT="$\displaystyle Tm=Tm_{[\mbox{Na}^+]=1\;\mathrm{M}}+ (-3.22 \times \Chi_{GC} - 6.39) \times \ln [\mbox{Na}^+]
$">
</DIV><P></P>
Where <SPAN  CLASS="textit"><SPAN CLASS="MATH"><IMG
 WIDTH="37" HEIGHT="31" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img104.png"
 ALT="$ \Chi_{GC}$"></SPAN></SPAN> is the frequence of GC base pairs in the duplex.
</LI>
<LI><SPAN  CLASS="textit">owc2004</SPAN>
 <P><!-- MATH
 \begin{displaymath}
\frac{1}{Tm}=\frac{1}{Tm_{[\mbox{Na}^+]=1\;\mathrm{M}}}+ (3.85 \times \Chi_{GC} - 6.18) \times \frac{1}{100000} \times \ln [\mbox{Na}^+]
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="452" HEIGHT="50" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img107.png"
 ALT="$\displaystyle \frac{1}{Tm}=\frac{1}{Tm_{[\mbox{Na}^+]=1\;\mathrm{M}}}+ (3.85 \times \Chi_{GC} - 6.18) \times \frac{1}{100000} \times \ln [\mbox{Na}^+]
$">
</DIV><P></P>
Where <SPAN  CLASS="textit"><SPAN CLASS="MATH"><IMG
 WIDTH="37" HEIGHT="31" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img104.png"
 ALT="$ \Chi_{GC}$"></SPAN></SPAN> is the frequence of GC base pairs in the duplex.
</LI>
<LI><SPAN  CLASS="textit">owc2104</SPAN>
 <P><!-- MATH
 \begin{displaymath}
Tm=Tm_{[\mbox{Na}^+]=1\;\mathrm{M}}+ (-4.62 \times \Chi_{GC} + 4.52) \times \ln [\mbox{Na}^+]
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="386" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img108.png"
 ALT="$\displaystyle Tm=Tm_{[\mbox{Na}^+]=1\;\mathrm{M}}+ (-4.62 \times \Chi_{GC} + 4.52) \times \ln [\mbox{Na}^+]
$">
</DIV><P></P>
 <P><!-- MATH
 \begin{displaymath}
- 0.985 \times \ln [(\mbox{Na}^+)^2]
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="96" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img109.png"
 ALT="$\displaystyle - 0.985 \times \ln [($">Na<IMG
 WIDTH="33" HEIGHT="37" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img110.png"
 ALT="$\displaystyle ^+)^2]
$">
</DIV><P></P>
Where <SPAN  CLASS="textit"><SPAN CLASS="MATH"><IMG
 WIDTH="37" HEIGHT="31" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img104.png"
 ALT="$ \Chi_{GC}$"></SPAN></SPAN> is the frequence of GC base pairs in the duplex.
</LI>
<LI><SPAN  CLASS="textit">owc2204</SPAN>
 <P></P><DIV ALIGN="CENTER"><!-- MATH
 \begin{multline*}
\frac{1}{Tm}=\frac{1}{Tm_{[\mbox{Na}^+]=1\;\mathrm{M}}}+ (4.29 \times \Chi_{GC} - 3.95) \times
\end{multline*}
 -->
<IMG
 WIDTH="520" HEIGHT="52" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img111.png"
 ALT="\begin{multline*}
\frac{1}{Tm}=\frac{1}{Tm_{[\mbox{Na}^+]=1\;\mathrm{M}}}+ (4.29 \times \Chi_{GC} - 3.95) \times
\end{multline*}"></DIV>
<BR CLEAR="ALL">
<P><P></P>
 <P></P><DIV ALIGN="CENTER"><!-- MATH
 \begin{multline*}
\frac{1}{100000} \times \ln [\mbox{Na}^+] +
 9.40 \times \frac{1}{1000000} \times \ln [\mbox{Na}^+]
\end{multline*}
 -->
<IMG
 WIDTH="521" HEIGHT="43" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img112.png"
 ALT="\begin{multline*}
\frac{1}{100000} \times \ln [\mbox{Na}^+] +
9.40 \times \frac{1}{1000000} \times \ln [\mbox{Na}^+]
\end{multline*}"></DIV>
<BR CLEAR="ALL">
<P><P></P>
Where <SPAN  CLASS="textit"><SPAN CLASS="MATH"><IMG
 WIDTH="37" HEIGHT="31" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img104.png"
 ALT="$ \Chi_{GC}$"></SPAN></SPAN> is the frequence of GC base pairs in the duplex.
</LI>
<LI><SPAN  CLASS="textit">san96</SPAN>
 <P><!-- MATH
 \begin{displaymath}
12.5  \log [\mbox{Na}^+]
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="60" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img113.png"
 ALT="$\displaystyle 12.5 \log [$">Na<IMG
 WIDTH="19" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img114.png"
 ALT="$\displaystyle ^+]
$">
</DIV><P></P>
</LI>
<LI><SPAN  CLASS="textit">san04</SPAN>
 <P><!-- MATH
 \begin{displaymath}
\Delta{}S=\Delta{}S_{[\mbox{Na}^+]=1\;\mathrm{M}}+ 0.368 * (N - 1) \times \ln [\mbox{Na}^+]
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="335" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img115.png"
 ALT="$\displaystyle \Delta{}S=\Delta{}S_{[\mbox{Na}^+]=1\;\mathrm{M}}+ 0.368 * (N - 1) \times \ln [\mbox{Na}^+]
$">
</DIV><P></P>
Where <SPAN  CLASS="textit">N</SPAN> is the length of the duplex.
</LI>
<LI><SPAN  CLASS="textit">schlif</SPAN>
 <P><!-- MATH
 \begin{displaymath}
Tm=Tm_{[\mbox{Na}^+]=1\;\mathrm{M}}+ 16.6 \times \log [\mbox{Na}^+]
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="269" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img116.png"
 ALT="$\displaystyle Tm=Tm_{[\mbox{Na}^+]=1\;\mathrm{M}}+ 16.6 \times \log [\mbox{Na}^+]
$">
</DIV><P></P>
</LI>
<LI><SPAN  CLASS="textit">tanna06</SPAN>
 <P><!-- MATH
 \begin{displaymath}
\Delta{}S=\Delta{}S_{[\mbox{Na}^+]=1\;\mathrm{M}}- 3.22 \times (N - 1) \times g1
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="296" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img117.png"
 ALT="$\displaystyle \Delta{}S=\Delta{}S_{[\mbox{Na}^+]=1\;\mathrm{M}}- 3.22 \times (N - 1) \times g1
$">
</DIV><P></P>
Where <SPAN  CLASS="textit">N</SPAN> is the length of the duplex.
 <P><!-- MATH
 \begin{displaymath}
g1=a1 + \frac{b1}{N}
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="96" HEIGHT="52" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img118.png"
 ALT="$\displaystyle g1=a1 + \frac{b1}{N}
$">
</DIV><P></P>
 <P><!-- MATH
 \begin{displaymath}
a1= -0.07 \times \ln [\mbox{Na}^+] + 0.012 \times (\ln [\mbox{Mg}^{2+}])^2
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="120" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img119.png"
 ALT="$\displaystyle a1= -0.07 \times \ln [$">Na<IMG
 WIDTH="118" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img120.png"
 ALT="$\displaystyle ^+] + 0.012 \times (\ln [$">Mg<IMG
 WIDTH="39" HEIGHT="37" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img121.png"
 ALT="$\displaystyle ^{2+}])^2
$">
</DIV><P></P>
 <P><!-- MATH
 \begin{displaymath}
b1= 0.013 \times (\ln [\mbox{Mg}^{2+}])^2
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="120" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img122.png"
 ALT="$\displaystyle b1= 0.013 \times (\ln [$">Mg<IMG
 WIDTH="39" HEIGHT="37" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img121.png"
 ALT="$\displaystyle ^{2+}])^2
$">
</DIV><P></P>
item <SPAN  CLASS="textit">tanna07</SPAN>
 <P><!-- MATH
 \begin{displaymath}
\Delta{}S=\Delta{}S_{[\mbox{Na}^+]=1\;\mathrm{M}}- 3.22 \times (N - 1) \times g1
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="296" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img117.png"
 ALT="$\displaystyle \Delta{}S=\Delta{}S_{[\mbox{Na}^+]=1\;\mathrm{M}}- 3.22 \times (N - 1) \times g1
$">
</DIV><P></P>
Where <SPAN  CLASS="textit">N</SPAN> is the length of the duplex.
 <P><!-- MATH
 \begin{displaymath}
g1=a1 + \frac{b1}{N}
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="96" HEIGHT="52" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img118.png"
 ALT="$\displaystyle g1=a1 + \frac{b1}{N}
$">
</DIV><P></P>
 <P><!-- MATH
 \begin{displaymath}
a1= -0.075 \times \ln [\mbox{Na}^+] + 0.012 \times (\ln [\mbox{Mg}^{2+}])^2
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="128" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img123.png"
 ALT="$\displaystyle a1= -0.075 \times \ln [$">Na<IMG
 WIDTH="118" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img120.png"
 ALT="$\displaystyle ^+] + 0.012 \times (\ln [$">Mg<IMG
 WIDTH="39" HEIGHT="37" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img121.png"
 ALT="$\displaystyle ^{2+}])^2
$">
</DIV><P></P>
 <P><!-- MATH
 \begin{displaymath}
b1= 0.018 \times (\ln [\mbox{Mg}^{2+}])^2
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="120" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img124.png"
 ALT="$\displaystyle b1= 0.018 \times (\ln [$">Mg<IMG
 WIDTH="39" HEIGHT="37" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img121.png"
 ALT="$\displaystyle ^{2+}])^2
$">
</DIV><P></P>
</LI>
<LI><SPAN  CLASS="textit">wet91</SPAN>
 <P><!-- MATH
 \begin{displaymath}
Tm=Tm_{[\mbox{Na}^+]=1\;\mathrm{M}}+ 16.6  \log \frac{[\mbox{Na}^+]}{1 + 0.7 [\mbox{Na}^+]} + 3.85
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="354" HEIGHT="56" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img125.png"
 ALT="$\displaystyle Tm=Tm_{[\mbox{Na}^+]=1\;\mathrm{M}}+ 16.6 \log \frac{[\mbox{Na}^+]}{1 + 0.7 [\mbox{Na}^+]} + 3.85
$">
</DIV><P></P>
</LI>
</UL>

<P>
<BR><P></P>
<DIV ALIGN="CENTER">
<TABLE CELLPADDING=3 BORDER="1">
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">correction</SPAN></TH>
<TH ALIGN="CENTER"><SPAN  CLASS="textbf">contexts</SPAN></TH>
<TH ALIGN="CENTER"><SPAN  CLASS="textbf">Article</SPAN></TH>
</TR>
<TR><TD ALIGN="CENTER">ahs01</TD>
<TD ALIGN="CENTER">DNA <br/> Na&gt;0</TD>
<TD ALIGN="CENTER">Nicolas Von Ahsen et al ,2001, Clinical Chemistry, 47, 1956-1961</TD>
</TR>
<TR><TD ALIGN="CENTER">kam71</TD>
<TD ALIGN="CENTER">DNA <br/> Na&gt;0 <br/>Na&gt;=0.069 <br/> Na&lt;=1.02</TD>
<TD ALIGN="CENTER">Frank-Kamenetskii et al. 1971, Biopolymers 10, 2623-2624</TD>
</TR>
<TR><TD ALIGN="CENTER">marschdot</TD>
<TD ALIGN="CENTER">DNA <br/> Na&gt;=0.069 <br/> Na&lt;=1.02</TD>
<TD ALIGN="CENTER">Marmur, J., and Doty, P. (1962), J. Mol. Biol. 5, 109-118 <br/> Blake and Delcourt. (1998) Nucleic Acids Res. 26, 3323-3332 and corrigendum</TD>
</TR>
<TR><TD ALIGN="CENTER">owc1904</TD>
<TD ALIGN="CENTER">DNA <br/> Na&gt;0</TD>
<TD ALIGN="CENTER">Richard Owczarzy et al.,2004, Biochemistry,43, 3537-3554</TD>
</TR>
<TR><TD ALIGN="CENTER">owc2004</TD>
<TD ALIGN="CENTER">DNA <br/> Na&gt;0</TD>
<TD ALIGN="CENTER">Richard Owczarzy et al., 2004, Biochemistry, 43, 3537-3554</TD>
</TR>
<TR><TD ALIGN="CENTER">owc2104</TD>
<TD ALIGN="CENTER">DNA <br/> Na&gt;0</TD>
<TD ALIGN="CENTER">Richard Owczarzy et al., 2004, Biochemistry, 43, 3537-3554</TD>
</TR>
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">owc2204</SPAN></TH>
<TD ALIGN="CENTER">DNA <br/> Na&gt;0</TD>
<TD ALIGN="CENTER">Richard Owczarzy et al., 2004, Biochemistry, 43, 3537-3554</TD>
</TR>
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">owc2204</SPAN></TH>
<TD ALIGN="CENTER">DNA <br/> Na&gt;0</TD>
<TD ALIGN="CENTER">Richard Owczarzy et al., 2004, Biochemistry, 43, 3537-3554</TD>
</TR>
<TR><TD ALIGN="CENTER">san96</TD>
<TD ALIGN="CENTER">DNA <br/> Na&gt;=0.1</TD>
<TD ALIGN="CENTER">SantaLucia et al.(1996), Biochemistry 35 : 3555-3562</TD>
</TR>
<TR><TD ALIGN="CENTER">san04</TD>
<TD ALIGN="CENTER">DNA <br/> Na&gt;=0.05 <br/> Na&lt;=1.1 <br/> oligonucleotides inferior to 16 bases</TD>
<TD ALIGN="CENTER">Santalucia et al (2004), Annu. Rev. Biophys. Biomol. Struct 33 : 415-440 <br/> John Santalucia, Jr., 1998,Proc. Natl. Acad. Sci. USA, 95, 1460-1465 </TD>
</TR>
<TR><TD ALIGN="CENTER">schlif</TD>
<TD ALIGN="CENTER">DNA <br/> Na&gt;=0.07 <br/> Na&lt;=0.12</TD>
<TD ALIGN="CENTER">Schildkraut, C., and Lifson, S. (1965), Biopolymers 3, 195-208</TD>
</TR>
<TR><TD ALIGN="CENTER">tanna06</TD>
<TD ALIGN="CENTER">DNA <br/> Na&gt;=0.001 <br/> Na&lt;=1</TD>
<TD ALIGN="CENTER">Zhi-Jie Tan et al. 2006, Biophysical Journal, 90, 1175-1190</TD>
</TR>
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">tanna07</SPAN></TH>
<TD ALIGN="CENTER">RNA <br/> Na&gt;=0.003 <br/> Na&lt;=1</TD>
<TD ALIGN="CENTER">Zhi-Jie Tan et al, 2007, Biophysical Journal, 92, 3615-3632</TD>
</TR>
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">wet91</SPAN></TH>
<TD ALIGN="CENTER">RNA, DNA and RNA/DNA <br/> Na&gt;0</TD>
<TD ALIGN="CENTER">James G. Wetmur 1991, Critical reviews in biochemistry and molecular, biology, 26, 227-259</TD>
</TR>
</TABLE>
</DIV><BR>

<P>

<H3><A NAME="SECTION00054200000000000000">
Magnesium corrections</A>
</H3>
  
<UL>
<LI><SPAN  CLASS="textit">owcmg08</SPAN>
 <P><!-- MATH
 \begin{displaymath}
\frac{1}{Tm_{[\mbox{Mg}^{2+}]}} = \frac{1}{Tm_{[\mbox{Na}^+]=1\;\mathrm{M}}} + a
 - b (\ln [\mbox{Mg}^{2+}]) + \Chi_{GC} (c + d \ln [\mbox{Mg}^{2+}]) + \frac{1}{2 (Nbp-1)}
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="583" HEIGHT="50" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img126.png"
 ALT="$\displaystyle \frac{1}{Tm_{[\mbox{Mg}^{2+}]}} = \frac{1}{Tm_{[\mbox{Na}^+]=1\;\...
...box{Mg}^{2+}]) + \Chi_{GC} (c + d \ln [\mbox{Mg}^{2+}]) + \frac{1}{2 (Nbp-1)}
$">
</DIV><P></P>
 <P><!-- MATH
 \begin{displaymath}
(-e + f \ln [\mbox{Mg}^{2+}] + g (\ln [\mbox{Mg}^{2+}])^{2}])
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="80" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img127.png"
 ALT="$\displaystyle (-e + f \ln [$">Mg<IMG
 WIDTH="77" HEIGHT="37" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img128.png"
 ALT="$\displaystyle ^{2+}] + g (\ln [$">Mg<IMG
 WIDTH="50" HEIGHT="37" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img129.png"
 ALT="$\displaystyle ^{2+}])^{2}])
$">
</DIV><P></P>
Where <SPAN  CLASS="textit"><SPAN CLASS="MATH"><IMG
 WIDTH="37" HEIGHT="31" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img104.png"
 ALT="$ \Chi_{GC}$"></SPAN></SPAN> is the frequence of GC base pairs in the duplex.
 <SPAN  CLASS="textit">Nbp</SPAN> is the number of base pairs
 and a, b, c, d, e, f, g fixed to :

<P>
a = 3.92 &#215;<SPAN CLASS="MATH"><IMG
 WIDTH="37" HEIGHT="16" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img130.png"
 ALT="$ 10^{-5}$"></SPAN> 
<BR> b = 9.11 &#215;<SPAN CLASS="MATH"><IMG
 WIDTH="37" HEIGHT="16" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img131.png"
 ALT="$ 10^{-6}$"></SPAN> 
<BR> c = 6.26 &#215;<SPAN CLASS="MATH"><IMG
 WIDTH="37" HEIGHT="16" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img130.png"
 ALT="$ 10^{-5}$"></SPAN> 
<BR> d = 1.42 &#215;<SPAN CLASS="MATH"><IMG
 WIDTH="37" HEIGHT="16" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img130.png"
 ALT="$ 10^{-5}$"></SPAN> 
<BR> e = 4.82 &#215;<SPAN CLASS="MATH"><IMG
 WIDTH="37" HEIGHT="16" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img132.png"
 ALT="$ 10^{-4}$"></SPAN> 
<BR> f = 5.25 &#215;<SPAN CLASS="MATH"><IMG
 WIDTH="37" HEIGHT="16" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img132.png"
 ALT="$ 10^{-4}$"></SPAN> 
<BR> g = 8.31 &#215;<SPAN CLASS="MATH"><IMG
 WIDTH="37" HEIGHT="16" ALIGN="BOTTOM" BORDER="0"
 SRC="html/img130.png"
 ALT="$ 10^{-5}$"></SPAN>. 
<BR></LI>
<LI><SPAN  CLASS="textit">tanmg06</SPAN>
 <P><!-- MATH
 \begin{displaymath}
\Delta{}S=\Delta{}S_{[\mbox{Na}^+]=1\;\mathrm{M}}- 3.22 \times (N - 1) \times g2
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="296" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img133.png"
 ALT="$\displaystyle \Delta{}S=\Delta{}S_{[\mbox{Na}^+]=1\;\mathrm{M}}- 3.22 \times (N - 1) \times g2
$">
</DIV><P></P>
Where <SPAN  CLASS="textit">N</SPAN> is the length of the duplex.
 <P><!-- MATH
 \begin{displaymath}
g2=a2 + \frac{b2}{(N)^2}
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="115" HEIGHT="52" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img134.png"
 ALT="$\displaystyle g2=a2 + \frac{b2}{(N)^2}
$">
</DIV><P></P>
 <P><!-- MATH
 \begin{displaymath}
a2= 0.02 \times \ln [\mbox{Mg}^{2+}] + 0.0068 \times (\ln [\mbox{Mg}^{2+}])^2
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="107" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img135.png"
 ALT="$\displaystyle a2= 0.02 \times \ln [$">Mg<IMG
 WIDTH="133" HEIGHT="37" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img136.png"
 ALT="$\displaystyle ^{2+}] + 0.0068 \times (\ln [$">Mg<IMG
 WIDTH="39" HEIGHT="37" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img121.png"
 ALT="$\displaystyle ^{2+}])^2
$">
</DIV><P></P>
 <P><!-- MATH
 \begin{displaymath}
b2= 1.18 \times \ln [\mbox{Mg}^{2+}] + 0.344 \times (\ln [\mbox{Mg}^{2+}])^2
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="106" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img137.png"
 ALT="$\displaystyle b2= 1.18 \times \ln [$">Mg<IMG
 WIDTH="125" HEIGHT="37" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img138.png"
 ALT="$\displaystyle ^{2+}] + 0.344 \times (\ln [$">Mg<IMG
 WIDTH="39" HEIGHT="37" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img121.png"
 ALT="$\displaystyle ^{2+}])^2
$">
</DIV><P></P>
item <SPAN  CLASS="textit">tanmg07</SPAN>
 <P><!-- MATH
 \begin{displaymath}
\Delta{}S=\Delta{}S_{[\mbox{Na}^+]=1\;\mathrm{M}}- 3.22 \times (N - 1) \times g2
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="296" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img133.png"
 ALT="$\displaystyle \Delta{}S=\Delta{}S_{[\mbox{Na}^+]=1\;\mathrm{M}}- 3.22 \times (N - 1) \times g2
$">
</DIV><P></P>
Where <SPAN  CLASS="textit">N</SPAN> is the length of the duplex.
 <P><!-- MATH
 \begin{displaymath}
g2=a2 + \frac{b2}{(N)^2}
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="115" HEIGHT="52" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img134.png"
 ALT="$\displaystyle g2=a2 + \frac{b2}{(N)^2}
$">
</DIV><P></P>
 <P><!-- MATH
 \begin{displaymath}
a2= \frac{-0.6}{N} + 0.025 \times \ln [\mbox{Mg}^{2+}] + 0.0068 \times (\ln [\mbox{Mg}^{2+}])^2
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="171" HEIGHT="50" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img139.png"
 ALT="$\displaystyle a2= \frac{-0.6}{N} + 0.025 \times \ln [$">Mg<IMG
 WIDTH="133" HEIGHT="37" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img136.png"
 ALT="$\displaystyle ^{2+}] + 0.0068 \times (\ln [$">Mg<IMG
 WIDTH="39" HEIGHT="37" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img121.png"
 ALT="$\displaystyle ^{2+}])^2
$">
</DIV><P></P>
 <P><!-- MATH
 \begin{displaymath}
b2= \ln [\mbox{Mg}^{2+} + 0.38 \times (\ln [\mbox{Mg}^{2+}])^2
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="58" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img140.png"
 ALT="$\displaystyle b2= \ln [$">Mg<IMG
 WIDTH="112" HEIGHT="37" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img141.png"
 ALT="$\displaystyle ^{2+} + 0.38 \times (\ln [$">Mg<IMG
 WIDTH="39" HEIGHT="37" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img121.png"
 ALT="$\displaystyle ^{2+}])^2
$">
</DIV><P></P>
</LI>
</UL> 

<P>
<BR><P></P>
<DIV ALIGN="CENTER">
<TABLE CELLPADDING=3 BORDER="1">
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">correction</SPAN></TH>
<TH ALIGN="CENTER"><SPAN  CLASS="textbf">limits</SPAN></TH>
<TH ALIGN="CENTER"><SPAN  CLASS="textbf">Article</SPAN></TH>
</TR>
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">oxcmg08</SPAN></TH>
<TD ALIGN="CENTER">DNA <br/> Mg&gt;=0.0005 <br/> Mg&lt;=0.6</TD>
<TD ALIGN="CENTER">Richard Owczarzy et al.,2008, Biochemistry, 47, 5336-5353</TD>
</TR>
<TR><TD ALIGN="CENTER">tanmg06</TD>
<TD ALIGN="CENTER">DNA <br/> Mg&gt;=0.0001 <br/> Mg&lt;=1 <br/> oligomer length superior to 6 base pairs</TD>
<TD ALIGN="CENTER">Zhi-Jie Tan et al. 2006, Biophysical Journal, 90, 1175-1190</TD>
</TR>
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">tanmg07</SPAN></TH>
<TD ALIGN="CENTER">RNA <br/> Mg&gt;=0.1 <br/> Mg&lt;=0.3</TD>
<TD ALIGN="CENTER">Zhi-Jie Tan et al, 2007, Biophysical Journal, 92, 3615-3632</TD>
</TR>
</TABLE>
</DIV><BR>

<P>

<H3><A NAME="SECTION00054300000000000000">
Mixed Na Mg corrections</A>
</H3> 
 
<UL>
<LI><SPAN  CLASS="textit">owcmix08</SPAN>
 <P><!-- MATH
 \begin{displaymath}
\frac{1}{Tm_{[\mbox{Mg}^{2+}]}} = \frac{1}{Tm_{[\mbox{Na}^+]=1\;\mathrm{M}}} + a
 - b (\ln [\mbox{Mg}^{2+}]) + \Chi_{GC} (c + d \ln [\mbox{Mg}^{2+}]) + \frac{1}{2 (Nbp-1)}
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="583" HEIGHT="50" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img126.png"
 ALT="$\displaystyle \frac{1}{Tm_{[\mbox{Mg}^{2+}]}} = \frac{1}{Tm_{[\mbox{Na}^+]=1\;\...
...box{Mg}^{2+}]) + \Chi_{GC} (c + d \ln [\mbox{Mg}^{2+}]) + \frac{1}{2 (Nbp-1)}
$">
</DIV><P></P>
 <P><!-- MATH
 \begin{displaymath}
(-e + f \ln [\mbox{Mg}^{2+}] + g (\ln [\mbox{Mg}^{2+}])^{2}])
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="80" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img127.png"
 ALT="$\displaystyle (-e + f \ln [$">Mg<IMG
 WIDTH="77" HEIGHT="37" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img128.png"
 ALT="$\displaystyle ^{2+}] + g (\ln [$">Mg<IMG
 WIDTH="50" HEIGHT="37" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img129.png"
 ALT="$\displaystyle ^{2+}])^{2}])
$">
</DIV><P></P>
Where <SPAN  CLASS="textit"><SPAN CLASS="MATH"><IMG
 WIDTH="37" HEIGHT="31" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img104.png"
 ALT="$ \Chi_{GC}$"></SPAN></SPAN> is the frequence of GC base pairs in the duplex.
 <SPAN  CLASS="textit">Nbp</SPAN> is the number of base pairs.
 b, c, e, f are fixed as in the magnesium correction owcmg08.
 <P><!-- MATH
 \begin{displaymath}
a = 3.92\cdot{}10^{-5} (0.843 - 0.352 [\mbox{Mon}^+]^{0.5} \ln [\mbox{Mon}^+])
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="209" HEIGHT="37" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img142.png"
 ALT="$\displaystyle a = 3.92\cdot{}10^{-5} (0.843 - 0.352 [$">Mon<IMG
 WIDTH="57" HEIGHT="37" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img143.png"
 ALT="$\displaystyle ^+]^{0.5} \ln [$">Mon<IMG
 WIDTH="26" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img144.png"
 ALT="$\displaystyle ^+])
$">
</DIV><P></P>
 <P><!-- MATH
 \begin{displaymath}
d = 1.42\cdot{}10^{-5} (1.279 - 4.03\cdot{}10^{-3} \ln [\mbox{Mon}^+] -
 8.03\cdot{}10^{-3} \ln [\mbox{Mon}^+]^{2})
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="262" HEIGHT="37" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img145.png"
 ALT="$\displaystyle d = 1.42\cdot{}10^{-5} (1.279 - 4.03\cdot{}10^{-3} \ln [$">Mon<IMG
 WIDTH="132" HEIGHT="37" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img146.png"
 ALT="$\displaystyle ^+] -
8.03\cdot{}10^{-3} \ln [$">Mon<IMG
 WIDTH="33" HEIGHT="37" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img147.png"
 ALT="$\displaystyle ^+]^{2})
$">
</DIV><P></P>
 <P><!-- MATH
 \begin{displaymath}
g = 8.31\cdot{}10^{-5} (0.486 - 0.258 \ln [\mbox{Mon}^+] + 5.25\cdot{}10^{-3}
 \ln [\mbox{Mon}^+]^{3}
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="225" HEIGHT="37" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img148.png"
 ALT="$\displaystyle g = 8.31\cdot{}10^{-5} (0.486 - 0.258 \ln [$">Mon<IMG
 WIDTH="132" HEIGHT="37" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img149.png"
 ALT="$\displaystyle ^+] + 5.25\cdot{}10^{-3}
\ln [$">Mon<IMG
 WIDTH="27" HEIGHT="37" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img150.png"
 ALT="$\displaystyle ^+]^{3}
$">
</DIV><P></P>
</LI>
<LI><SPAN  CLASS="textit">tanmix07</SPAN>
 <P><!-- MATH
 \begin{displaymath}
\Delta{}S=\Delta{}S_{[\mbox{Na}^+]=1\;\mathrm{M}}- 3.22 \times ((N - 1) \times (x1 \times g1 + x2 \times g2) + g12))
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="479" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img151.png"
 ALT="$\displaystyle \Delta{}S=\Delta{}S_{[\mbox{Na}^+]=1\;\mathrm{M}}- 3.22 \times ((N - 1) \times (x1 \times g1 + x2 \times g2) + g12))
$">
</DIV><P></P>
Where <SPAN  CLASS="textit">N</SPAN> is the length of the duplex.
 <P><!-- MATH
 \begin{displaymath}
g12 = -0.6 \times x1 \times x2 \times \ln [\mbox{Na}^+] \times \frac{\ln [(\frac{1}{x1} - 1) \times \mbox{Na}^+]}{N}
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="193" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img152.png"
 ALT="$\displaystyle g12 = -0.6 \times x1 \times x2 \times \ln [$">Na<IMG
 WIDTH="171" HEIGHT="60" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img153.png"
 ALT="$\displaystyle ^+] \times \frac{\ln [(\frac{1}{x1} - 1) \times \mbox{Na}^+]}{N}
$">
</DIV><P></P>
See what is g1 and g2 in the sodium corrections tanna06 and tanna07 (g1) and
  magnesium corrections tanmg06 and tanmg07 (g2).
  Formula representing the fractional contribution of Na+ ions.
 <P><!-- MATH
 \begin{displaymath}
x1 = \frac{[\mbox{Na}^+]}{(\mbox{Na}^+ + (\frac{8.1 - 32.4}{N}) \times (5.2 - \ln [\mbox{Na}^+]) \times \mbox{Mg}^{2+})}
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="354" HEIGHT="56" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img154.png"
 ALT="$\displaystyle x1 = \frac{[\mbox{Na}^+]}{(\mbox{Na}^+ + (\frac{8.1 - 32.4}{N}) \times (5.2 - \ln [\mbox{Na}^+]) \times \mbox{Mg}^{2+})}
$">
</DIV><P></P>
Formula representing the fractional contribution of Mg2+ ions.
 <P><!-- MATH
 \begin{displaymath}
x2= 1-x1
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="87" HEIGHT="31" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img155.png"
 ALT="$\displaystyle x2= 1-x1
$">
</DIV><P></P>
</LI>
</UL>

<P>
<BR><P></P>
<DIV ALIGN="CENTER">
<TABLE CELLPADDING=3 BORDER="1">
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">correction</SPAN></TH>
<TH ALIGN="CENTER"><SPAN  CLASS="textbf">limits</SPAN></TH>
<TH ALIGN="CENTER"><SPAN  CLASS="textbf">Article</SPAN></TH>
</TR>
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">oxcmix08</SPAN></TH>
<TD ALIGN="CENTER">DNA <br/> Mg&gt;=0.0005 <br/> Mg&lt;=0.6 <br/> Na+K+Tris/2&gt;0</TD>
<TD ALIGN="CENTER">Richard Owczarzy et al.,2008, Biochemistry, 47, 5336-5353</TD>
</TR>
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">tanmix07</SPAN></TH>
<TD ALIGN="CENTER">DNA and RNA <br/> Mg&gt;=0.1 <br/> Mg&lt;=0.3 <br/> Na+K+Tris/2&gt;=0.1 <br/> Na+K+Tris/2&lt;=0.3</TD>
<TD ALIGN="CENTER">Zhi-Jie Tan et al, 2007, Biophysical Journal, 92, 3615-3632</TD>
</TR>
</TABLE>
</DIV><BR>

<P>
If the user doesn't enter any ion correction, the algorithm from Owczarzy et al. (2008) will
be used by default :
<P><!-- MATH
 \begin{displaymath}
[\mbox{Mon}^+] = [\mbox{Na}^+] + [\mbox{k}^+] + [\mbox{Tris}^+]
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img156.png"
 ALT="$\displaystyle [$">Mon<IMG
 WIDTH="45" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img157.png"
 ALT="$\displaystyle ^+] = [$">Na<IMG
 WIDTH="43" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img158.png"
 ALT="$\displaystyle ^+] + [$">k<IMG
 WIDTH="43" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img158.png"
 ALT="$\displaystyle ^+] + [$">Tris<IMG
 WIDTH="19" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img114.png"
 ALT="$\displaystyle ^+]
$">
</DIV><P></P>
Where <!-- MATH
 $[\mbox{Tris}^+]$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">Tris<IMG
 WIDTH="19" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img8.png"
 ALT="$ ^+]$"></SPAN> is equal to half of total tris buffer concentration. (in the option -t, it is the Tris buffer concentration
which is entered).

<P>

<UL>
<LI>if <!-- MATH
 $[\mbox{Mon}^+]$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">Mon<IMG
 WIDTH="19" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img8.png"
 ALT="$ ^+]$"></SPAN> = 0, a default sodium correction will be used.
</LI>
<LI>if <!-- MATH
 $[\mbox{Mg}^{2+}]$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">Mg<IMG
 WIDTH="26" HEIGHT="34" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img101.png"
 ALT="$ ^{2+}]$"></SPAN>^0.5 / <!-- MATH
 $[\mbox{Mon}^+]$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">Mon<IMG
 WIDTH="19" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img8.png"
 ALT="$ ^+]$"></SPAN> &lt; 0.22, a default sodium correction is used.
Monovalent ion influence is dominant, divalent cations can be 
disregarded.
</LI>
<LI>if <!-- MATH
 $[\mbox{Mg}^{2+}]$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">Mg<IMG
 WIDTH="26" HEIGHT="34" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img101.png"
 ALT="$ ^{2+}]$"></SPAN>^0.5 / <!-- MATH
 $[\mbox{Mon}^+]$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">Mon<IMG
 WIDTH="19" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img8.png"
 ALT="$ ^+]$"></SPAN> &gt;= 0.22 and <!-- MATH
 $[\mbox{Mg}^{2+}]$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">Mg<IMG
 WIDTH="26" HEIGHT="34" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img101.png"
 ALT="$ ^{2+}]$"></SPAN>^0.5 / <!-- MATH
 $[\mbox{Mon}^+]$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">Mon<IMG
 WIDTH="19" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img8.png"
 ALT="$ ^+]$"></SPAN> &lt; 6, 
a default mixed Na Mg correction is used.
We can have a competitive DNA or RNA binding between monovalent and divalent 
cations.
</LI>
<LI>if <!-- MATH
 $[\mbox{Mg}^{2+}]$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">Mg<IMG
 WIDTH="26" HEIGHT="34" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img101.png"
 ALT="$ ^{2+}]$"></SPAN>^0.5 / <!-- MATH
 $[\mbox{Mon}^+]$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">Mon<IMG
 WIDTH="19" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img8.png"
 ALT="$ ^+]$"></SPAN> &gt;= 6, a default magnesium correction is used.
Divalent cation influence is dominant, monovalent cations can be 
disregarded.
</LI>
</UL>

<P>
Moreover, if the user wants to use a sodium correction but also enters a potassium, Tris buffer
and/or a magnesium concentration, a sodium equivalent concentration which takes into account the other
ion concentrations is computed before applying the sodium correction.
Several sodium equivalence ready to use are proposed by this program :

<UL>
<LI><SPAN  CLASS="textit">ahs01</SPAN>
 <P><!-- MATH
 \begin{displaymath}
[\mbox{NaEq}^+]=[\mbox{Na}^+]+[\mbox{K}^+]+\frac{[\mbox{Tris}^+]}{2}+3.79 \sqrt{[\mbox{Mg}^{2+}] - [\mbox{dNTP}]}
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img156.png"
 ALT="$\displaystyle [$">NaEq<IMG
 WIDTH="45" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img157.png"
 ALT="$\displaystyle ^+] = [$">Na<IMG
 WIDTH="43" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img158.png"
 ALT="$\displaystyle ^+] + [$">K<IMG
 WIDTH="265" HEIGHT="57" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img159.png"
 ALT="$\displaystyle ^+]+\frac{[\mbox{Tris}^+]}{2}+3.79 \sqrt{[\mbox{Mg}^{2+}] - [\mbox{dNTP}]}
$">
</DIV><P></P>
</LI>
<LI><SPAN  CLASS="textit">mit96</SPAN>
 <P><!-- MATH
 \begin{displaymath}
[\mbox{NaEq}^+]=[\mbox{Na}^+]+[\mbox{K}^+]+\frac{[\mbox{Tris}^+]}{2}+4 \sqrt{[\mbox{Mg}^{2+}] - [\mbox{dNTP}]}
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img156.png"
 ALT="$\displaystyle [$">NaEq<IMG
 WIDTH="45" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img157.png"
 ALT="$\displaystyle ^+] = [$">Na<IMG
 WIDTH="43" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img158.png"
 ALT="$\displaystyle ^+] + [$">K<IMG
 WIDTH="245" HEIGHT="57" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img160.png"
 ALT="$\displaystyle ^+]+\frac{[\mbox{Tris}^+]}{2}+4 \sqrt{[\mbox{Mg}^{2+}] - [\mbox{dNTP}]}
$">
</DIV><P></P>
</LI>
<LI><SPAN  CLASS="textit">pey00</SPAN>
 <P><!-- MATH
 \begin{displaymath}
[\mbox{NaEq}^+]=[\mbox{Na}^+]+[\mbox{K}^+]+\frac{[\mbox{Tris}^+]}{2}+3.3 \sqrt{[\mbox{Mg}^{2+}] - [\mbox{dNTP}]}
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img156.png"
 ALT="$\displaystyle [$">NaEq<IMG
 WIDTH="45" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img157.png"
 ALT="$\displaystyle ^+] = [$">Na<IMG
 WIDTH="43" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img158.png"
 ALT="$\displaystyle ^+] + [$">K<IMG
 WIDTH="257" HEIGHT="57" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img161.png"
 ALT="$\displaystyle ^+]+\frac{[\mbox{Tris}^+]}{2}+3.3 \sqrt{[\mbox{Mg}^{2+}] - [\mbox{dNTP}]}
$">
</DIV><P></P>
</LI>
</UL>

<P>
For further information, see the referenced articles :
<BR><P></P>
<DIV ALIGN="CENTER">
<TABLE CELLPADDING=3 BORDER="1">
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">correction</SPAN></TH>
<TH ALIGN="CENTER"><SPAN  CLASS="textbf">limits</SPAN></TH>
<TH ALIGN="CENTER"><SPAN  CLASS="textbf">Article</SPAN></TH>
</TR>
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">ahs01</SPAN></TH>
<TD ALIGN="CENTER">DNA</TD>
<TD ALIGN="CENTER">Nicolas Von Ahsen et al. 2001, Clinical Chemistry, 47, 1956-1961</TD>
</TR>
<TR><TD ALIGN="CENTER">mit96</TD>
<TD ALIGN="CENTER">DNA</TD>
<TD ALIGN="CENTER">Mitsuhashi M. et al, 1996, J. Clin. Lab. Anal, 10, 277-284</TD>
</TR>
<TR><TD ALIGN="CENTER">pey00</TD>
<TD ALIGN="CENTER">DNA</TD>
<TD ALIGN="CENTER">Peyret N., 2000, Ph.D Thesis, Section .5.4.2, 128, Wayne State University, Detroit, MI</TD>
</TR>
</TABLE>
</DIV><BR>

<P>

<H2><A NAME="SECTION00055000000000000000">
Correction for the concentration of denaturing agents</A>
</H2>  

<P>
<SMALL>MELTING</SMALL> is currently accurate when the hybridisation is performed at pH <SPAN CLASS="MATH"><IMG
 WIDTH="40" HEIGHT="31" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img162.png"
 ALT="$ 7\pm 1$"></SPAN>,  
but some temperature corrections for the formamide and DMSO concentrations exists and can be
applied. However, these corrections are rough approximations and results accuracy may be lost.

<P>

<H3><A NAME="SECTION00055100000000000000">
DMSO corrections, DMSO in %</A>
</H3>
  
<UL>
<LI><SPAN  CLASS="textit">ahs01</SPAN>
  <P><!-- MATH
 \begin{displaymath}
Tm=Tm(DMSO=0)-0.75 \times DMSO
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="293" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img163.png"
 ALT="$\displaystyle Tm=Tm(DMSO=0)-0.75 \times DMSO
$">
</DIV><P></P>
</LI>
<LI><SPAN  CLASS="textit">cul76</SPAN>
  <P><!-- MATH
 \begin{displaymath}
Tm=Tm(DMSO=0)-0.5 \times DMSO
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="286" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img164.png"
 ALT="$\displaystyle Tm=Tm(DMSO=0)-0.5 \times DMSO
$">
</DIV><P></P>
</LI>
<LI><SPAN  CLASS="textit">esc80</SPAN>
  <P><!-- MATH
 \begin{displaymath}
Tm=Tm(DMSO=0)-0.675 \times DMSO
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="301" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img165.png"
 ALT="$\displaystyle Tm=Tm(DMSO=0)-0.675 \times DMSO
$">
</DIV><P></P>
</LI>
<LI><SPAN  CLASS="textit">mus81</SPAN>
  <P><!-- MATH
 \begin{displaymath}
Tm=Tm(DMSO=0)-0.6 \times DMSO
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="286" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img166.png"
 ALT="$\displaystyle Tm=Tm(DMSO=0)-0.6 \times DMSO
$">
</DIV><P></P>
</LI>
</UL>

<P>
For further information, see the referenced articles :
<BR><P></P>
<DIV ALIGN="CENTER">
<TABLE CELLPADDING=3 BORDER="1">
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">correction</SPAN></TH>
<TH ALIGN="CENTER"><SPAN  CLASS="textbf">limits</SPAN></TH>
<TH ALIGN="CENTER"><SPAN  CLASS="textbf">Article</SPAN></TH>
</TR>
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">ahs01</SPAN></TH>
<TD ALIGN="CENTER">DNA <br/> not tested with experimental results</TD>
<TD ALIGN="CENTER">Nicolas Von Ahsen et al. 2001, Clinical Chemistry, 47, 1956-1961</TD>
</TR>
<TR><TD ALIGN="CENTER">cul76</TD>
<TD ALIGN="CENTER">DNA <br/> not tested with experimental results</TD>
<TD ALIGN="CENTER">Cullen Br et al., 1976, 3, 49-62</TD>
</TR>
<TR><TD ALIGN="CENTER">esc80</TD>
<TD ALIGN="CENTER">DNA <br/> not tested with experimental results</TD>
<TD ALIGN="CENTER">Escara JF et al., 1980, 19, 1315-1327</TD>
</TR>
<TR><TD ALIGN="CENTER">mus80</TD>
<TD ALIGN="CENTER">DNA <br/> not tested with experimental results</TD>
<TD ALIGN="CENTER">Musielski H. et al., 1981,Z allg Microbiol 1981; 21, 447-456</TD>
</TR>
</TABLE>
</DIV><BR>

<P>

<H3><A NAME="SECTION00055200000000000000">
formamide corrections</A>
</H3>
  
<UL>
<LI><SPAN  CLASS="textit">bla96</SPAN>
  <P><!-- MATH
 \begin{displaymath}
Tm=Tm(formamide=0)+ (0.453 \times \Chi_{GC} - 2.88) \times formamide
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="470" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img167.png"
 ALT="$\displaystyle Tm=Tm(formamide=0)+ (0.453 \times \Chi_{GC} - 2.88) \times formamide
$">
</DIV><P></P>
Where <SPAN CLASS="MATH"><IMG
 WIDTH="37" HEIGHT="31" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img104.png"
 ALT="$ \Chi_{GC}$"></SPAN> is the frequence of GC base pairs in the sequence.
  formamide is in mol/L
</LI>
<LI><SPAN  CLASS="textit">lincorr</SPAN>
  <P><!-- MATH
 \begin{displaymath}
Tm=Tm(formamide=0)-0.65 \times formamide
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="350" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img168.png"
 ALT="$\displaystyle Tm=Tm(formamide=0)-0.65 \times formamide
$">
</DIV><P></P>
Where formamide is in %.
  
</LI>
</UL>

<P>
For further information, see the referenced articles :
<BR><P></P>
<DIV ALIGN="CENTER">
<TABLE CELLPADDING=3 BORDER="1">
<TR><TD ALIGN="CENTER">correction</TD>
<TD ALIGN="CENTER">limits</TD>
<TD ALIGN="CENTER">Article</TD>
</TR>
<TR><TD ALIGN="CENTER">bla96</TD>
<TD ALIGN="CENTER">DNA</TD>
<TD ALIGN="CENTER">R. D. Blake et al., 1996</TD>
</TR>
<TR><TD ALIGN="CENTER">&nbsp;</TD>
<TD ALIGN="CENTER">not tested</TD>
<TD ALIGN="CENTER">Vol. 24, No. 11 2095-2103</TD>
</TR>
<TR><TD ALIGN="CENTER">&nbsp;</TD>
<TD ALIGN="CENTER">with e</TD>
<TD ALIGN="CENTER">&nbsp;</TD>
</TR>
<TR><TD ALIGN="CENTER">&nbsp;</TD>
<TD ALIGN="CENTER">values</TD>
<TD ALIGN="CENTER">&nbsp;</TD>
</TR>
<TR><TD ALIGN="CENTER">&nbsp;</TD>
<TD ALIGN="CENTER">formamide in mol/L</TD>
<TD ALIGN="CENTER">&nbsp;</TD>
</TR>
<TR><TD ALIGN="CENTER">lincorr</TD>
<TD ALIGN="CENTER">DNA</TD>
<TD ALIGN="CENTER">McConaughy et al., 1969</TD>
</TR>
<TR><TD ALIGN="CENTER">&nbsp;</TD>
<TD ALIGN="CENTER">not tested</TD>
<TD ALIGN="CENTER">Biochemistry 8, 3289-3295.</TD>
</TR>
<TR><TD ALIGN="CENTER">&nbsp;</TD>
<TD ALIGN="CENTER">with e</TD>
<TD ALIGN="CENTER">Record, M.T., Jr, 1967</TD>
</TR>
<TR><TD ALIGN="CENTER">&nbsp;</TD>
<TD ALIGN="CENTER">in %</TD>
<TD ALIGN="CENTER">Biopolymers, 5, 975-992.</TD>
</TR>
<TR><TD ALIGN="CENTER">&nbsp;</TD>
<TD ALIGN="CENTER">values</TD>
<TD ALIGN="CENTER">Casey J. et al, 1977</TD>
</TR>
<TR><TD ALIGN="CENTER">&nbsp;</TD>
<TD ALIGN="CENTER">Formamide in</TD>
<TD ALIGN="CENTER">Nucleic acids research, 4, 1539-1532.</TD>
</TR>
<TR><TD ALIGN="CENTER">&nbsp;</TD>
<TD ALIGN="CENTER">%</TD>
<TD ALIGN="CENTER">Hutton Jr, 1977</TD>
</TR>
<TR><TD ALIGN="CENTER">&nbsp;</TD>
<TD ALIGN="CENTER">&nbsp;</TD>
<TD ALIGN="CENTER">Nucleic acids research, 4, 3537-3555.</TD>
</TR>
</TABLE>
</DIV><BR>

<P>

<H2><A NAME="SECTION00056000000000000000">
Long sequences </A>
</H2> 

<P>
It is important to realise that the nearest-neighbor approach has been established  
on small oligonucleotides. Therefore the use of <SMALL>MELTING</SMALL> in the non-approximative  
mode is really accurate only for relatively short sequences (Although if the sequences are 
two short, let's say <SPAN CLASS="MATH"><IMG
 WIDTH="17" HEIGHT="31" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img169.png"
 ALT="$ &lt;$"></SPAN> 6&nbsp;bp, the influence of extremities becomes too important and the  
reliability decreases a lot). For long sequences an approximative mode has been designed. 
This mode is launched if the sequence length is higher than the value given by the option -T 
(the default threshold is 60 bp).

<P>
The melting temperature can be computed by one of the following formulas:   

<UL>
<LI>ahs01
<P><!-- MATH
 \begin{displaymath}
Tm = 80.4 + 0.345 \times \%GC + \log[\mbox{Na}^+] \times (17.0 - 0.135 \times \%GC) - \frac{550}{size}
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="237" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img170.png"
 ALT="$\displaystyle Tm = 80.4 + 0.345 \times \%GC + \log[$">Na<IMG
 WIDTH="245" HEIGHT="50" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img171.png"
 ALT="$\displaystyle ^+] \times (17.0 - 0.135 \times \%GC) - \frac{550}{size}
$">
</DIV><P></P>
</LI>
<LI>che93
<P><!-- MATH
 \begin{displaymath}
Tm = 69.3 + 0.41 \times \%GC - \frac{650}{size}
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="237" HEIGHT="50" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img172.png"
 ALT="$\displaystyle Tm = 69.3 + 0.41 \times \%GC - \frac{650}{size}
$">
</DIV><P></P>
</LI>
<LI>che93corr
<P><!-- MATH
 \begin{displaymath}
Tm = 69.3 + 0.41 \times \%GC - \frac{535}{size}
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="237" HEIGHT="50" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img173.png"
 ALT="$\displaystyle Tm = 69.3 + 0.41 \times \%GC - \frac{535}{size}
$">
</DIV><P></P>
</LI>
<LI>marschdot
<P><!-- MATH
 \begin{displaymath}
Tm = 81.5 + 16.6 \times \log[\mbox{Na}^+] + 0.41 \times \%GC - \frac{675}{size}
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="172" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img174.png"
 ALT="$\displaystyle Tm = 81.5 + 16.6 \times \log[$">Na<IMG
 WIDTH="177" HEIGHT="50" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img175.png"
 ALT="$\displaystyle ^+] + 0.41 \times \%GC - \frac{675}{size}
$">
</DIV><P></P>
</LI>
<LI>owe69
<P><!-- MATH
 \begin{displaymath}
Tm = 87.16 + 0.345 \times \%GC + \log[\mbox{Na}^+] \times (20.17 - 0.066 \times \%GC)
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="245" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img176.png"
 ALT="$\displaystyle Tm = 87.16 + 0.345 \times \%GC + \log[$">Na<IMG
 WIDTH="201" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img177.png"
 ALT="$\displaystyle ^+] \times (20.17 - 0.066 \times \%GC)
$">
</DIV><P></P>
</LI>
<LI>san98
<P><!-- MATH
 \begin{displaymath}
Tm = 77.1 + 11.7 \times \log[\mbox{Na}^+] + 0.41 \times \%GC - \frac{528}{size}
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="172" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img178.png"
 ALT="$\displaystyle Tm = 77.1 + 11.7 \times \log[$">Na<IMG
 WIDTH="177" HEIGHT="50" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img179.png"
 ALT="$\displaystyle ^+] + 0.41 \times \%GC - \frac{528}{size}
$">
</DIV><P></P>
</LI>
<LI>wetdna91
<P><!-- MATH
 \begin{displaymath}
Tm = 81.5 + 16.6\log\frac{[\mbox{Na}^+]}{1+0.7[\mbo\times{Na}^+]} + 0.41\% GC - \frac{500}{size}
- \%Mismatching
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="527" HEIGHT="56" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img180.png"
 ALT="$\displaystyle Tm = 81.5 + 16.6\log\frac{[\mbox{Na}^+]}{1+0.7[\mbo\times{Na}^+]} + 0.41\% GC - \frac{500}{size}
- \%Mismatching
$">
</DIV><P></P>
</LI>
<LI>wetrna91
<P><!-- MATH
 \begin{displaymath}
Tm = 78 + 16.6\log\frac{[\mbox{Na}^+]}{1+0.7[\mbo\times{Na}^+]} + 0.7\% GC - \frac{500}{size}
- \%Mismatching
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="507" HEIGHT="56" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img181.png"
 ALT="$\displaystyle Tm = 78 + 16.6\log\frac{[\mbox{Na}^+]}{1+0.7[\mbo\times{Na}^+]} + 0.7\% GC - \frac{500}{size}
- \%Mismatching
$">
</DIV><P></P>
</LI>
<LI>wetdnarna91
<P><!-- MATH
 \begin{displaymath}
Tm = 67 + 16.6\log\frac{[\mbox{Na}^+]}{1+0.7[\mbo\times{Na}^+]} + 0.8\% GC - \frac{500}{size}
- \%Mismatching
\end{displaymath}
 -->
</P><DIV ALIGN="CENTER" CLASS="mathdisplay">
<IMG
 WIDTH="507" HEIGHT="56" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img182.png"
 ALT="$\displaystyle Tm = 67 + 16.6\log\frac{[\mbox{Na}^+]}{1+0.7[\mbo\times{Na}^+]} + 0.8\% GC - \frac{500}{size}
- \%Mismatching
$">
</DIV><P></P>
</LI>
</UL>

<P>
For further information, see the referenced articles :
<BR><P></P>
<DIV ALIGN="CENTER">
<TABLE CELLPADDING=3 BORDER="1">
<TR><TH ALIGN="CENTER"><SPAN  CLASS="textbf">formula</SPAN></TH>
<TH ALIGN="CENTER"><SPAN  CLASS="textbf">limits</SPAN></TH>
<TH ALIGN="CENTER"><SPAN  CLASS="textbf">Article</SPAN></TH>
</TR>
<TR><TD ALIGN="CENTER">ahs01</TD>
<TD ALIGN="CENTER">DNA <br/> no mismatch</TD>
<TD ALIGN="CENTER">Nicolas Von Ahsen et al. 2001, Clinical Chemistry, 47, 1956-1961</TD>
</TR>
<TR><TD ALIGN="CENTER">che93</TD>
<TD ALIGN="CENTER">DNA <br/> no mismatch <br/>Na=0 <br/> Mg=0.0015 <br/> Tris=0.01 <br/>K=0.05 </TD>
<TD ALIGN="CENTER">Marmur J et al., 1962, Journal of molecular biology, 5, 109-118</TD>
</TR>
<TR><TD ALIGN="CENTER">che93corr</TD>
<TD ALIGN="CENTER">DNA <br/> no mismatch <br/>Na=0 <br/> Mg=0.0015 <br/> Tris=0.01 <br/>K=0.05 </TD>
<TD ALIGN="CENTER">Marmur J et al., 1962, Journal of molecular biology, 5, 109-118</TD>
</TR>
<TR><TD ALIGN="CENTER">marschdot</TD>
<TD ALIGN="CENTER">DNA <br/> no mismatch</TD>
<TD ALIGN="CENTER">James G. Wetmur,1991, Critical reviews in biochemistry and molecular biology, 26, 227-259 <br/> Marmur J et al., 1962,Journal of molecular biology, 5, 109-118 <br/> Chester N et al., 1993,Analytical Biochemistry, 209, 284-290 <br/> Schildkraut C et al., 1965, Biopolymers, 3, 95-110 <br/> Wahl GM et al., 1987, Methods Enzymol;152:399 - 407 <br/> Britten RJ et al.,1974, Methods Enzymol ;29:363-418 <br/> Hall TJ et al., 1980, J Mol Evol ;16:95-110</TD>
</TR>
<TR><TD ALIGN="CENTER">owe69</TD>
<TD ALIGN="CENTER">DNA <br/> no mismatch</TD>
<TD ALIGN="CENTER">Owen RJ et al., 1969, Biopolymers, 7:503-16 <br/> Frank-Kamenetskii MD.,1971, Biopolymers;10:2623- 4 <br/> Blake RD, 1996, Encyclopedia of molecular biology and molecular medicine, Vol. 2., :1-19 <br/> Blake RD et al.,1998, Nucleic Acids Res;26:3323-32</TD>
</TR>
<TR><TD ALIGN="CENTER">san98</TD>
<TD ALIGN="CENTER">DNA  <br/> no mismatch</TD>
<TD ALIGN="CENTER">Santalucia J Jr, 1998, Proc Nacl Acad Sci USA, 95, 1460-1465 <br/> Nicolas Von Ahsen et al. 2001, Clinical Chemistry, 47, 1956-1961</TD>
</TR>
<TR><TD ALIGN="CENTER">wetdna91</TD>
<TD ALIGN="CENTER">DNA</TD>
<TD ALIGN="CENTER">James G. Wetmur,1991, Critical reviews in biochemistry and molecular biology, 26, 227-259</TD>
</TR>
<TR><TD ALIGN="CENTER">wetrna91</TD>
<TD ALIGN="CENTER">RNA</TD>
<TD ALIGN="CENTER">James G. Wetmur,1991,  Critical reviews in biochemistry and molecular biology, 26, 227-259</TD>
</TR>
<TR><TD ALIGN="CENTER">wetdnarna91</TD>
<TD ALIGN="CENTER">DNA/RNA</TD>
<TD ALIGN="CENTER">James G. Wetmur,1991,  Critical reviews in biochemistry and molecular biology, 26, 227-259</TD>
</TR>
</TABLE>
</DIV><BR>

<P>

<DIV ALIGN="CENTER"><A NAME="2273"></A>
<TABLE>
<CAPTION ALIGN="BOTTOM"><STRONG>Figure:</STRONG>
Comparison of experimental and computed Tm for various sets of
 DNA approximative formulas. <!-- MATH
 $[\mbox{Na}^+] = 1$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">Na<IMG
 WIDTH="49" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img15.png"
 ALT="$ ^+] = 1$"></SPAN>&nbsp;M</CAPTION>
<TR><TD><IMG
  WIDTH="586" HEIGHT="342" ALIGN="BOTTOM" BORDER="0"
 SRC="html/DNAApproximativeMode.png"
 ALT="Image DNAApproximativeMode"></TD></TR>
</TABLE>
</DIV>

<P>

<DIV ALIGN="CENTER"><A NAME="2274"></A>
<TABLE>
<CAPTION ALIGN="BOTTOM"><STRONG>Figure:</STRONG>
Comparison of experimental and computed Tm for various sets of
 RNA approximative formulas. <!-- MATH
 $[\mbox{Na}^+] = 1$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">Na<IMG
 WIDTH="49" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img15.png"
 ALT="$ ^+] = 1$"></SPAN>&nbsp;M</CAPTION>
<TR><TD><IMG
  WIDTH="417" HEIGHT="292" ALIGN="BOTTOM" BORDER="0"
 SRC="html/RNAApproximativeMode.png"
 ALT="Image RNAApproximativeMode"></TD></TR>
</TABLE>
</DIV>

<P>

<DIV ALIGN="CENTER"><A NAME="2275"></A>
<TABLE>
<CAPTION ALIGN="BOTTOM"><STRONG>Figure:</STRONG>
Comparison of experimental and computed Tm for various sets of
 DNARNA approximative formulas. <!-- MATH
 $[\mbox{Na}^+] = 1$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">Na<IMG
 WIDTH="49" HEIGHT="35" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img15.png"
 ALT="$ ^+] = 1$"></SPAN>&nbsp;M, <!-- MATH
 $[\mbox{nucleic acid}] = 4\cdot{}10^{-4}$
 -->
<SPAN CLASS="MATH"><IMG
 WIDTH="9" HEIGHT="32" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img7.png"
 ALT="$ [$">nucleic acid<IMG
 WIDTH="83" HEIGHT="34" ALIGN="MIDDLE" BORDER="0"
 SRC="html/img16.png"
 ALT="$ ] = 4\cdot{}10^{-4}$"></SPAN>&nbsp;M</CAPTION>
<TR><TD><IMG
  WIDTH="485" HEIGHT="313" ALIGN="BOTTOM" BORDER="0"
 SRC="html/DNARNAApproximativeMode.png"
 ALT="Image DNARNAApproximativeMode"></TD></TR>
</TABLE>
</DIV>

<P>

<H2><A NAME="SECTION00060000000000000000">
Bibliography</A>
</H2><DL COMPACT><DD><P></P><DT><A NAME="Allawi_36_1997">1</A>
<DD>
SantaLucia&nbsp;J. Allawi&nbsp;H.T.
<BR>Thermodynamics and nmr of internal g-t mismatches in dna.
<BR><EM>Biochemistry</EM>, 36:10581-10594, 1997.

<P></P><DT><A NAME="Allawi_37_1998">2</A>
<DD>
SantaLucia&nbsp;J. Allawi&nbsp;H.T.
<BR>Nearest neighbor thermodynamics parameters for internal g.a
  mismatches in dna.
<BR><EM>Biochemistry</EM>, 37:2170-2179, 1998.

<P></P><DT><A NAME="Allawi_26_1998">3</A>
<DD>
SantaLucia&nbsp;J. Allawi&nbsp;H.T.
<BR>Thermodynamics of internal c.t mismatches in dna.
<BR><EM>Biochemistry</EM>, 26:2694-2701, 1998.

<P></P><DT><A NAME="Amanda_34_2006">4</A>
<DD>
Nathan Haines M. Coleen&nbsp;Zink Amanda S.&nbsp;O'toole, Stacy&nbsp;Miller and Martin&nbsp;J
  Serra.
<BR>Comprehensive thermodynamic analysis of 3' double-nucleotide
  overhangs neighboring watson-crick terminal base pairs.
<BR><EM>Nucleic Acids research</EM>, 34:3338-3344, 2006.

<P></P><DT><A NAME="Amanda_11_2005">5</A>
<DD>
Stacy&nbsp;Miller Amanda S.&nbsp;O'toole and Martin&nbsp;J Serra.
<BR>Stability of 3' double nucleotide overhangs that model the 3'ends of
  sirna.
<BR><EM>RNA</EM>, 11:512-516, 2005.

<P></P><DT><A NAME="Blake_26_1998">6</A>
<DD>
R.&nbsp;D. Blake and S.&nbsp;G. Delcourt.
<BR>Thermal stability of dna.
<BR><EM>Nucleic Acids Res</EM>, 26:3323-3332 and corrigendum, 1998.

<P></P><DT><A NAME="R_24_1996">7</A>
<DD>
R.&nbsp;D. Blake and Scott&nbsp;G. Delcourt.
<BR>Thermodynamic effects of formamide on dna stability.
<BR><EM>Nucleic Acids Research</EM>, 24, 11:2095-2103, 1996.

<P></P><DT><A NAME="Bommarito_28_2000">8</A>
<DD>
SantaLucia&nbsp;J. Bommarito&nbsp;S., Peyret&nbsp;N.
<BR>Thermodynamic parameters for dnasequences with dangling ends.
<BR><EM>Nucleic Acids Res</EM>, 28:1929-1934, 2000.

<P></P><DT><A NAME="Breslauer_83_1986">9</A>
<DD>
Bl Breslauer&nbsp;K.J., Frank&nbsp;R.

<P></P><DT><A NAME="Britten_29_1974">10</A>
<DD>
Neufeld&nbsp;BR Britten&nbsp;RJ, Graham&nbsp;DE.
<BR>Analysis of repeating dna sequences by reassociation.
<BR><EM>Methods Enzymol</EM>, 29:363-418, 1974.

<P></P><DT><A NAME="Broda_44_2005">11</A>
<DD>
Zofia Gdaniec Tadeusz&nbsp;Kulinski Broda&nbsp;Magdalena, Elbieta&nbsp;Kierzek and Ryszard
  Kierzek.
<BR>Thermodynamic stability of rna structures formed by cng trinucleotide
  repeats. implication for prediction of rna structure.
<BR><EM>Biochemistry</EM>, 44:10873-10882, 2005.

<P></P><DT><A NAME="Chester_209_1993">12</A>
<DD>
Marshak&nbsp;DR Chester&nbsp;N.
<BR>dimethyl sulfoxide-mediated primer tm reduction : a method for
  analyzing the role of renaturation temperature in the polymerase chain
  reaction.
<BR><EM>Analytical Biochemistry</EM>, 209:284-290, 1993.

<P></P><DT><A NAME="Cullen_3_1976">13</A>
<DD>
Bick&nbsp;Md Cullen&nbsp;Br.
<BR>Thermal denaturation of dna from bromodeoxyuridine substitued cells.
<BR><EM>Nucleic acids research</EM>, 3:49-62, 1976.

<P></P><DT><A NAME="David_288_1999">14</A>
<DD>
Michael&nbsp;Zucker David H.&nbsp;Mathews, Jeffrey&nbsp;Sabina and Douglas&nbsp;H Turner.
<BR>Expanded sequence dependence of thermodynamic parameters improves
  predictionof rna secondary structure.
<BR><EM>J. Mol. Biol</EM>, 288:911-940, 1999.

<P></P><DT><A NAME="Amber_46_2007">15</A>
<DD>
Amber&nbsp;R. Davis and Brent&nbsp;M. Znosko.
<BR>Thermodynamic characterization of single mismatches found in
  naturally occurring rna.
<BR><EM>Biochemistry</EM>, 46:13425-13436, 2007.

<P></P><DT><A NAME="Amber_47_2008">16</A>
<DD>
Amber&nbsp;R. Davis and Brent&nbsp;M. Znosko.
<BR>Thermodynamic characterization of naturally occurring rna single
  mismatches with g-u nearest neighbors.
<BR><EM>Biochemistry</EM>, 47:10178-10187, 2008.

<P></P><DT><A NAME="Elzbieta_34_2006">17</A>
<DD>
Anna Ciesielska Douglas H.&nbsp;Turner Elzbieta&nbsp;Kierzek, David H.&nbsp;Mathews and
  Ryszard Kierzek.
<BR>Nearest neighbor parameters for watson crick complementary
  heteroduplexesformed between 2-o-methyl rna and rna oligonucleotides.
<BR><EM>Nucleic acids research</EM>, 34:3609-3614, 2006.

<P></P><DT><A NAME="Escara_19_1980">18</A>
<DD>
Hutton&nbsp;Jr Escara&nbsp;JF.
<BR>Thermal stability and renaturation of dna in dimethyl
  sulfoxidesolutions: acceleration of the renaturation rate.
<BR><EM>Biopolymers</EM>, 19:1315-1327, 1980.

<P></P><DT><A NAME="Frank_10_1971">19</A>
<DD>
M.&nbsp;D. Frank-Kamenetskii.
<BR>Simplification of the empirical relationship between melting
  temperature of dna, its gc content and concentration of sodium ions in
  solution.
<BR><EM>Biopolymers</EM>, 10:2623-2624, 1971.

<P></P><DT><A NAME="Freier_83_1986">20</A>
<DD>
Jaeger J.A. Sugimoto N. Caruthers M.H. Neilson T. Turner&nbsp;D.H. Freier&nbsp;S.M.,
  Kierzek&nbsp;R.
<BR><EM>Biochemistry</EM>, 83:9373-9377, 1986.

<P></P><DT><A NAME="Fumiaki_43_2004">21</A>
<DD>
Masahito&nbsp;Yamamoto Fumiaki&nbsp;Tanaka, Atsushi&nbsp;Kameda and Azuma Ohuchi.
<BR>Thermodynamic parameters based on a nearest neighbor model for dna
  sequences with a single bulge loop.
<BR><EM>Biochemistry</EM>, 43:7143-7150, 2004.

<P></P><DT><A NAME="Hall_16_1980">22</A>
<DD>
Davidson EH Britten&nbsp;RJ Hall&nbsp;TJ, Grula&nbsp;JW.
<BR>Evolution of sea urchin non-repetitive dna.
<BR><EM>J Mol Evol</EM>, 16:95-110, 1980.

<P></P><DT><A NAME="Hiroyuki_49_2005">23</A>
<DD>
Daijiro&nbsp;Matsunaga Hiroyuki&nbsp;Asanuma and Makoto Komiyama.
<BR>Clear-cut photo-regulation of the formation and dissociation of the
  dna duplex by modified oligonucleotide involving multiple azobenzenes.
<BR><EM>Nucleic acids Symposium Series</EM>, 49:35-36, 2005.

<P></P><DT><A NAME="Casey_4_1977">24</A>
<DD>
Casey J. and Davidson N.
<BR><EM>Nucleic acids research</EM>, 4:1539-1532, 1977.

<P></P><DT><A NAME="SantaLucia_95_1998">25</A>
<DD>
SantaLucia J.
<BR>A unified view of polymer, dumbbell, and oligonucleotide dna
  nearest-neighbor thermodynamics.
<BR><EM>Proc Natl Acad Sci USA</EM>, 95:1460-1465, 1998.

<P></P><DT><A NAME="Jaya_46_2007">26</A>
<DD>
Cody K. Cass Erica L.&nbsp;Wunderlich Jaya&nbsp;Badhwar, Saradasri&nbsp;Karri and Brent&nbsp;M.
  Znosco.
<BR>Thermodynamic characterization of rna duplexes containing naturally
  occuring 1x2 nucleotide internal loops.
<BR><EM>Biochemistry</EM>, 46:14715-14724, 2007.

<P></P><DT><A NAME="Wetmur_26_1991">27</A>
<DD>
Wetmur J.G.
<BR>Dna probes: applications of the principles of nucleic acid
  hybridization.
<BR><EM>Crit Rev Biochem Mol Biol</EM>, 26:227-259, 1991.

<P></P><DT><A NAME="Joshua_46_2007">28</A>
<DD>
Kelly A. Klapec Yukiko Stranger-Jones Allison C.Zyra Vasiliy Sim Chad A.
  Griffith Jason D.&nbsp;Long Joshua M.&nbsp;Blose, Michelle L.&nbsp;Manni and Martin&nbsp;J.
  Serra.
<BR>Non-nearest-neighbor dependence of stability for rna bulge loops
  based on the complete set of group i single nucleotide bulge loops.
<BR><EM>Biochemistry</EM>, 46:15123-15135, 2007.

<P></P><DT><A NAME="Hutton_4_1977">29</A>
<DD>
Hutton Jr.
<BR><EM>Nucleic acids research</EM>, 4:3537-3555, 1977.

<P></P><DT><A NAME="Junji_29_2001">30</A>
<DD>
Hiroyuki Kamiya3 Kyohko Yasuda2 Hiroyoshi Fujiki1 Hiroshi Kasai3
  andNaoki&nbsp;Sugimoto Junji&nbsp;Kawakami1, 2.
<BR>Thermodynamic stability of base pairs between 2-hydroxyadenine and
  incoming nucleotides as a determinant of nucleotide incorporation specificity
  during replication.
<BR><EM>Nucleic acids research</EM>, 29:3289-3296, 2001.

<P></P><DT><A NAME="Mitsuhashi_10_1996">31</A>
<DD>
Mitsuhashi M.
<BR>Technical report: Part 1. basic requirements for designing optimal
  oligonucleotide probe sequences.
<BR><EM>J. Clin. Lab. Anal</EM>, 10:277-284, 1996.

<P></P><DT><A NAME="Marmur_5_1962">32</A>
<DD>
J.&nbsp;Marmur and P.&nbsp;Doty.
<BR>Determination of the base composition of deoxyribonucleic acid from
  its thermal denaturation temperature.
<BR><EM>J. Mol. Biol.</EM>, 5:109-118, 1962.

<P></P><DT><A NAME="McConaughy_8_1969">33</A>
<DD>
Laird&nbsp;C.D. McConaughy, B.L. and B.I. McCarthy.
<BR><EM>Biochemistry</EM>, 8:3289-3295, 1969.

<P></P><DT><A NAME="Musielski_21_1981">34</A>
<DD>
Laue R Michel&nbsp;S Musielski&nbsp;H., Mann&nbsp;W.
<BR>Influence of dimethylsulfoxide on transcription by bacteriophage
  t3-induced rna polymerase.
<BR><EM>Z allg Microbiol</EM>, 21:447-456, 1981.

<P></P><DT><A NAME="Peyret_5_2000">35</A>
<DD>
Peyret N.
<BR>Prediction of nucleic acid hybridization : parameters and algorithms.
<BR><EM>Ph.D Thesis 4.2, 128, Wayne State University, Detroit, MI.</EM>, 5,
  2000.

<P></P><DT><A NAME="Nicolas_47_2001">36</A>
<DD>
Carl T&nbsp;Wittwer Nicolas Von&nbsp;Ahsen and Ekkehard Schutz.
<BR>Oligonucleotide melting temperatures under pcr conditions :
  deoxynucleotide triphosphate and dimethyl sulfoxide concentrations with
  comparison to alternative empirical formulas.
<BR><EM>Clinical Chemistry</EM>, 47:1956-1961, 2001.

<P></P><DT><A NAME="Owczarzy_47_2008">37</A>
<DD>
You Y. Behlke M.B. Walder&nbsp;J.A. Owczarzy&nbsp;R., Moreira&nbsp;B.G.
<BR>Predicting stability of dna duplexes in solutions containing
  magnesium and monovalent cations.
<BR><EM>Biochemistry</EM>, 47:5336-5353, 2008.

<P></P><DT><A NAME="Owen_7_1969">38</A>
<DD>
Lapage&nbsp;SP. Owen&nbsp;RJ, Hill&nbsp;LR.
<BR>Determination of dna base compositions from melting profiles in
  dilute buffers.
<BR><EM>Biopolymers</EM>, 7:503-516, 1969.

<P></P><DT><A NAME="Patricia_43_2004">39</A>
<DD>
Raymond J.&nbsp;Peterson Patricia M.&nbsp;McTigue and Jason&nbsp;D. Kahn.
<BR>Sequence-dependent thermodynamic parameters for locked nucleic acid
  (lna)dna duplexformation.
<BR><EM>Biochemistry</EM>, 43:5388-5405, 2004.

<P></P><DT><A NAME="Peyret_38_1999">40</A>
<DD>
Allawi H.T. SantaLucia&nbsp;J. Peyret&nbsp;N., Seneviratne&nbsp;P.A.
<BR>Nearest neighbor thermodynamics and nmr of dna sequences with
  internal a.a, c.c, g.g and t.t mismatches. dependence and ph effects.
<BR><EM>Biochemistry</EM>, 38:3468-3477, 1999.

<P></P><DT><A NAME="Blake_2_1996">41</A>
<DD>
Blake RD.
<BR>Denaturation of dna.
<BR><EM>In: Meyers RA, ed. Encyclopediaof molecular biology and
  molecular medicine</EM>, 2:1-19, 1996.

<P></P><DT><A NAME="Record_5_1967">42</A>
<DD>
Jr&nbsp;Record, M.T.
<BR><EM>Biopolymers</EM>, 5:975-992, 1967.

<P></P><DT><A NAME="Richard_43_2004">43</A>
<DD>
Bernardo G. Moreira Jeffrey A.Manthey Lingyan Huang Mark A.&nbsp;Behlke
  Richard&nbsp;Owczarzy, Yong&nbsp;You and Joseph A.Walder.
<BR>Effects of sodium ions on dna duplex oligomers: Improved predictions
  of melting temperatures.
<BR><EM>Biochemistry</EM>, 43:3537-3554, 2004.

<P></P><DT><A NAME="SantaLucia_33_2004">44</A>
<DD>
Hicks&nbsp;Donald SantaLucia&nbsp;J.
<BR>The thermodynamics of dna structural motifs.
<BR><EM>Annu. Rev. Biophys. Struct</EM>, 33:415-440, 2004.

<P></P><DT><A NAME="SantaLucia_35_1996">45</A>
<DD>
Seneviratne&nbsp;P.A. SantaLucia J.&nbsp;Jr, Allawi&nbsp;H.T.
<BR>Improved nearest-neighbor parameters for predicting dna duplex
  stability.
<BR><EM>Biochemistry</EM>, 35:3555-3562, 1996.

<P></P><DT><A NAME="Schildkraut_3_1965">46</A>
<DD>
C.&nbsp;Schildkraut and S.&nbsp;Lifson.
<BR>Dependence of the melting temperature of dna on salt concentration.
<BR><EM>Biopolymers</EM>, 3:195-208, 1965.

<P></P><DT><A NAME="Stacy_36_2008">47</A>
<DD>
Karen Giovannitti Dan&nbsp;Piper Stacy&nbsp;Miller, Laura E.&nbsp;Jones and Martin&nbsp;J. Serra.
<BR>Thermodynamic analysis of 5 and 3 single- and 3 double-nucleotide
  overhangs neighboring wobble terminal base pairs.
<BR><EM>Nucleic Acids research</EM>, 36:5652-5659, 2008.

<P></P><DT><A NAME="Sugimoto_34_1995">48</A>
<DD>
Katoh M. Matsumura A. Nakamuta H. Ohmichi T. Yoneyama M. Sasaki&nbsp;M. Sugimoto&nbsp;N.,
  Nakano&nbsp;S.
<BR>Thermodynamic parameters to predict stability of rna/dna hybrid
  duplexes.
<BR><EM>Biochemistry</EM>, 34:11211-11216, 1995.

<P></P><DT><A NAME="Sugimoto_354_1994">49</A>
<DD>
Nakano S. Ohmichi T. Sasaki&nbsp;M. Sugimoto&nbsp;N., Katoh&nbsp;M.
<BR>Rna/dna hybrid duplexes with identical nearest-neighbor base-pairs
  have identical stability.
<BR><EM>FEBS Letters</EM>, 354:74-78, 1994.

<P></P><DT><A NAME="Sugimoto_24_1996">50</A>
<DD>
Yoneyama M. Honda&nbsp;K. Sugimoto&nbsp;N., Nakano&nbsp;S.
<BR>Improved thermodynamic parameters and helix initiation factor to
  predict stability of dna duplexes.
<BR><EM>Nuc Acids Res</EM>, 24:4501-4505, 1996.

<P></P><DT><A NAME="Zhi_90_2006">51</A>
<DD>
Zhi-Jie Tan and Shi-Jie Chen.
<BR>Nucleic acid helix stability: effects of salt concentration, cation
  valence and size, and chain length.
<BR><EM>Biophysical Journal</EM>, 90:1175-1190, 2006.

<P></P><DT><A NAME="Zhi_92_2007">52</A>
<DD>
Zhi-Jie Tan and Shi-Jie Chen.
<BR>Rna helix stability in mixed na+/mg2+ solutions.
<BR><EM>Biophysical Journal</EM>, 92:3615-3632, 2007.

<P></P><DT><A NAME="Tatsuo_124_2002">53</A>
<DD>
Daisuke&nbsp;Miyoshi Tatsuo&nbsp;Ohmichi, Shu-ichi&nbsp;Nakano and Naoki Sugimoto.
<BR>Long rna danglingend has large energetic contribution to duplex
  stability.
<BR><EM>J. Am. Chem. Soc.</EM>, 124:10367-10372, 2002.

<P></P><DT><A NAME="Wahl_152_1987">54</A>
<DD>
Kimmel&nbsp;AR. Wahl&nbsp;GM, Berger&nbsp;SL.
<BR>Molecular hybridization of immobilized nucleic acids: theoretical
  concepts and practical considerations.
<BR><EM>Methods Enzymol</EM>, 152:399-407, 1987.

<P></P><DT><A NAME="Watkins_33_2005">55</A>
<DD>
Santalucia J.&nbsp;Jr. Watkins&nbsp;N.E.
<BR>Nearest-neighbor thermodynamics of deoxyinosine pairs in dna
  duplexes.
<BR><EM>Nuc Acids Res</EM>, 33:6258-6267, 2005.

<P></P><DT><A NAME="Wright_46_2007">56</A>
<DD>
Yanker D.M. Znosko&nbsp;B.M. Wright&nbsp;D.J., Rice&nbsp;J.L.
<BR>Nearest neighbor parameters for inosine-uridine pairs in rna
  duplexes.
<BR><EM>Biochemistry</EM>, 46:4625-4634, 2007.

<P></P><DT><A NAME="Xia_37_1998">57</A>
<DD>
Burkard M.E. Kierzek R. Schroeder S.J. Jiao X. Cox C. Turner&nbsp;D.H. Xia&nbsp;T.,
  SantaLucia&nbsp;J.
<BR>Thermodynamics parameters for an expanded nearest-neighbor model for
  formation of rna duplexes with watson-crick base pairs.
<BR><EM>Biochemistry</EM>, 37:14719-14735, 1998.

<P></P><DT><A NAME="Zhi_34_2006">58</A>
<DD>
Douglas H.&nbsp;Turner Zhi John&nbsp;Lu and David&nbsp;H. Mathews.
<BR>A set of nearest neighbor parameters for predicting the enthalpy
  change of rna secondary structure formation.
<BR><EM>Nucleic Acids Research</EM>, 34:4912-4924, 2006.
</DL>  

<P>

<H1><A NAME="SECTION00070000000000000000">
See Also </A>
</H1>
New versions and 
related material can be found at 

<P>
<TT><A NAME="tex2html29"
  HREF="http://www.pasteur.fr/recherche/unites/neubiomol/meltinghome.html">http://www.pasteur.fr/recherche/unites/neubiomol/meltinghome.html</A></TT> 

<P>
<TT><A NAME="tex2html30"
  HREF="https://sourceforge.net/projects/melting/">https://sourceforge.net/projects/melting/</A></TT>
<P>
<TT><A NAME="tex2html31"
  HREF="http://www.ebi.ac.uk/compneur-srv/melting/">http://www.ebi.ac.uk/compneur-srv/melting/</A></TT>
<P>
You can use <SMALL>MELTING</SMALL> through a web server at 
<TT><A NAME="tex2html32"
  HREF="http://bioweb.pasteur.fr/seqanal/interfaces/melting.html">http://bioweb.pasteur.fr/seqanal/interfaces/melting.html</A><A NAME="tex2html33"
  HREF="http://www.ebi.ac.uk/compneur-srv/melting/melt.php">http://www.ebi.ac.uk/compneur-srv/melting/melt.php</A></TT>   

<P>

<H1><A NAME="SECTION00080000000000000000">
Copyright </A>
</H1>
Melting is copyright 
&#169;1997, 2009 by Nicolas Le Nov&#232;re and Marine Dumousseau.  

<P>
This program is free software; 
you can redistribute it and/or modify it under the terms of the GNU General 
Public License as published by the Free Software Foundation; either version 
2 of the License, or (at your option) any later version.   
  This program 
is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; 
without even the implied warranty of MERCHANTABILITY or FITNESS FOR A 
PARTICULAR PURPOSE.  See the GNU General Public License for more details. 

<P>
You should have received a copy of the GNU General Public License 
along with this program; if not, write to the Free Software Foundation, 
Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307 USA   

<P>

<H1><A NAME="SECTION00090000000000000000">
Acknowledgements</A>
</H1>
Nicolas Joly is an efficient and kind debugger and advisor.  Catherine
Letondal wrote the HTML interface to melting. Thanks to Nirav Merchant,
Taejoon Kwon, Leo Schalkwyk, Mauro Petrillo, Andrew Thompson, Wong Chee Hong, Ivano
Zara for their bug fixes and comments. Thanks to Richard Owczarzy for his magnesium 
correction. Thanks to Charles Plessy for the graphical interface files. Finally thanks
to the usenet helpers, particularly Olivier Dehon and Nicolas Chuche.

<P>

<H1><A NAME="SECTION000100000000000000000">
Authors </A>
</H1>
Nicolas Le Nov&#232;re and Marine Dumousseau, 
<BR>EMBL-EBI, 
Wellcome-Trust Genome Campus
Hinxton Cambridge, CB10 1SD, UK
lenov@ebi.ac.uk

<P>

<H1><A NAME="SECTION000110000000000000000">
History </A>
</H1>

<P>
The Java version has been rewriten from the beginning.
See the file ChangeLog for the changes of the versions 4 and more recent.

<P>

<H1><A NAME="SECTION000120000000000000000">
About this document ...</A>
</H1>
 <STRONG><SMALL>MELTING</SMALL> - nearest-neighbor computation of nucleic acid hybridation</STRONG><P>
This document was generated using the
<A HREF="http://www.latex2html.org/"><STRONG>LaTeX</STRONG>2<tt>HTML</tt></A> translator Version 2002-2-1 (1.71)
<P>
Copyright &#169; 1993, 1994, 1995, 1996,
Nikos Drakos, 
Computer Based Learning Unit, University of Leeds.
<BR>Copyright &#169; 1997, 1998, 1999,
<A HREF="http://www.maths.mq.edu.au/~ross/">Ross Moore</A>, 
Mathematics Department, Macquarie University, Sydney.
<P>
The command line arguments were: <BR>
 <STRONG>latex2html</STRONG> <TT>-dir html -split 0 melting.tex</TT>
<P>
The translation was initiated by Nicolas Le Nov&#232;re and Marine Dumousseau on 2009-08-09
<DIV CLASS="navigation"><HR>
<!--Navigation Panel-->
<IMG WIDTH="81" HEIGHT="24" ALIGN="BOTTOM" BORDER="0" ALT="next_inactive"
 SRC="/usr/share/latex2html/icons/nx_grp_g.png"> 
<IMG WIDTH="26" HEIGHT="24" ALIGN="BOTTOM" BORDER="0" ALT="up"
 SRC="/usr/share/latex2html/icons/up_g.png"> 
<IMG WIDTH="63" HEIGHT="24" ALIGN="BOTTOM" BORDER="0" ALT="previous"
 SRC="/usr/share/latex2html/icons/prev_g.png">   
<BR></DIV>
<!--End of Navigation Panel-->
<ADDRESS>
Nicolas Le Nov&#232;re and Marine Dumousseau
2009-08-09
</ADDRESS>
</BODY>
</HTML>
